-- MySQL dump 10.13  Distrib 5.7.17, for Linux (x86_64)
--
-- Host: lamia.upf.edu    Database: etoxvault
-- ------------------------------------------------------
-- Server version	5.1.42-log

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Dumping data for table `model`
--

LOCK TABLES `model` WRITE;
/*!40000 ALTER TABLE `model` DISABLE KEYS */;
INSERT INTO `model` VALUES (500,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','QT prolongation (at 10nM)','LQT10','2','QT prolongation, LQT','CAV12; HERG4; KCNQ1','Predicted QT after the drug administration at 10nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 10nM','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of Luo-Rudy dynamic AP model of guinea pig cardiac cell. The effect of th','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 10nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/1','Verified','% (of normal QT in ms)',NULL),(501,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','QT prolongation (at 10nM)','LQT10p','2','QT prolongation, LQT','CAV12p; HERG4p; KCNQ1p','Predicted QT after the drug administration at 10nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 10nM','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of Luo-Rudy dynamic AP model of guinea pig cardiac cell. The effect of th','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 10nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/4','Verified','% (of normal QT in ms)',NULL),(22,'2012-05-16 00:00:00','Olivier Taboureau','otab@cbs.dtu.dk','Model developed for eTOX','Phospholipidosis','DIPL 1 DTU','beta','Phospholipidosis','Lowe R. et al. Mol. Pharm. 2010, 7, 1708-1714; Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27','Phospholipidosis','The model is qualitative with a positive or negative outcome. Positive: Strong evidence of phospholipidosis. Negative: No strong evidence of phospholipidosis.','Lowe R. et al. Mol. Pharm. 2010, 7, 1708-1714.',185,'small molecules','No data extra curation was done',55.00,0.0000,1.0000,0.0000,'N/A','compound name','small molecules','Binary classification','Support Vector Machine','185/485','881 PubChem fingerprints','N/A','No Applicability Domain measure was applied.','No Applicability Domain measure was applied.',87,76,7,15,NULL,NULL,NULL,'485 compounds from Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27.',485,87,261,10,127,NULL,NULL,NULL,NULL,NULL,'The model reflects molecular fragments associated with phospholipidosis. ','N/A','N/A','Information about the data can be found at these references:Lowe R. et al. Mol. Pharm. 2010, 7, 1708-1714; Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27','DTU','mammal','http://potentia.cbs.dtu.dk/etox/ws/','boolean','negative,positive','Compounds predicted to be positive are likely to produce phospholipidosis',NULL,'Not verified','na',NULL),(26,'2012-03-19 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Databse (https://www.ebi.ac.uk/chembl/)\r\nDoddareddy MR, Klaasse EC, Shagufta, Ijzerman AP, Bender A.ChemMedChem (2010) 5; 716-729\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-169.','3D Ligand-Based Pharmacophore Model for hERG inhibition','hERG inhibition 1 IL','1','hERG, inhibition, pharmacophore',NULL,'hERG inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The probability for a molecule to inhibit hERG is defined by a pharmacophore fit score and reported with three levels of possible inhibition (LOW, MEDIUM, HIGH).','The obtained dependent variable has been classified as \"low\" probability when the it was not retrieved by the pharmacophore model by virtual screening, \"medium\" when it was retrieved with a fit score of 30, and \"high\" with a fit score > 30.','Doddareddy MR, Klaasse EC, Shagufta, Ijzerman AP, Bender A.ChemMedChem (2010) 5; 716-729',24,'Molecules with Ki < 0.1 microM (most actives) were chosen. Conformations were generated for each molecule. Each conformer was clustered based on the 3D alignments of their pharmacophore features.  12 different clusters were obtained and 2 molecules per cluster were selected for the training set.','The data was curated. The most active molecules (Ki < 0.1 microM) were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) conformations with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D-pharmacophore feature alignments. 12 clusters were obtained. 2 representative molecules for each of the clusters were selected for the training set.\r\n',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3-D Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm, idbgen, developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand-based modelling algorithm in LigandScout developed at InteLigand.','The test set consisted of 105 inhibitors and 133 non-inhibitors extracted from the chEMBL database. ','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features are automatically were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best solution of consensus features resulting from the experiments.','na','na',19,0,0,5,NULL,NULL,NULL,'The test set consisted of 105 inhibitors and 133 non-inhibitors extracted from the chEMBL database. A maximum of 25 conformations for each ligand was generated using LigandScout.',235,83,60,70,22,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1,00, 0.83',NULL,NULL,'3D-Chemical Feature Ligand-based pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','The dependent variable is based on pharmacophore fit scoring function and has been defined as \"low\" probability when it is not a hit during pharmacophore model screening, \"medium\" when it is a hit with a pharmacophore fit score of 30 , and \"high\" when it is a hit with a pharmacophore fit score > 30.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(27,'2012-04-03 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-9.','3D-Ligand-Based Pharmacophore model for CYP1A2 inhibition','CYP 1A2 Inhibition 1 IL','1','CYP 1A2, inhibition, pharmacophore',NULL,'CYP1A2 inhibition:Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The probability for a molecule to inhibit CYP1A2 is defined by a pharmacophore fit score and reported as a Positive or Negative categorical result.','The obtained dependent variable has been categorically named as positive if the query molecule hits CYP1A2, and negative when it was not a hit. The value unknown is the result provided when it is not possible to obtain a prediction for the molecule.','chOX Database (https://www.ebi.ac.uk/chembl/etox/)',23,'Molecules with the highest inhibition ( IC50 < 1.00 ) Conformations were generated for each molecule then clustered based on the 3D alignments of their pharmacophore features. 10 clusters were obtained and 1 to 3 molecules per cluster were selected for the training set.','The database was curated. The data was curated.  The most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) conformations with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments. 10 clusters were obtained. 1 to 3 representative molecules from each of the clusters were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with idbgen, the pattern matching virtual screening algorithm developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand-based pharmacophore modelling algorithm in LigandScout developed at InteLigand.','The test set included 160 inhibitors and 351 non-inhibitors of CYP 1A2 extracted from the chEMBL database','Hydrogen bond donors and acceptors (vectors (if direction is known) and/or spheres (if direction is not defined), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',21,0,0,2,NULL,NULL,NULL,'The test set included 160 inhibitors and 351 non-inhibitors of CYP 1A2 extracted from the chEMBL database in smiles format. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',511,147,54,297,13,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 0.87, 0.82, 059',NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit the isoform 1A2 of Cytochrome P450. . Molecules with the result of Negative have a low likelihood to inhibit the isoform 1A2 of Cytochrome P450. . Unknown indicates that a prediction could not be determined.','/ADME/Metabolism/CYP Inhibition/CYP 1A2 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(28,'2012-04-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Databse (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-9.','3D-Ligand-Based Pharmacophore model for CYP2C9 inhibition','CYP 2C9 Inhibition 1 IL','1','CYP 2C9, inhibition, pharmacophore',NULL,'CYP2C9 inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The probability for a molecule to inhibit CYP2C9 is defined by a pharmacophore fit score and reported as a positive or negative categorical value.','The obtained dependent variable has been categorically named as positive when the screened molecule was a hit, and negative when it wasn\'t a hit. The value unknown is the result provided when it is not possible to obtain a prediction for the molecule.','Data provided by Bayer Health Care Pharmaceuticals (Bayer marketed drugs and from World Drug Index) to the eTOX Intranet. ',22,'Of the 90 molecules screened for 2C9 there were 39 molecules that were defined as 2C9 inhibitors (24 Inhibitors marketed; 15 from the world drug index). A molecule was considered to be an inhibitor with  IC50 < 30 um. \r\nThe 22 molecules selected had IC50\'s < 10 microM.','The data were curated.  Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand. A maximum of 25 conformations per molecule were generated and all conformations for each ligand were annotated with pharmacophore features in order to be used for 3D virtual screening. ',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D chemical feature Ligand-Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm,  idbgen, developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set was created using molecules from the external DB.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',18,0,0,4,NULL,NULL,NULL,'The external validation set was created using 91 inhibitors and 88 non-inhibitors taken from the chEMBL database. A maximum of 500 conformations were generated a for each molecule before virtual screening.',179,85,31,57,6,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 0.98, 0.67',NULL,NULL,'LB pharmacophore modelling ','NA',NULL,'See Public Data on the eTOX Intranet called METB04 and METB05','IL','human',NULL,'boolean','Positive, Negative','Positive compounds are likely to inhibit the isoform 2c9 of Cytochrome P450.','/ADME/Metabolism/CYP Inhibition/CYP 2C9 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(29,'2012-05-14 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Databse (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-9.','3D-Ligand-Based Pharmacophore model for CYP2D6 inhibition','CYP 2D6 Inhibition 1 IL','1','CYP 2D6, inhibition, pharmacophore',NULL,'CYP2D6 inhibition: CYP 2D6 inhibition predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional feature-based pharmacophore model. A positive categorical value indicates the likelihood of interaction. Negative categorical value indicates a low likelihood of interaction. ','The obtained dependent variable has been classified as positive when the molecule was a hit, and negative when it wasn\'t a hit. The value unknown is the result provided when it is not possible to obtain a prediction for the query.','Data submitted by Bayer Health Care (Bayer marketed drugs and from World Drug Index)on the eTOX Intranet. ',12,'Among 90 molecules screened for 2D6 activity there were 12 molecules that we defined as 2D6 inhibitors:. Molecules exhibiting IC50\'s < 30 microM were considered to be active inhibitors of the 2D6 isoform.','The data set was curated. A maximum of 25 conformers were generated per molecule. Each conformation was annotated with pharmacophore features. ',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'Ligand Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set was created using molecules found in the external DB that were reported to have CYP 2D6 inhibition activity (selected molecules had IC50\'s < 30). ','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',11,0,0,1,NULL,NULL,NULL,'The external validation set was created using 90 inhibitors and 91 non-inhibitors extracted from the chEMBL database. A maximum of 500 conformations was generated for each molecule. Pharmacophore feature annotations for each conformation of each molecule were performed using idbgen algorithm in LigandsScout.',181,76,27,64,14,NULL,NULL,'AUC 1,5,10,100 % = 0.00, 0.76, 0.83, 0.58',NULL,NULL,'3D-Chemical FeatureLigand-Based pharmacophore modelling ','NA',NULL,'Refer to the  eTOX Intranet entries called METB04 and METB05','IL','human',NULL,'boolean','Positive, Negative','Query compounds marked Positive are likely to inhibit the isoform 2D6 of the Cytochrome P450.  Query compounds marked negative have low likelihood of inhibiting the 2D6 isoform of cytochrome P450.','/ADME/Metabolism/CYP Inhibition/CYP 2D6 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(30,'2012-05-14 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Databse (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-169.','3D-Ligand-Based Pharmacophore model for CYP3A4 inhibition','CYP 3A4 Inhibition 1 IL','1','CYP 3A4, inhibition, pharmacophore',NULL,'CYP3A4 inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-chemical feature based pharmacophore model. The likelihood that the query molecule may  inhibit CYP3A4 given the categorical value positive or negative when there is low likelihood that query molecule will inhibit the CYP 3A4 isoform.','The dependent variable has been categorically named positive when the virtually screened molecule is a hit, and negative when it isn\'t a hit. The value unknown provided when it is not possible to obtain a prediction for the molecule.','Compound experimental data submitted by Bayer Health Care (marketed drugs and world drug index) to the eTOX Intranet. ',21,'35 of 90 molecules were experimentally determined to inhibit the 3A4 isoform of cytochrome P450 (IC50 < 30 microM). Only molecules (21) with IC50 values less than 10 microM were used in the training set. \r\nThe resulting training set. ','The data set was curated. Training set molecules in smiles format were converted to 3D structures and a maximum of 25 conformations for each molecule were computed and each conformation was annotated with pharmacophore features using with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. ',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set was created using molecules with CYP 3A4 inhibition activity found in the external database.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',18,0,0,4,NULL,NULL,NULL,'The external validation set was created using 95 CYP 3A4 inhibitors and 107 non-inhibitors extracted from the chEMBL database. A maximum of 500 conformations for each molecule was generated and each conformation was annotated with pharmacophore features using idbgen implemented into LigandScout.',202,91,19,88,4,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 0.90, 0.71, 0.66',NULL,NULL,'3D chemical feature ligand-based pharmacophore modelling','NA ',NULL,'Refer to eTOX Intranet entries METB04 and METB05','IL','human',NULL,'boolean','Positive, Negative','Molecules with a categorical value Positive are likely to inhibit the isoform 3A4 of the Cytochrome P450. Those with categorical value of Negative indicates low possibility to inhibit CYP 3A4. Unknown indicates that no prediction can be made.','/ADME/Metabolism/CYP Inhibition/CYP 3A4 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(31,'2012-05-15 00:00:00','InteLigand Support','Support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Databse (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-169.','3D-Ligand-Based Pharmacophore model for Cav1.2 inhibition','Cav1.2 Inhibition 1 IL','1','Cav1.2, inhibition, pharmacophore',NULL,'Cav1.2 inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model. The likelihood for a query molecule to inhibit Cav1.2 is defined categorically as positive while the likelihood not to inhibit Cav1.2 is assigned the categorical value of negative.','The dependent variable has been assigned the categorical value positive when the screened molecule is a hit, and negative when it is not a hit. The value unknown is the result provided when it is not possible to obtain a prediction.','25 molecules defined to inhibit Cav1.2 with IC50 values less than 30 microM were extracted from the chOX database.',5,'25 of the 48 Cav1.2 inhibitors with IC50 values less than 30 microM were selected. Conformations were generated for each molecule then clustered based on the 3D alignments of their pharmacophore features. Five clusters were obtained.','The dataset was curated. A maximum of 25 conformations were generated for each molecule, followed by annotation of each conformation with pharmacophore features. The data set was clustered based on the 3D alignments of their pharmacophore features. Five clusters were obtained.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D chemical feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set consisted of 11 of the CAV1.2 inhibitors with IC50 < 30 microM from the chOX that were not included in the training set.  21 molecules with IC50 values greater than 30 microM were defined as CAV1.2 and served as inactives for the test set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',5,NULL,NULL,0,NULL,NULL,NULL,'The external validation set consisted of 11 CAV1.2 inhibitors with IC50 values less than 30 microM and 21 inactive (IC50 values > 30 microM) taken from the chOX database.',32,9,4,17,2,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.74',NULL,NULL,NULL,NULL,NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Positive molecules are likely to inhibit Cav1.2. Molecules assigned the categorical value Negative are not likely to inhibit Cave1.2.  Molecules assigned the value Unknown indicates that a prediction could not be made.','/Toxicity/Target Safety Pharmacology/Cav1.2 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(32,'2012-05-16 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Databse (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T.,  J Chem Inf Model. 2005 Jan-Feb;45(1):160-169.\r\nHansen et al, J.Chem Inf Model (2009) 49, 9, 2077-2081. DOI 10.1021/ci900161g','3D Ligand-based Pharmacophore Model for AMES Mutagenicity','AMES Mutagenicity 1 IL','1','AMES Mutagenicity, 3D-pharmacophore, Ligand-based, pharmacophore',NULL,'AMES Mutagenicity: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model. It predicts using a 3D pharmacophore fit score whether the query molecule will have either the likelihood to give positive or negative result in the AMES test. ','The dependent variable has a categorical value of positive when the query molecule fits the pharmacophore model (a hit), and negative when it does not. Unknown is the result provided when it is not possible to obtain a prediction.','Hansen et al, J.Chem Inf Model (2009) 49, 9, 2077-2081. DOI 10.1021/ci900161g',7,'A set of 7 molecules representative of each cluster from a pool of molecules from training set source.  ','The training set was curated from the training set source. Starting from smiles codes a maximum of 25 conformations were generated for each molecule then clustered based on the 3D alignments of their pharmacophore features.  7 different clusters were obtained and a representative molecule for each cluster was chosen for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.  ','The test set of 6505 molecules were taken from the training set source. Molecules not used in the training set were used in the test set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',7,0,0,3,NULL,NULL,NULL,'The external validation set data set was created using the 3494 actives and 3008 inactive molecules from the benchmark dataset that were not used in the training set.',6505,1117,2396,606,2383,NULL,NULL,'AUC 1,5,10,100 % = 0.97, 0.89, 0.84, 0.58',NULL,NULL,'3D chemical feature ligand-based pharmacophore modelling','NA',NULL,'Refer to eTOX public data AMES01','IL',NULL,NULL,'boolean','Positive, Negative','A query compound with a Positive value is likely to give a positive AMES test result while one with a Negative value has a low likelihood of giving a positive value in the AMES test.  A result of Unknown means that the likelihood could not be determined.','/Toxicity/Genotoxicity/Mutagenicity/AMES Mutagenicity/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(33,'2012-05-14 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/) code: HEP04\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nRodgers AD, et al, Chem. Res. Toxicol. (2010) 23, 4, 724-32 doi: 10.1021/tx900451r  \r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Ligand Based Pharmacophore model for DILI Human','DILI Human 1 IL','1','Drug Induced Liver Injury, DILI, pharmacophore',NULL,'DILI Human: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model. The likelihood that the query molecule \r\ninduces liver injury is assigned a positive or negative categorical value as a result.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Training set data extracted from Chem Res Toxicol. 2010 23(4):724-732 code: HEP04',9,'9 active molecules were selected based on pharmacophore alignment clustering experiments. Representative molecules for each of the clusters containing the largest number of molecules were selected for the training set. ','The data was curated.  650 molecules actives were selected based on data reported in the training set source and found in chOX database.  The molecules were converted from 2D smiles to 3D with a maximum of 500 conformations per molecule using LigandScout. The molecules were clustered based on 3D- pharmacophore feature alignments. 9 representative molecules for each cluster with the largest number of molecules were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','Active molecules from the pool of 650 molecules in the training set source that were not included in the training set were used in the test set. ','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',8,0,0,1,NULL,NULL,NULL,'The external validation set included 636 actives and 295 inactives for drug induced liver injury from the dataset HEP04 found in the chOX database.',931,179,229,66,457,NULL,NULL,'AUC 1,5,10,100 % = 0.75, 0.58, 0.61, 0.53',NULL,NULL,'3D-Chemical feature Ligand Based  pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to induce liver injury in humans. Molecules with the result of Negative have a low likelihood to induce liver injury. Unknown indicates that a prediction could not be determined.','/Toxicity/Organ Toxicity/Hepatotoxicity/DILI Human/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(34,'2012-05-15 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.\r\nRodgers AD, et al, Chem. Res. Toxicol. (2010) 23, 4, 724-32 doi: 10.1021/tx900451r ','3D-Ligand Based Pharmacophore model for DILI Rodent','DILI Rodent 1 IL','1','Drug Induced Liver Injury, DILI, pharmacophore',NULL,'DILI Rodent: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to induce liver injury assigned a positive categorical value. A low likelihood for a molecule to induce liver injury is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Data Published in the eTOX intranet (code: HEP04). Compounds referenced in the paper Rodgers AD, et al, Chem. Res. Toxicol. (2010) 23, 4, 724-32 doi: 10.1021/tx900451r ',18,'684 molecules reported to be active for human liver injury in rodents were extracted from the chOX Database and a subset of 18 molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments.  ','The data was curated.  650 molecules actives were selected based on data reported in the training set source and found in eTox database.  The molecules were converted from 2D smiles to 3D with a maximum of 500 conformations per molecule using LigandScout. The molecules were clustered based on 3D- pharmacophore feature alignments. 18 representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set molecules were selected from the 650 actives that were not used in the training set.','Hydrogen bond donors and acceptors (vectors (if direction is known) and/or spheres (if direction is not defined), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',15,0,0,3,NULL,NULL,NULL,'The external validation set created included 666 actives and 262 inactive for drug induced liver injury from the dataset HEP04 in the chOX database.',928,455,185,201,77,NULL,NULL,'AUC 1,5,10,100 % = 0.73, 0.74, 0.67, 0.52',NULL,NULL,'3D chemical feature ligand-based pharmacophore modelling','NA',NULL,NULL,'IL','rodent',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to induce liver injury in humans. Molecules with the result of Negative do not have a likelihood to induce liver injury. Unknown indicates that a prediction could not be determined.','/Toxicity/Organ Toxicity/Hepatotoxicity/DILI Rodent/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(35,'2012-05-18 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.\r\nRodgers AD, et al, Chem. Res. Toxicol. (2010) 23, 4, 724-32 doi: 10.1021/tx900451r','3D Ligand-Based Pharmacophore model for DILI Non Rodent','DILI Non-Rodent 1 IL','1','Drug Induced Liver Injury, DILI, pharmacophore',NULL,'DILI Non-Rodent:Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to induce liver injury will give a positive categorical value. A low likelihood for a molecule to induce liver injury will give a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Data found in chOX database (code: HEP04) and referenced in Rodgers AD, et al, Chem. Res. Toxicol. (2010) 23, 4, 724-32 doi: 10.1021/tx900451r ',23,'148 molecules reported to be active for human liver injury in rodents were extracted from the chOX Database and a subset of 23 molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments. ','The data was curated.  148 active molecules were selected based on data reported in the training set source and found in eTox database.  The molecules were converted from 2D smiles to 3D with a maximum of 500 conformations generated per molecule using LigandScout. The molecules were clustered based on 3D- pharmacophore feature alignments. 23 representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical feature Ligand Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','Test set molecules consisted of active molecules not used in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',15,0,0,8,NULL,NULL,NULL,'The external validation set consisted of 125 actives and 131 inactive for drug induced liver injury from the dataset HEP04 published in the chOX database.',256,27,104,27,98,NULL,NULL,'AUC 1,5,10,100 % = 0.24, 0.52, 0.52, 0.52',NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','non-rodent',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to induce liver injury in humans. Molecules with the result of Negative do not have a likelihood to induce liver injury. Unknown indicates that a prediction could not be determined.','/Toxicity/Organ Toxicity/Hepatotoxicity/DILI Non-Rodent/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(36,'2012-05-18 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore model for Rat Carcinogenesis','Carcinogenicity Rat 1 IL','1','Carcinogenesis, Rat, Pharmacophore',NULL,'Carcinogenesis Rat: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to induce carcinogenesis in rat is assigned a positive categorical value. A low likelihood for a molecule to induce carcinogenesis in rat is assigned a value of negative. ','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Data found in the chOX database.',6,'6 of the most potent molecules reported fro carcinogenesis in rates were selected among 271 actives reported in the chOX database. ','The data was curated.  6 molecules actives were selected based on data reported in the training set source and found in chOX database.  The molecules were converted from 2D smiles to 3D and a maximum of 500 conformations per molecule were computed using LigandScout. A ligand-based pharmacophore model was generated based on multiple pharmacophore alignment experiments.\r\n',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D chemical feature Ligand Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment  Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','Active molecules not used in the training set were used in the test set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',5,0,0,1,NULL,NULL,NULL,'The external validation included a set of 265 actives. No inactives were available for model validation.',265,180,0,0,85,NULL,NULL,NULL,NULL,NULL,'3D-chemical feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','rat',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to carcinogenicity in rats. Molecules with the result of Negative have a low likelihood of inducing carcinogenicity in rats. Unknown indicates that a prediction could not be determined.','/Toxicity/Carcinogenicity/Carcinogenicity Rat/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(37,'2012-05-18 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore model for Mouse Carcinogenesis','Carcinogenicity Mouse 1 IL','1','Carcinogenesis, Mouse, Pharmacophore',NULL,'Carcinogenesis Mouse: It predicts through a pharmacophore model the capability for a molecule to induce carcinogenesis in mouse giving a positive or negative categorical value as result.','The obtained dependent variable has been categorically named as positive when the screened molecule was a hit, and negative when it wasn\'t a hit. The value unknown is the result provided when it is not possible to obtain a prediction for the molecule.','21 of 362 actives for carcinogenesis in mouse found in the chOX database were selected for the training set.',21,'362 molecules reported to be active for carcinogenicity in mouse were extracted from the chOX database and a subset of 21 molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments. ','The data was curated. 362 molecules actives were selected based on data reported in the training set source and found in eTox database.  The molecules were converted from 2D smiles to 3D with a maximum of 500 conformations per molecule using LigandScout. The molecules were clustered based on 3D- pharmacophore feature alignments. 21 representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','Test set molecules included the pool of actives that were not used in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of chemical features.','na','na',16,0,0,5,NULL,NULL,NULL,'The external validation set included 362 actives from the chOX database. No inactive molecules were available.',362,201,0,0,161,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','mouse',NULL,'boolean','Positive, Negative','Positive compounds are likely to induce cancer in mouse.\r\n\r\nMolecules with the result of Positive have a likelihood to induce carcinogenicity in mouse. Molecules with the result of Negative have have a low likelihood to induce carcinogenesis in mouse. Unknown indicates that a prediction could not be determined.','/Toxicity/Carcinogenicity/Carcinogenicity Mouse/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(38,'2012-05-18 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for Mutagenic Chromosomal Damage','Mutagenic Chromosomal Damage 1 IL','1','Mutagenesis, Chromosomal Damage, Pharmacophore',NULL,'Mutagenic Chromosomal Damage: It predicts through a pharmacophore model the likelihood for a molecule to induce mutagenesis with positive or negative categorical value given as a result.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','38 active molecules were extracted from in the chOX database. A subset of 6 active molecules were used for the training set.',6,'6 active molecules were selected based on pharmacophore alignment clustering experiments of 38 actives. Representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set. ','The data was curated.  38 molecules actives were selected based on data reported in the training set source and found in eTox database.  The molecules were converted from 2D smiles to 3D with a maximum of 500 conformations per molecule using LigandScout. The molecules were clustered based on 3D- pharmacophore feature alignments. 6 representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','32 Active molecules not used in the training set were used in the test set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is known) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',5,0,0,1,NULL,NULL,NULL,'The external validation set included 32 actives from the chOX database. Known inactive molecules were not available.',32,29,0,0,3,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL',NULL,NULL,'boolean','Positive, Negative','Positive compounds are likely to induce chromosomal damage. \r\nMolecules with the result of Positive have a likelihood to induce chromosomal damage. Molecules with the result of Negative have a low  likelihood to induce chromosomal damage. Unknown indicates that a prediction could not be determined.','/Toxicity/Genotoxicity/Mutagenicity/Mutagenetic Chromosomal Damage/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(39,'2012-05-18 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore model for Acetylcholinesterase Inhibition','AChE Inhibition 1 IL','1','Acetylcholinesterase, Inhibition, Pharmacophore, AChE',NULL,'Acetylcholinesterase Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model. A positive categorical value is given when there is likelihood that the molecule will inhibit AChE and negative when there is low likelihood. ','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','11 of the most potent AChE inhibitors were extracted from the chOX database.',11,'Molecules with IC50 < 1.0 microM (most actives) were chosen. Conformations were generated for each molecule and then clustered based on the 3D alignments of their pharmacophore features.  ','The data was curated.  Actives were selected based on data reported in the training set source.  The molecules were converted from 2D smiles to 3D with a maximum of 500 conformations per molecule using LigandScout. The molecules were clustered based on 3D- pharmacophore feature alignments. 11 representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set.\r\n',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included molecules from the external data set.','Hydrogen bond donors and acceptors (vectors if direction is known) and/or spheres (if direction is not defined), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space).','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features.','na','na',7,0,0,4,NULL,NULL,NULL,'The test set included 74 inhibitors and 76 non-inhibitors taken from the chOX database. A maximum of 500 conformers were generated for each molecule before virtual screening.',150,61,27,49,13,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.88',NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Compounds marked Positive have a likelihood to inhibit Acetylcholinesterase. Compounds marked Negative have a low likelihood of inhibiting Acetylcholinesterase. Unknown indicates that a prediction could not be made.','/Toxicity/Target Safety Pharmacology/AchE Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(40,'2012-05-18 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.ChOX Database','3D Ligand-Based Pharmacophore Model for MAPkinase1 inhibition','MAPK1 Inhibition 1 IL','1','MAPkinase1, Inhibition, Pharmacophore',NULL,'MAPKinase Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to induce liver injury assigned a positive categorical value. A low likelihood for a molecule to inhibit MAPK1 is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','MAPK1 inhibitors with IC50 < 1 microM found in the chOX database.',8,'The training set included MAPK1 strong inhibitors (IC50 < 1 microM) extracted from chOX database.','The data was curated.  8 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.\r\n',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.',' The test set included MAPK1 inhibitors from the external data set that were not included in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',6,0,0,2,NULL,NULL,NULL,'The test set included 22 inhibitors and 42 non-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.',64,20,13,29,2,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 0.98, 074',NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Positive compounds are likely to inhibit MAPK1.\r\nMolecules with the result of Positive have a likelihood to inhibit MAPK1. Molecules with the result of Negative have a low likelihood to inhibit MAPK1. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/MAPK1 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(41,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Ligand-Based Pharmacophore model for Calpain 1 Inhibition','CANPL1 Inhibition 1 IL','1','Calpain 1, Inhibition, Pharmacophore',NULL,'Calpain 1 Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit calpain is assigned a positive categorical value. A low likelihood for a molecule to inhibit calpaan is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','51 calpain inhibitors (IC50 < 0.015 microM) were taken from the chOX database to build the training set.',51,'The training set included 51 Calpain inhibitors with IC50 < 0.015 microM (highly actives) from chOX database. Conformations were generated for each and each conformation was annotated with pharmacophore features.','The data was curated.  51 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D SDF format with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','Training set smiles were converted to 3D with the idbgen module of LigandScout3.0. This was followed by a clusterization and the production of the pharmacophore model using the ligand-based pharmacophore model tool in LigandScout 3.0.',' The test set included both active inhibitors and inactive inhibitors that were not used in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',49,0,0,2,NULL,NULL,NULL,'The test set included 114 inhibitors and 95 non-inhibitors taken from the chOX database. Each molecule was converted from 2D smiles code to 3D conformations with a maximum of 500 conformers. Each conformation for each molecule was annotated with pharmacophore features to be used for virtual screening in LigandScout. ',209,107,7,88,7,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.68',NULL,NULL,'3D-Chemical Ligand-Based pharmacophore modelling','NA ',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit Calpain 1. Molecules with the result of Negative have a low likelihood to inhibit Calpain 1. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/CANPL1 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(42,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore model for Factor 10 Inhibition','F10 Inhibition 1 IL','1','Factor 10, Inhibition, Pharmacophore',NULL,'Factor 10 Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Coagulation Factor X is assigned a positive categorical value. A low likelihood for a molecule to inhibit Coagulation Factor X is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Training set source consisted of a subset of 21 out of 40 molecules with highest activity values for inhibition of Coagulation Factor X extracted from the chOX database.',21,'40 molecules with the highest activity for inhibition of Factor 10 were extracted from the chOX database. A subset of 21 molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments. ','The data was curated.  40 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). The molecules were clustered based on 3D- pharmacophore feature alignments. 21 representative molecules for each of the clusters with the largest number of molecules in the cluster were selected for the training set.\r\n3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout. ',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore model','raining set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 477 (92 actives, 385 inactives) molecules from CHOX Database external to the training set.  \r\n','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',18,0,0,3,NULL,NULL,NULL,'The external set included 477 (92 actives ad 385 inactives) molecules from CHOX Database. \r\nThe molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.',477,85,218,167,7,NULL,NULL,'AUC 1,5,10,100 % = 0.74, 0.87, 0.93, 0.86',NULL,NULL,'3D chemical feature Ligand-Based pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit Coagulation Factor X. Molecules with the result of Negative have a low likelihood to inhibit Coagulation Factor X. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/F10 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(43,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Ligand-Based Pharmacophore model for Angiotensin Converting Enzyme Inhibition','ACE Inhibition 1 IL','1','Angiotensin Converting Enzyme, ACE, Inhibition, Pharmacophore',NULL,'ACE Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Angiotensin converting enzyme is assigned a positive categorical value. A low likelihood for a molecule to inhibit Angiotensin converting enzyme is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Training set included a pool of 27 highly active Angiotensin Converting Enzyme inhibitors taken from a pool of 50 of the most active ACE inhibitors extracted from the chOX database.',27,'50 highly active molecules for ACE inhibition were extracted from the chOX Database.  A subset of 27 molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments and chemical diversity.','The data was curated. 50 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments. 27 representative molecules for each of the clusters with the largest chemical diversity were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 88 ACE inhibitors and 92 non-inhibitors from the chOX database that were not included in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',27,0,0,0,NULL,NULL,NULL,'The test set included 88 inhibitors and 92 non-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',180,88,25,67,0,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.84',NULL,NULL,'3D-Chemical Feature Ligand Based pharmacophore modelling','NA ',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit Angiotensin converting enzyme. Molecules with the result of Negative have a low likelihood to to inhibit Angiotensin converting enzyme. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/ACE Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(44,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for Serine Beta-Lactamase-Like Protein Inhibition','LACTB Inhibition 1 IL','1','Serine beta-lactamase-like protein, Inhibition, Pharmacophore, LACTB',NULL,'LACTB Inhibition:Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit serine beta-lactamase-like protein is assigned a positive categorical value. A low likelihood for a molecule to inhibit serine beta-lactamase-like protein is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','A subset of 30 highly active molecules were selected from an initial pool of the 50 most active molecules for inhibition of serine beta-lactamase-like protein found in the chOX database',30,'The training set included 50 molecules with the highest inhibition activity values. A subset of 30 highly active molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments. Molecules from each cluster were selected based on chemical diversity.','The data was curated.  50 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments. 30 representative molecules for each of the clusters were selected based on chemical diversity. ',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 67 inhibitors and 72 non-inhibitors molecules that were not in the training set and taken from the chOX database.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',28,0,0,2,NULL,NULL,NULL,'The test set included 67 inhibitors and 72 not-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',134,55,28,44,7,NULL,NULL,'AUC 1,5,10,100 % = 0.00,0.29, 0.35, 0.60',NULL,NULL,'3D chemical feature Ligand-Based pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit serine beta-lactamase-like protein. Molecules with the result of Negative have a low likelihood to inhibit serine beta-lactamase-like protein. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/LACTB Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(45,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for Retinoic Acid Receptor alpha inhibition','RAR-alpha Inhibition 1 IL','1','Retinoic Acid Receptor alpha, Inhibition, Pharmacophore, RAR-Alpha',NULL,'RAR-alpha Inhibition:Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Retinoic Acid Receptor alpha protein is assigned a positive categorical value. A low likelihood for a molecule to inhibit Retinoic Acid Receptor alpha protein is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','The training set included 5 of the molecules with highest inhibition activity values and chemical diversity from the chOX database.',5,'Molecules with highest inhibition values were chosen from chOX database. Conformations were generated for each molecule then clustered based on the 3D alignments of their pharmacophore features. 5 molecules from clusters with great chemical diversity were selected for the training set. ','The data was curated.  8 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments.5 representative molecules for each of the clusters based on chemical diversity were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.\r\n','The test set included 45 inhibitors and 24 non-inhibitors taken from the chOX database.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',4,0,0,1,NULL,NULL,NULL,'The test set included 45 inhibitors and 24 non-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',69,23,18,22,6,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 0.91, 0.61',NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore modelling ','NA ',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit Retinoic Acid Receptor alpha protein. Molecules with the result of Negative have a low likelihood to inhibit Retinoic Acid Receptor alpha protein. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/RAR-alpha Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(46,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore model for Multidrug resistance protein 1 Inhibition','ABCB1 Inhibition 1 IL','1','Multidrug resistance protein 1, inhibition, pharmacophore, ABCB1',NULL,'ABCB1 Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Multidrug resistance protein alpha is assigned a positive categorical value. A low likelihood for a molecule to induce liver injuryto inhibit Multidrug resistance protein alpha is assigned a value of negative.\r\n','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','The training set included 17 molecules with the highest inhibition activity values and chemical diversity from the chOX database.',17,'17 molecules with highest inhibition activity were selected.  Conformations were generated for each molecule then each conformation of each molecule were clustered based on the 3D alignments of their pharmacophore features. The  17 molecules represented a subset with the highest inhibition activity values and chemical diversity. ','The data was curated.  17 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.\r\n',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 111 inhibitors and 19 non-inhibitors taken from the chOX database.',NULL,NULL,'na','na',13,0,0,4,NULL,NULL,NULL,'The test set included 111 inhibitors and 19 non-inhibitors taken from the chOX database. For each molecule was generated a maximum of 500 conformers before screening.The test set included 111 inhibitors and 19 non-inhibitors taken from the chOX database.',130,50,14,5,61,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.62',NULL,NULL,'3D-Chemical Feature Ligand-Based Pharamacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit Multi-drug resistance protein alpha. Molecules with the result of Negative have a low likelihood to inhibit Multidrug resistance protein alpha. Unknown indicates that a prediction could not be determined.','/ADME/Transport/Transporters/ABCB1 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(47,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore model for ATP-binding cassette sub-family G member 2 Inhibition','ABCG2 Inhibition 1 IL','1','ATP-binding cassette sub-family G member 2, inhibition, pharmacophore, ABCG2',NULL,'ABCG2 Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit ATP-binding cassette sub-family G member protein is assigned a positive categorical value. A low likelihood for a molecule to inhibit ATP-binding cassette sub-family G member protein is assigned a value of negative.\r\n','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','24 molecules with highest inhibition activity for ABCG2 Inhibition extracted from the chOX database.',24,'The training set included 24 molecules with the highest inhibition activity values.','The data was curated. 24 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 32 inhibitors and 18 non-inhibitors taken from the chOX database.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',18,0,0,6,NULL,NULL,NULL,'The test set included 32 inhibitors and 18 non-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',50,11,15,21,3,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.61',NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit ATP-binding cassette sub-family G member protein. Molecules with the result of Negative have a low likelihood to induce liver injuryto inhibit ATP-binding cassette sub-family G member protein. Unknown indicates that a prediction could not be determined.','/ADME/Transport/Transporters/ABCG2 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(48,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for Solute Carrier Family 22 Member 1 Inhibition','SLC22A1 Inhibition 1 IL','1','Solute carrier family 22 member 1, inhibition, pharmacophore, SLC22A1',NULL,'SLC22A1 Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule  to inhibit Solute carrier family 22 member 1 protein is assigned a positive categorical value. A low likelihood for a molecule  to inhibit Solute carrier family 22 member 1 protein is assigned a categorical value of negative. ','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','The training set included 5 molecules with the highest inhibition activity values (IC50 < 10 microM) extracted from the chOX database.',5,'The training set included 5 molecules with the highest inhibition activity values (IC50 < 10 microM). Conformations were generated for each molecule. 3D pharmacophore models were created based on multiple 3D-pharmacophore feature alignments using LigandScout. ','The data was curated.  8 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 25 inhibitors and 14 non-inhibitors taken from the chOX database that were not included in the training set data.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',3,0,0,2,NULL,NULL,NULL,'The test set included 25 inhibitors and 14 non-inhibitors taken from the chOX database. For each molecule  a maximum of 500 conformations were generated. The multiconformational datasets were annotated with pharmacophore features and used for virtual screening with LigandScout.',39,19,2,12,6,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.65',NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit solute carrier family 22 member 1 protein. Molecules with the result of Negative have a low likelihood to inhibit solute carrier family 22 member 1 protein. Unknown indicates that a prediction could not be determined.','/ADME/Transport/Transporters/SLC22A1 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(49,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nPreissner, S et al., (2009) Nucleic Acids Res, 38:D237-D243; doi:10.1093/nar/gkp970\r\nSuperCYP Database:\r\nhttp://bioinformatics.charite.de/supercyp/\r\nSussman, J.L.,et al,  (1998) Protein Data Bank (PDB): database of\r\nthree-dimensional structural information of biological\r\nmacromolecules. Acta Crystallogr. D Biol. Crystallogr., 54,\r\n1078–1084; PDB code: 1R9O.\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Consensus Ligand-Based and Structure-Based Pharmacophore Model for CYP 2C9 Selectivity','CYP 2C9 Selectivity 1 IL','1','CYP 2C9, Selectivity, pharmacophore',NULL,'CYP 2C9 Selectivity: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule for a molecule to selectively bind the CYP450 isoform 2C9 is assigned a positive categorical value. A low likelihood for a molecule to selectively bind the CYP450 isoform 2C9 is assigned a categorical value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','16 selective substrates for 2C9 were extracted from the Supercyps Database and from protein databank. PDB Code: 1R9O. ',16,'16 selective substrates for 2C9 were extracted from the Supercyps Database and from protein databank. PDB Code: 1R9O. 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout. 3D Structure-based pharmacophore model was generated based on the interactions between CYP2C9 and the crystal structure derived conformation from the PDB code: 1R9O. A 3D-shared feature pharmacophore model was created from alignment of the resulting Ligand-Based Model and the structure-based pharmacophore model. ','The data was curated.  16 selective substrates for 2C9 were selected from the Supercyps database and Protein Databank (1R9O).  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout. The structure-based pharmacophore model was created automatically using LigandScout and PDB code 1R9O.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Consensus Pharmacophore model from Ligand-Based and Structure-Based pharmacophore methods.','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand. The structure-based pharmacophore model interactions derived using LigandScout algorithms.','The test set included 57 molecules selective for 2C9 and 200 molecules selective substrates for 2D6 and 3A4 and not substrates of 2C9.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.  Structure-based pharmacophore model features are only added to the model if there is a complimentary feature partner in the active site that is in the correct distances, geometry and tolerances. Excluded volumes are added to define the binding pocket.','na','na',15,0,0,4,NULL,NULL,NULL,'The test set included 57 molecules selective for 2C9 and 200 molecules selective substrates for 2D6 and 3A4 and not substrates of 2C9. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',257,19,155,45,38,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 0.95, 0.57',NULL,NULL,'3D-Chemical Feature Ligand-Based and Structure-Based pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to be substrate of cytochrome P450 isoform 2C9. Molecules with the result of Negative have a low likelihood  to be substrate of cytochrome P450 isoform 2C9. Unknown indicates that a prediction could not be determined.','/ADME/Metabolism/Phase I/CYP 2C9 Selectivity/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(50,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','Preissner, S et al., (2009) Nucleic Acids Res, 38:D237-D243; doi:10.1093/nar/gkp970\r\nSupercyps Database:\r\nhttp://bioinformatics.charite.de/supercyp/Sussman, J.L.,et al,  (1998) Protein Data Bank (PDB): database of\r\nthree-dimensional structural information of biological\r\nmacromolecules. Acta Crystallogr. D Biol. Crystallogr., 54,\r\n1078–1084; PDB code: 3UA1.\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Consensus Ligand-Based and Structure-Based Pharmacophore Model for CYP 3A4 Selectivity','CYP 3A4 Selectivity 1 IL','1','CYP 3A4, Selectivity, pharmacophore, Cytochrome P450',NULL,'CYP 3A4 Selectivity: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to selectively bind the CYP 3A4 isoform is assigned a positive categorical value. A low likelihood for a molecule to selectively bind the CYP 3A4 isoform is assigned a categorical value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','33 selective substrates for the CYP P450 isoform 3A4 were extracted from the Supercyps and from the protein databank, PDB code: 3UA1.',33,'The training set included 33 substrates for the Cytochrome P450 isoform 3A4 from the Supercyps database and protein data bank code: 3UA1.','The data was curated.  32 substrates for the CYP P450 isoform 3A4 from the Supercyps database  were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout. 1 3D structure-based pharmacophore model was created from a ligand in the active site of an x-ray derived structure CYP 3A4 using LigandScout. The shared -feature (consensus) pharmacophore model was created using the 3D ligand-and structure-based pharmacophore models.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based and Structure-Based consensus pharmacophore model ','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.  3D-Structure-based pharmacophore models are automatically derived using algorithms in LigandScout developed at InteLigand.','The test set included 208 molecules selective for CYP 3A4 and 128 decoy molecules selective for CYP 2D6 and 2D9 and not substrates of 3A4. ','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3D geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set. Structure-based pharmacophore features are based on interactions between a ligand and macromolecule in the active site.  A feature is added to the model only if there are complimentary binding partners in the active site with correct distances and physico-chemical properties.','na','na',26,0,0,7,NULL,NULL,NULL,'The test set included 208 molecules selective for CYP 3A4 and 128 decoy molecules selective for CYP 2D6 and 2D9 and not substrates of 3A4.  The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',336,131,65,63,77,NULL,NULL,'AUC 1,5,10,100 % = 0.97, 0.86, 0.80, 0.58',NULL,NULL,'3D-Chemical Feature Ligand- and Structure-Based Consensus Pharmacophore Model ','NA ',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to be substrate of CYP 3A4. Molecules with the result of Negative have a low likelihood to be substrate of CYP 3A4. Unknown indicates that a prediction could not be determined.','/ADME/Metabolism/Phase I/CYP 3A4 Selectivity/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(51,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','Preissner, S et al., (2009) Nucleic Acids Res, 38:D237-D243; doi:10.1093/nar/gkp970\r\nSupercyps Database: \r\nhttp://bioinformatics.charite.de/supercyp/\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Consensus Ligand-Based and Structure-Based Pharmacophore Model for CYP 2D6 Selecivity','CYP2D6 Selectivity 1 IL','1','CYP 2D6,  Selecivity, pharmacophore, cytochrome P450',NULL,'CYP 2D6 Selecivity: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to selectively bind the CYP 2D6 isoform is assigned a positive categorical value. A low likelihood for a molecule to selectively bind the CYP 2D6 isoform is assigned a categorical value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','23 active CYP 2D6 molecules were extracted from the Supercyps Database. Models based on 3 active site conformations of CYP 2D6 derived from Molecular Dynamics Simulations from the Nico Vermeulen LAB.',23,'The training set included 23 selective substrates for CYP 2D6 taken from the Supercyps database. ','The data was curated.  23 substrates for the CYP P450 isoform 3A4 from the Supercyps database  were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout. 3 3D structure-based pharmacophore models were created from a ligand in the active sites of CYP 2D6 using LigandScout. The structures were derived from Molecular Dynamics simulations from Nico Vermeulen group. The shared -feature (consensus) pharmacophore model was created using the 3D ligand-and structure-based pharmacophore models.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based and Structure-Based Consensus Pharmacophore Model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.  3D-Structure-based pharmacophore models are automatically derived using algorithms in LigandScout developed at InteLigand.','The test set included 71 molecules selective for CYP 2D6 and 186 decoy selective substrates for CYP 3A4 and CYP 2C9 and not substrate of CYP 2D6.  ','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set. Structure-based pharmacophore features are based on interactions between a ligand and macromolecule in the active site.  A feature is added to the model only if there are complimentary binding partners in the active site with correct distances and correct physico-chemical properties.','na','na',18,0,0,5,NULL,NULL,NULL,'The test set included 71 molecules selective for CYP 2D6 and 186 decoy molecules selective for CYP 3A4 and 2C9 and not a substrate of 2D6.  The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.',0,39,146,20,32,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 0.98, 0.97, 0.73',NULL,NULL,'3D Chemical Feature Ligand- and Structure-Based Consensus Pharmacophore Model','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to be substrate of CYP 2D6. Molecules with the result of Negative have a low likelihood to be substrate of CYP 2D6. Unknown indicates that a prediction could not be determined.','/ADME/Metabolism/Phase I/CYP 2D6 Selectivity/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(52,'2012-05-21 00:00:00','Claire Ellison','c.m.ellison@ljmu.ac.uk','K. R. Przybylak and  M.T. D. Cronin, Mol. Inf.  2011, 30(5), 415 – 429.\r\nDOI: 10.1002/minf.201000164','Drug-Induced Phospholipidosis (DIPL) Structural Alerts','DIPL 1 LJMU','1','Phospholipidosis, DIPL','DIPL 1LM; DIPL 1MN; DIPL 1DTU; DIPL 1FIMIM; DIPL 2FIMIM','Phospholipidosis','Binary classificaiton. Inducer (positive) or non-inducer (negative) of phospholipidosis.','N. L. Kruhlak, S. S. Choi, J. F. Contrera, J. L. Weaver, J. M. Willard,\r\nK. L. Hastings, L. F. Sancilio, Toxicol. Mech. Meth. 2008, 18(2-3),\r\n217–227.\r\nDOI:10.1080/15376510701857262',446,'Diverse collection','From a total of 583 compounds, published by Kruhlak et al., 98 in house chemicals without structures were removed resulting in a total of 485 chemicals. A further 35 chemicals for which descriptors could not be calculated (because of software limitations). Finally four duplicate chemicals were removed giving dataset of 446 compounds.',0.21,NULL,NULL,NULL,'N/A','chemical name','N/A','Structural alerts','Structural alerts implemented using the KNIME platform.','External validation only','N/A','N/A','Applicability domain is intrinsic.','Defined by the presence of structural alerts.',78,312,42,14,NULL,NULL,'N/A','62 chemicals compiled from the literature',62,10,44,6,2,NULL,NULL,'N/A',NULL,NULL,'Structural alerts  reflect meaningful molecular fragments associated with phospholipidosis: hydrophobic, cyclic moieties with peripheral, amine groups. ','Further information can be found in original paper: K. R. Przybylak and  M.T. D. Cronin, Mol. Inf.  2011, 30(5), 415 – 429.\r\nDOI: 10.1002/minf.201000164','Additional references can be found in original paper: K. R. Przybylak and  M.T. D. Cronin, Mol. Inf.  2011, 30(5), 415 – 429.DOI: 10.1002/minf.201000164 ','K. R. Przybylak and  M.T. D. Cronin, Mol. Inf.  2011, 30(5), 415 – 429.DOI: 10.1002/minf.201000164','LJMU','mammal','http://etox.ljmu.ac.uk/etoxwsapi/v2/dir','boolean','Negative, Positive','Results given as positive (Inducer) or negative (non-inducer) for phospholipidosis. Drugs which are classified as inducers are likely to cause drug-induced phospholipidosis','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/1','Verified','na',NULL),(53,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','The dataset PHOSPHO1 was downloaded from eTOX web site, and was published on Mol. Pharmaceutics, 2010, 7 (5), pp 1708–1714.','Phospholipidosis','DIPL 1 LMD','1','Phospholipidosis','J Chem Inf Model. 2013 Jun 24;53(6):1436-46. doi: 10.1021/ci400113t','Phospholipidosis','positive: the compound is predicted to cause phospholipidosis negative: the compound is predicted to not cause phospholidosis unknown: some error occurred','PHOSP01 eTOX dataset',176,'small molecules','Training set was autoscaled as per Volsurf+ v1.7l method',55.68,-1.0000,1.0000,NULL,'No','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','3D QSAR','PLS with categorization as implemented in eTOXlab v 0.9','5% of the original PHOSPHO1 dataset were randomly assigned to the test set, and never used in the process of building the model.','128 Volsurf+ descriptors','All Volsurf+ descriptors were used','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',79,63,15,19,NULL,NULL,'specificity: 0.808, sensitivity: 0.806, MCC:0.611','A set of 8 compounds was randomly selected from the whole set of compounds, and was never used in building the model, only to test the performance after building the model.',8,3,3,1,1,NULL,NULL,NULL,NULL,NULL,'N/A','N/A','N/A','N/A','LMD','N/A','http://etoxsys.eu/current','boolean','negative,positive','A positive/negative prediction means the compound is predicted to cause/not cause phospholipidosis. ','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/1','Verified','na',NULL),(54,'2012-05-18 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','Wang et al. doi:10.1023/A:1008763405023','XLOGP','XLOGP 1 MN','beta','logp, xlogp','ALOGP, CLOGP doi:10.1021/jp980230o','LogP - octanol/water partition coefficient.','The value of LogP','Data was taken from XLOGP2 paper by Wang et al. doi:10.1023/A:1008763405023',1853,'variety of small neutral organic molecules.','Since this model is a reimplementation of the published method (see Supporting info) no curation was applied. In the original work of  Wang et al. molecular modelling was don using Sybyl software. Molecules were minimized using Tripos force field and atomic partial charges were calculated using the MNDO method.',NULL,-3.5600,7.6600,1.7687,'no','compound name','N/A','multi linear regression','Contributions of structural features and correction factors were derived by multivariate regression analysis of the training set.','As a test set an external data set published by Mannhold et al. doi:10.1002/qsar.19960150506 was used.','90 structural features and 10 correction factors','none','The method is parameterized for neutral organic molecules.','No Applicability Domain measure was applied.',NULL,NULL,NULL,NULL,0.9327,0.3921,'N/A','The external data set used as a test set was published as a reference data set used for the performance comparison of various LogP procedures.',138,NULL,NULL,NULL,NULL,0.8780,0.5039,'N/A',NULL,NULL,'Some insight into mechanistic like model interpretation can be found in the original publication by Wang et al.','Wang et al. doi:10.1021/jp980230o','N/A','The model available in eTOXsys is a reimplementation of the original XLOGP2 method by Wang et al. Additional information in available on http://www.molecular-networks.com/products/adrianacode (accessed 15.11.2013) http://www.ics.uci.edu/~dock/manuals/xlogp2.1/ (accessed 15.11.2013)','MN','N/A','N/A','float','(-inf,inf)','The returned value is a log10 of octanol/water partition coefficient (LogP) as estimated by XLOGP2 method.',NULL,'Not verified','na',NULL),(55,'2012-05-18 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','Terfloth et al. doi:10.1021/ci700010t','CYP Isoform Specificity','CYP Isoform Specificity 1 MN','beta','CYP, cytochrome P450,  3A4, 2D6,  2C9','CYP 3A4 Selectivity 1 IL, CYP 2D6 Selectivity 1 IL, CYP 2C9 Selectivity 1 IL','The Isoform Specificity of Cytochrome P450 3A4, 2D6, and 2C9 substrates.','Dependent variable indicate the predominant CYP isoform for particular compound. Possible values: 3A4, 2D6, and 2C9. ','Data was extracted form the following literature: (1) Manga et al. doi:10.1080/10629360412331319871 (2) Bertz et al. doi:10.2165/00003088-199732030-00004 (3) Tredger, J. M.; Stoll, S. Cytochromes P450 - Their impact on drug treatment. Hosp. Pharm. 2002, 9, 167-173.\r\nPart of the data was extracted from Metabolite reaction database (Metabolite database; MDL Inc. http://www.mdl.com/products/predictive/metabolite/index.jsp – accessed March 4, 2007).',379,'Drug-like molecules. Percentages of CYP targets: 3A4 59.1% / 2D6 30.1% / 2C9 10.8%.','The detailed data curation procedure can be found in the original paper. Following is only a summary: missing stereochemistry was added; wrong constitution were corrected; duplicates were removed.',NULL,NULL,NULL,NULL,'N/A','SMILES, names and DB IDs','N/A','Decision tree','The model was built by combining the variable selection with a C4.5/J48 (Weka) decision tree for which the minimum number of instances per leaf was set to 6.','The model was developed using the whole data set. However, in order to create and prove the performance of the modeling methodology the data set was split in to modeling and test sets in proportion 146/233. The modeling part included all compounds from Manga et al paper. The testing part included compounds extracted from Metabolite database.','3D Autocorrelation, Global Molecular, Shape/Size-Related Descriptors, and Substructure Counts','Variables was selected by the CfsSubsetEval algorithm implemented in Weka with ExhaustiveSearch search method.','The model is applicable to drug-like small organic molecules. The performance of the model was validated by 10-fold and LOO validation yielding approximately 86% of correctly classified compounds. Performance against training set was approximately 90%.','No Applicability Domain measure was applied. If compound cannot be predicted \"unknown\" prediction is issued.',NULL,NULL,NULL,NULL,NULL,NULL,'N/A','No external validation was applied, see supporting information.',0,NULL,NULL,NULL,NULL,NULL,NULL,'N/A',NULL,NULL,'The original publication provides insights on mechanistic basis of the model.','Terfloth et al. doi:10.1021/ci700010t','N/A','In the course of model development the train and validation sets were used to prove the performance of the modeling methodology. The accuracy of prediction for training set was 87.7 % and the accuracy of prediction for validation set was 79,8 %.','MN','human','N/A','boolean','3A4, 2D6, 2C9','Outcome is the predominant CYP isoform. Possible values: 3A4, 2D6, 2C9 or unknown.',NULL,'Not verified','na',NULL),(56,'2012-05-18 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','N/A; see also: Training set source','Drug Induced Phospholipidosis','DIPL 1 MN','0.2','Phospholipidosis, DIPL','DIPL 1 LMD, DIPL 1 LJMU, DIPL 1 FIMIM, DIPL 2 FIMIM','Drug induced phospholipidosis.','This is the qualitative model. Possible values: positive, negative. Positive: Strong evidence of phospholipidosis. Negative: No strong evidence of phospholipidosis.','Data used in the model development was collected from various literature sources and from ChOX DB: Lowe et al. doi:10.1021/mp100103e Pelletier et al.  doi:10.1021/ci6004542 Przybylak et al. doi:10.1002/minf.201000164 Cerep data set, accessed 2013.11.18 (http://www.cerep.fr/cerep/users/pages/downloads/Documents/Marketing/Pharmacology & ADME/Application notes/2011 Phospholipidosis.pdf); ChOX database, accessed 2011.09.05. Collected sets were carefully curated. Duplicates were removed. The data was form variety of species (rats, dogs, monkeys) and as well as humans.',234,'The set is relatively diverse. The molecular weight varies from 33 up to 1297 with mean 349.9 and standard deviation 162.6. Negatives are more diverse then positives.','Removal of counter ions. Removal of duplicates. Generation of 3D conformations',50.90,0.0000,0.0000,0.0000,'N/A','Name or ID as used in the source plus the source name e.g. originalID_nameOfTheSet.','N/A','Consensus kNN','80 diverse molecular descriptors were calculated for 3D structures and used to build a consensus model. It comprises of 10 kNN models (euclidean distance, no weighting) built for maximal k=5 optimized by LOO CV and 12 variables  selected by Simulated Annealing algorithm driven by optimization of 10-fold CV Kappa statistic.','The final model was developed using the whole set.\r\nThe correctness and performance of modeling methodology was confirmed by the following procedure: (1) The set was randomly divided into training and validation sets (in proportion 4/1). The class distribution was kept uniform. (2) Full modeling procedure was carried for all training sets. (3) Models\' performances were assessed using validation sets. The above procedure was repeated 5 times and the average value of accuracy and other statistics were calculated.\r\nThe external validation goodness of fit statistics in this document are the average values (rounded to integers) calculated for validation sets generated in 5 random training/validation splits.','The total number of 80 descriptors was used. AdrianaCode descriptors (Global molecular descriptors; Topological descriptors; Shape descriptors; Functional group counts)\r\nMoses based descriptors (Net charge at pH 4; Simple Amphiphilic Moment; Charge Separation; Critical Micelle Concentration [reimplementation of the published method by Mozrzymas et al: doi:10.2174/138620710790218195]) DIPL specific functional group counts (including Cyclic aliphatic amines and Amines attached to aromatic rings)\r\n','Simulated Annealing algorithm was used for the variable selection. The algorithm was implemented according to Zheng and Tropsha doi:10.1021/ci980033m','The model is meant to be applicable to any drug like or small organic compounds. Within 95% CI it predicts correctly 75-88% of true positive and 74-88% of true negative compounds.','The Domain of Applicability was defined as described in Sushko et al. doi:10.1021/ci100253r The model uses STD-PROB measure. It can be interpreted as a probability that compound should be classified into the opposite class. The performance of STD-PROB measure was confirmed by the Ascertained Optimality Excess Criterion as proposed in Dragos et al. doi:10.1021/ci9000579',111,110,5,8,0.0000,0.0000,'N/A','External validation was omitted due to the extensive internal validation approach. The external validation goodness of fit statistics in this document are the average values (rounded to integers) calculated for validation sets generated in 5 random training/validation splits.',0,21,18,4,5,0.0000,0.0000,'N/A; 10-fold CV gave TP: 97, TN: 93, FP: 22, FN: 22',0.0000,0.0000,'Analysis of the most important variables gives some insights in to mechanistic basis of the model. Among the most important variables are several related to specific moieties typical to phospholipidosis positive compounds, namely nitrogen connected directly or indirectly to aromatic systems. Variables pertaining to hydrogen bonds potential are also among the most important ones, as well as, LogP, and shape properties. ','N/A','N/A','N/A','MN','Multiple','N/A','boolean','negative, positive','Positive compounds: induction of phospholipidosis is highly possible. Negative compounds: induction of phospholipidosis is unlikely.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/1\r\n                             ','Not verified','na',NULL),(58,'2012-05-25 00:00:00','Daan Geerke','d.p.geerke@vu.nl','L. Capoferri et al., manuscript in preparation','LIE prediction of binding affinity to CYP450 1A2','CYP 1A2 Affinity 1 VUA','1.0','CYP 1A2, binding affinity, free energies of binding, IC50s','CYP 2C9 Affinity 1 VUA, CYP 2D6 Affinity 1 VUA, CYP 2D6 Affinity 2 VUA, MAPK1 Affinity 1 VUA','CYP 1A2 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol)','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5, with [S] = K_m), were taken from the Marketed Drugs (MTB04) and World Drug Index Drugs (METB05) CYP inhibition datasets supplied by Bayer (see eTOX intranet), from \r\n[Cohen et al. \"In vitro drug interactions of cytochrome P450: an evaluation of fluorogenic to conventional substrates.\" Drug Metab. Dispos. 2003, 31, 1005-1015.] DOI: 10.1124/dmd.31.8.1005 ,\r\nand from\r\n[Korhonen et al. \"Predictive three-dimensional quantitative structure-activity relationship of Cytochrome P450 1A2 inhibitors.\" J. Med. Chem. 2005, 48, 3808-3815.] DOI: 10.1021/jm0489713].',35,'A structurally heterogeneous set of 1A2 inhibitors/substrates (in terms of size, number of rings, and presence of number of different functional groups).','See \"Training set source\". IC50 values were converted into binding constants and free energies using the Cheng-Prusoff equation.',NULL,-42.5900,-10.7900,-25.5600,'yes','IDs according to original reference (L. Capoferri et al)','Only for compounds for which IC50s were taken from [Korhonen et al] (ID=27,28,30,31,32,33,34,35,36,38,39,40,41,42,43,50,51,52,53,54,55,56,57,58,59,60,61,62,64,65,67,68,70,72,74), the data source explicitly reports compound purities (>95% in those cases)','LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using linear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverables D3.5 and D3.13).','In total, 57 compounds (training+test set) were selected from the 73 compounds for which CYP 1A2 inhibition was quantitatively determined in the data sources reported under \"Training set source\". The 57 compounds were selected (automatically, based on predicted binding modes) such that only non-heme-coordinating binders were included in training and test sets. Based on visual inspection, the 57 compounds were divided in three main structural classes of compounds. 35 compounds were randomly selected as training compounds, evenly distributed over the three subclasses. For the remaining 22 compounds, external validation data (\"External val Q2\" and \"External val SDEP\") are reported based on the results for the 9 test compounds which fulfilled the applicability domain assessment (see \"AD assessment\").','DVele and DVvdw. Using LIE, binding free energies (affinities) are calculated from the average interaction energies (DVele and DVvdw) of the compound with its surroundings as obtained from MD simulations (see also deliverables D3.5 and D3.13).','N/A (cf. \"Descriptors definition\")','Diverse set of 1A2 binders was included in the training and test set. Compounds vary in their molecular weight, the number of rings, and the presence of functional groups.','Three or more AD criteria should be fulfilled, which are determined from (1) a Mahalanobis-distance analysis on measured simulation properties, (2) ligand-fingerprint similarities, (3) and (4) two residue-based protein-ligand interaction patterns in simulation (see D3.13), and (5) spread in experimental data.',NULL,NULL,NULL,NULL,0.5800,4.5800,'N/A','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5, with [S] = K_m), were taken from \r\n[Korhonen et al. \"Predictive three-dimensional quantitative structure-activity relationship of Cytochrome P450 1A2 inhibitors.\" J. Med. Chem. 2005, 48, 3808-3815.] DOI: 10.1021/jm0489713]. See \"Test set selection\" for further details.',9,NULL,NULL,NULL,NULL,-0.2000,4.7900,NULL,0.5000,5.0000,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 1A2.','See \"Mechanistic basis of the model\". A manuscript describing full details of the model is currently in preparation. See also eTOX deliverable D3.13 for details on the Applicability Domain assessment, and see the reference given below for details on the automated generation of starting structures for the MD simulations:\r\nC. R. Vosmeer, R. Pool, M. F. van Stee, L. Perić-Hassler, N. P. E. Vermeulen and D. P. Geerke. \"Towards Automated Binding Affinity Prediction Using an Iterative Linear Interaction Energy Approach.\" Int. J. Mol. Sci. 2014, 15, 798-816. DOI:10.3390/ijms15010798','LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034',NULL,'VUA','human','http://etox.chem.vu.nl/etoxws/','float','(-inf,inf)','Binding free energies (affinities) DG can be translated into dissociation or inhibition constants K by using DG = RT ln K. The more negative DG is for a compound, the stronger it binds to CYP 1A2.',NULL,'Verification ongoing','na',NULL),(59,'2012-05-25 00:00:00','Daan Geerke','d.p.geerke@vu.nl','E. Stjernschantz, C. Oostenbrink, Biophys. J. 2010, 98, 2662-2691. \r\nDOI: 10.1016/j.bpj.2010.02.034\r\nSee also eTOX Deliverable D3.5.','LIE prediction of binding affinity to CYP450 2C9','CYP 2C9 Affinity 1 VUA','1.0','CYP 2C9, binding affinity, free energies of binding, IC50s',NULL,'CYP 2C9 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5) from [R. Onderwater: “Molecular Toxicology of Thiourea-Containing Compounds”, PhD Thesis, 2005, VU University, Amsterdam, the Netherlands. DOI not available].',12,'A structurally homogeneous set of compounds (thiourea compounds: see Appendix of eTOX deliverable D3.5).','See \"Training set source\" and D3.5. IC50 values were taken from Onderwater and converted into binding affinities using the Cheng-Prusoff equation, and using identical values for the applied [S] (concentration of inhibited substrate) and K_m of the enzyme.',NULL,-43.2000,-26.3000,-33.0000,'yes',NULL,NULL,'LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using \r\nlinear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverable D3.5).',NULL,NULL,NULL,'Model is calibrated based on thiourea compounds. Further applicability is to be tested.','More than two AD criteria should be fulfilled, which are determined from a Mahalanobis-distance analysis on measured simulation properties, ligand-fingerprint similarities, and residue-based protein-ligand interaction patterns in simulation (D3.13).',NULL,NULL,NULL,NULL,0.6800,2.9000,NULL,'N/A',0,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 2C9.',NULL,'LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  C. R. Vosmeer, R. Pool, M. F. van Stee, L. Perić-Hassler, N. P. E. Vermeulen and D. P. Geerke. Int. J. Mol. Sci. 2014, 15, 798-816. DOI:10.3390/ijms15010798',NULL,'VUA','human',NULL,'float','(-inf,inf)','Binding free energies (affinities) DG can be translated into dissociation or inhibition constants K by using DG = RT ln K. The more negative DG is for a compound, the stronger it binds to CYP 2C9.',NULL,'Not verified','na',NULL),(60,'2012-05-25 00:00:00','Daan Geerke','d.p.geerke@vu.nl','L. Perić-Hassler, E. Stjernschantz, C. Oostenbrink and D. P. Geerke. \"CYP 2D6 Binding Affinity Predictions Using Multiple Ligand and Protein Conformations.\" Int. J. Mol. Sci. 2013, 14, 24514-24530. DOI:10.3390/ijms141224514','LIE prediction of binding affinity to CYP450 2D6','CYP 2D6 Affinity 1 VUA','1.0','CYP 2D6, binding affinity, free energies of binding, IC50s',NULL,'CYP 2D6 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5) from [R. Onderwater: “Molecular Toxicology of Thiourea-Containing Compounds”, PhD Thesis, 2005, VU University, Amsterdam, the Netherlands. DOI not available.]',10,'A structurally homogeneous set compounds (thiourea compounds: see Appendix of eTOX deliverable D3.5).','See \"Training set source\" and D3.5. IC50 values were taken from Onderwater and converted into binding affinities using the Cheng-Prusoff equation, and using identical values for the applied [S] (concentration of inhibited substrate) and K_m of the enzyme.',NULL,-35.7000,-23.3000,-28.0000,'yes',NULL,NULL,'LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using \r\nlinear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverable D3.5).',NULL,NULL,NULL,'Model is calibrated based on thiourea compounds. Further applicability is to be tested.','More than two AD criteria should be fulfilled, which are determined from a Mahalanobis-distance analysis on measured simulation properties, ligand-fingerprint similarities, and residue-based protein-ligand interaction patterns in simulation (D3.13).',NULL,NULL,NULL,NULL,0.7100,2.3000,NULL,'N/A',0,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 2D6.',NULL,'LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034; C. R. Vosmeer, R. Pool, M. F. van Stee, L. Perić-Hassler, N. P. E. Vermeulen and D. P. Geerke. Int. J. Mol. Sci. 2014, 15, 798-816. DOI:10.3390/ijms15010798',NULL,'VUA','human',NULL,'float','(-inf,inf)','Binding free energies (affinities) DG can be translated into dissociation or inhibition constants K by using DG = RT ln K. The more negative DG is for a compound, the stronger it binds to CYP 2D6.',NULL,'Not verified','na',NULL),(61,'2012-05-25 00:00:00','Daan Geerke','d.p.geerke@vu.nl','eTOX deliverable D3.9','LIE prediction of binding affinity to p38a MAP Kinase','MAPK1 Affinity 1 VUA','1.0','MAPK p38a, binding affinity, free energies of binding, IC50s',NULL,'MAPK p38a affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.9) from [L.L. Chang, K.L. Sidler, M.A. Cascieri, S. de Laszlo, G. Koch, B. Li, M. MacCoss, N. Mantlo, S. O\'Keefe, M. Pang, A. Rolando and W.K. Hagmann, Bioorg. Med. Chem. Lett. 2001, 11, 2549–2553] DOI: 10.1016/S0960-894X(01)00498-X',10,'A structurally homogeneous set of aromatic-substituted compounds (see Appendix of eTOX deliverable D3.9).','See \"Training set source\" and D3.9. IC50 values were converted into binding affinities using the Cheng-Prusoff equation, and the values given in Chang et al. for the applied [S] (concentration of inhibited substrate) and K_m of the enzyme.',NULL,-46.6000,-35.1000,-40.9000,'yes',NULL,NULL,'LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using \r\nlinear interaction energy (LIE) theory, to calculate the binding free energy (see also D3.5 and D3.9).',NULL,NULL,NULL,'Model is calibrated based on tri-aromatic substituted imidazole. Further applicability is to be tested.','More than two AD criteria should be fulfilled, which are determined from a Mahalanobis-distance analysis on measured simulation properties, ligand-fingerprint similarities, and residue-based protein-ligand interaction patterns in simulation (D3.13).',NULL,NULL,NULL,NULL,0.2300,3.7000,NULL,'N/A',0,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of p38a.',NULL,'LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034; C. R. Vosmeer, R. Pool, M. F. van Stee, L. Perić-Hassler, N. P. E. Vermeulen and D. P. Geerke. Int. J. Mol. Sci. 2014, 15, 798-816. DOI:10.3390/ijms15010798',NULL,'VUA','human',NULL,'float','(-inf,inf)','Binding free energies (affinities) DG can be translated into dissociation or inhibition constants K by using DG = RT ln K. The more negative DG is for a compound, the stronger it binds to p38a.',NULL,'Not verified','na',NULL),(68,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset was downloaded from chOX datasets website, published in \'Human UDP-glucuronosyltransferases: metabolism, expression, and disease\', Ann. Rev. Pharmacol. Toxicol. 2000, 40, p 581','Human glucuronosyltransferase substrate prediction model','UGT substrate 1 LMD','1','UGT glucuronosyltransferase \"phase II  metabolism\" substrate prediction','Pharmacogenetics. 2002 Nov;12(8):635-45','This model does not predict toxicity endpoints, but if a compound could be a substrate of human glucuronosyltransferase enzyme activity (without specifying a particular isoform, see training set source).','\"+1\" for a compound predicted to be UGT substrate, \"-1\" for those predicted to not be substrate','Dataset was downloaded from chOX datasets website, published in \'Human UDP-glucuronosyltransferases: metabolism, expression, and disease\', Ann. Rev. Pharmacol. Toxicol. 2000, 40, p 581',378,'For 214 compounds in the dataset some UGT isoform has a >0 specific enzyme activity (thus are substrates), while for 164 the specific activity is = 0 (and aren\'t metabolized by UGT).','Compounds in the original dataset were classified as substrates (\"+1\" category) if their specific activity value was > 0 (pm(min(mg) or non-substrates (\"-1\" category) is their specific activity value was 0. The original dataset included substrates and non-substrates for human UDP-glucuronosyltransferase isoforms 1-1, 1-6, 1-7, 1-8, 1-9, 1-10, 1A4, 2A1, 2B4, 2B7, 2B10, 2B15 and  2B17.',56.60,-1.0000,1.0000,0.1300,'No','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in 2 categories using Volsurf+ descriptors','RBF SVM categorization based on all 128 Volsurf+ descriptors.','A set of 22 randomly chosen compounds from the original dataset were assigned to the test set, and were never used in the model building step.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',210,139,17,3,NULL,NULL,NULL,'A set of 22 randomly chosen compounds from the original dataset were assigned to the test set, and were never used in the model building step. One of these compound could not be predicted, thus the effective size of the external test set was 21 compounds.',21,9,6,3,3,NULL,NULL,NULL,NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or +1','A \"+1\" outcome is issued for a compound predicted to be UGT substrate, or \"-1\" for those predicted to not be substrate','/ADME/Metabolism/Phase II/UGT Substrate/1','Verified','na',NULL),(70,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','As described in Deliverable D3.12, this dataset is the first human PK ChOX dataset compiled in ChOX database and originally published in \r\n\r\nLombardo, F., Obach, R. S., Dicapua, F. M., Bakken, G. a, Lu, J., Potter, D. M., Gao, F.,\r\net al. “A hybrid mixture discriminant analysis-random forest computational model for the\r\nprediction of volume of distribution of drugs in human” Journal of medicinal chemistry\r\n(2006) 49(7): 2262-7. doi:10.1021/jm050200r\r\n','Fraction unbound','Fraction unbound 1 LMD','1','Fu Funbound ADME','PLoS One. 2013 Oct 7;8(10):e74758. doi: 10.1371/journal.pone.0074758','This model does not predict toxicity endpoints, but what fraction of a compound is not bound to proteins (usually albumin) in blood plasma\r\n','Fraction of unbound compound, a number between 0 and 1 (but PLS might give values outside this range)','As described in Deliverable D3.12, this dataset is the first human PK ChOX dataset compiled in ChOX database and originally published in \r\n\r\nLombardo, F., Obach, R. S., Dicapua, F. M., Bakken, G. a, Lu, J., Potter, D. M., Gao, F.,\r\net al. “A hybrid mixture discriminant analysis-random forest computational model for the\r\nprediction of volume of distribution of drugs in human” Journal of medicinal chemistry\r\n(2006) 49(7): 2262-7. doi:10.1021/jm050200r\r\n',522,'As described in Deliverable D3.12, this dataset is the first human PK ChOX dataset compiled in ChOX database and originally published in \r\n\r\nLombardo, F., Obach, R. S., Dicapua, F. M., Bakken, G. a, Lu, J., Potter, D. M., Gao, F.,\r\net al. “A hybrid mixture discriminant analysis-random forest computational model for the\r\nprediction of volume of distribution of drugs in human” Journal of medicinal chemistry\r\n(2006) 49(7): 2262-7. doi:10.1021/jm050200r\r\n','PCA analysis and outlier removal by visual inspection',0.00,0.0000,1.0000,0.3780,'Skewness not assessed.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','Partial Least Squares (PLS)','Volsurf+ descriptors-based partial least squares model','Randomly selected 32 compounds from the original dataset, and they were never used in the model building step.','Descriptors calculated with Volsurf+ version 1.7l Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,0.5500,NULL,NULL,'32 compounds where randomly chosen from the initial dataset and assigned to an external set never used in model building step.',32,NULL,NULL,NULL,NULL,0.4300,0.2700,NULL,0.4650,0.2510,NULL,NULL,NULL,NULL,'LMD','human','http://etoxsys.eu/current','float','(-inf,inf)','Fraction of compound not bound to blood plasma proteins. Even though this is theoretically a number between 0 and 1, PLS might give values outside this range. A prediction < 0 should be considered equal to 0, and a prediction > 1 should be considered equal to 1.','/ADME/Distribution/Plasma Protein Binding/Fraction Unbound/1','Verified','dimensionless (ratio, fraction not bound/total)',NULL),(71,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset downloaded from eTOX web site, publised on \r\nR. Scott Obach, Franco Lombardo and Nigel J. Waters “Trend Analysis of a Database of\r\nIntravenous Pharmacokinetic Parameters in Humans for 670 Drug Compounds” Drug\r\nMetab. Dispos. (2008) 36(7):1385-1405','Logarithm of Volume of Distribution at steady state','Volume of distribution 1 LMD','1','Log(VDss) ADME','J Pharm Pharm Sci. 2015 Jul-Sep;18(3):515-27','This model does not predict toxicity endpoints, but the logarithm of volume of distribution of a compound in steady state\r\n','Logarithm of Volume of Distribution at steady state','As described in Deliverable D3.12, this dataset is the first human PK ChOX dataset compiled in ChOX database and originally published in \r\n\r\nR. Scott Obach, Franco Lombardo and Nigel J. Waters “Trend Analysis of a Database of\r\nIntravenous Pharmacokinetic Parameters in Humans for 670 Drug Compounds” Drug\r\nMetab. Dispos. (2008) 36(7):1385-1405',630,'Compounds are known drugs.','Principal Component Analysis was used to exclude outliers.',0.00,-1.4600,2.8500,0.0063,'Skewness not assessed.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','Partial Least Squares (PLS)','Volsurf+ descriptors-based partial least squares model','We selected those compounds (not already included in human ChOX PK first dataset) in the human ChOX PK second dataset as the test set.','Descriptors calculated with Volsurf+ version 1.7l Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,0.4920,NULL,NULL,'A set of 37 compounds randomly chosen from the dataset, never used in the model building step.',37,NULL,NULL,NULL,NULL,0.5600,0.3900,NULL,0.4400,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','(-inf,inf)','The prediction reported is the logarithm (base 10) of volume of distribution at steady state for the given compound.','/ADME/Distribution/Volume of Distribution/Volume of Distribution/1','Verified','L (log scale of Volume of Distribution)',NULL),(72,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset CACO-01 accessible from eTOX web side, published on J Chem Inf Comput Sci (2004) 44; 1585','CACO2 Logarithm of apparent permeability (LogPapp)','Gastro intestinal 1 LMD','1','CACO2 Log(Papp) ADME','Mol Pharm. 2015 May 4;12(5):1456-66. doi: 10.1021/mp500783g. Epub 2015 Apr 23','This model does not predict toxicity endpoints, but the logarithm of CACO2 apparent permeability of a compound in steady state\r\n','CACO2 Logarithm of apparent permeability (LogPapp)','As described in Deliverable D3.12, this dataset is the CACO-01 dataset of eTOX datasets intranet webpage:\r\n\r\nhttp://cadd.imim.es/etox-intranet/resources/public-data\r\n',73,'J Chem Inf Comput Sci (2004) 44; 1585','Principal Component Analysis was used to exclude from model those compounds lying outside the 95% ellipsoid, using Volsurf+ PCA implementation.',NULL,-6.9600,-4.2800,-5.1400,'Skewness not assessed.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','Partial Least Squares (PLS)','Volsurf+ descriptors-based partial least squares model','4 compounds of this dataset not included in the training set were used as the test set.','Descriptors calculated with Volsurf+ version 1.7l Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,0.6080,NULL,NULL,'A set of 4 compounds not present in the training set.',4,NULL,NULL,NULL,NULL,0.7900,0.3900,NULL,0.4410,0.5630,'N/A','N/A','N/A',NULL,'LMD','human','http://etoxsys.eu/current','float','(-inf,inf)','Prediction of the logarithm (base 10) of apparent permeability (LogPapp) for CACO2 cells.','/ADME/Absorption/Gastro Intestinal/Gastro Intestinal/1','Verified','cm/sec (log of apparent permeability coefficient, Papp)',NULL),(73,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset downloaded from eTOX web site, published on J Mol Graph Model (2010) 28; 899','Logarithm of brain/blood partition coefficient (LogBB)','Blood Brain Barrier 1 LMD','1','BBB Log(BB) ADME','J Pharm Pharm Sci. 2013;16(4):622-47.','This model does not predict toxicity endpoints, but the logarithm of blood brain partition coefficient (LogBB)\r\n','Logarithm of brain/blood partition coefficient (LogBB)','As described in Deliverable D3.12, this dataset is the BBB-02 dataset of eTOX datasets intranet webpage:\r\n\r\nhttp://cadd.imim.es/etox-intranet/resources/public-data\r\n',287,'As described in Deliverable D3.12, this dataset is the BBB-02 dataset of eTOX datasets intranet webpage:\r\n\r\nhttp://cadd.imim.es/etox-intranet/resources/public-data\r\n','Principal Component Analysis was used to exclude outliers.',0.00,-2.1500,1.6400,-0.0287,'Skewness not assessed.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','Partial Least Squares (PLS)','Volsurf+ descriptors-based partial least squares model','20 compounds of this dataset were randomly selected as the test set, never used in the building model step.','Descriptors calculated with Volsurf+ version 1.7l Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,0.4300,NULL,NULL,'A set of compounds not present in the training set.',20,NULL,NULL,NULL,NULL,0.3900,0.6100,NULL,0.3600,NULL,'N/A','N/A','N/A',NULL,'LMD','human','http://etoxsys.eu/current','float','(-inf,inf)','Prediction of a compound logarithm (base 10) of brain/blood partition coefficient (LogBB)','/ADME/Absorption/Blood Brain Barrier/Blood Brain Barrier/1','Verified','dimensionless (log brain/blood partition coefficient)',NULL),(74,'2014-09-19 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset AQU01 obtained from Etox web site, published on J Chem Inf Comput Sci (2004) 44; 1000','Logarithm of water solubility','LogSolubility 1 LMD','1','Log(Sol) ADME','Eur J Med Chem. 2008 Mar;43(3):501-12. Epub 2007 May 6.','This model does not predict toxicity endpoints, but the logarithm of water solubility (in molar concentration units)\r\n','Logarithm of water solubility in molar concentration units','As described in Deliverable D3.12, this dataset is the AQU 01 dataset of eTOX datasets intranet webpage:\r\n\r\nhttp://cadd.imim.es/etox-intranet/resources/public-data\r\n',1084,'Experimental values compiled from public literature.','Principal Component Analysis was used to exclude outliers.',0.00,-11.6000,1.5800,-3.0400,'Skewness not assessed.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','Partial Least Squares (PLS)','Volsurf+ descriptors-based partial least squares model','54 randomly selected compounds of this dataset were used as the test set, never used in the model building step.','Descriptors calculated with Volsurf+ version 1.7l. Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,0.8800,NULL,'Not assessed','A set of compounds not present in the training set.',54,NULL,NULL,NULL,NULL,0.9500,0.5300,NULL,0.8700,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','(-inf,inf)','Predicted logarithm in base 10 of water solubility in molar concentration units for the given compound.','/Other/Physicochemical Properties/LogSolubility/1','Verified','L (solubility in log scale)',NULL),(75,'2012-06-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset downloaded from ChOX database, published on R. Scott Obach, Franco Lombardo and Nigel J. Waters “Trend Analysis of a Database of\r\nIntravenous Pharmacokinetic Parameters in Humans for 670 Drug Compounds” Drug\r\nMetab. Dispos. (2008) 36(7):1385-1405','Fraction unbound qualitative classification model','Fraction unbound 2 LMD','1','Fu qualitative','PLoS One. 2013 Oct 7;8(10):e74758. doi: 10.1371/journal.pone.0074758','This model does not predict toxicity endpoints, but classifies the compound in either 1, 2 or 3 categories corresponding to low, medium or high fraction unbound.\r\n','Fraction unbound qualitative classification model','As described in Deliverable D3.12, this dataset is the human PK ChOX first dataset first published in:\r\n\r\nR. Scott Obach, Franco Lombardo and Nigel J. Waters “Trend Analysis of a Database of\r\nIntravenous Pharmacokinetic Parameters in Humans for 670 Drug Compounds” Drug\r\nMetab. Dispos. (2008) 36(7):1385-1405\r\n\r\n',528,'Compounds compiled from published data published in Drug\r\nMetab. Dispos. (2008) 36(7):1385-1405 were binned in three categories: 1, for compounds with fraction unbound between 0 and 0.1; 2 for compounds with Fu between 0.1 and 0.6 and 3 for compounds with Fu between 0.6 and 1.0. ','Principal Component Analysis was used to exclude outliers',0.00,1.0000,3.0000,1.9800,'No','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','classification support vector machine (C-SVM)','eTOX deliverable D3.12 describes the binning scheme used. Model classifies compounds in three different categories 1 (low), 2 (medium) and 3 (high) using a Volsurf+ descriptors-based support vector machine model (libSVM as implemented in R package). As three different categories are used, the goodness of fit measures true positives (TP), true negatives (TN) false positives (FP) and false negatives (FN) are replaced with the error rate (number of misclassifications/total compounds).','26 compounds from the original dataset were randomly assigned to the test set and never used in the model building step.','Descriptors calculated with Volsurf+ version 1.7l. Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',0,0,0,0,NULL,NULL,'Internal val. error rate=0.125','A set of 26 randomly chosen compounds from the original dataset not used while building the model.',26,0,0,0,0,NULL,NULL,'Error rate=0.307',NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','1, 2 or 3','Fraction unbound qualitative classification model. The 1 (\"low\") category includes Fu values between 0 and 0.1, the 2 (\"medium\") values between 0.1 and 0.6, and the 3 (\"high\") values between 0.6 and 1.0','/ADME/Distribution/Plasma Protein Binding/Fraction Unbound/2','Verified','na',NULL),(76,'2014-09-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset BBB02 downloaded from eTOX web site, published on J Mol Graph Model (2010) 28; 899','Logarithm of brain blood partition coefficient qualitative classification model','Blood Brain Barrier 2 LMD','1','BBB Log(BB) ADME','SAR QSAR Environ Res. 2013 Jan;24(1):61-74. doi: 10.1080/1062936X.2012.729224. Epub 2012 Oct 24','This model does not predict toxicity endpoints, but the logarithm of brain blood partition coefficient category: category \"-1\" for compounds with Log(BB) between-2.15 and 0 and \"+1\" for compounds with logBB between 0 and 1.7\r\n','\"-1\" for compounds with Log(BB) between-2.15 and 0 and \"+1\" if logBB between 0 and 1.7  ','As described in Deliverable D3.12, this dataset is eTOX dataset BBB-02\r\n\r\nhttp://cadd.imim.es/etox-intranet/resources/public-data \r\n',290,'Experimental values as retrieved from published literature were binned into 2 categories: -1 for compounds with Log(BB) between -2.15 and 0, and +1 for compounds with Log(BB) > 0.','Principal Component Analysis was used to exclude outliers.',47.00,-1.0000,1.0000,NULL,'No','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','classification support vector machine (C-SVM)','eTOX deliverable D3.12 describes the binning scheme used. Model classifies compounds in two different categories (low and high) using a Volsurf+ descriptors-based support vector machine model (libSVM implementation).','16 randomly selected compounds from the dataset were included in the test set, and never used in the model building step.','Descriptors calculated with Volsurf+ version 1.7l Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',123,142,12,13,NULL,NULL,NULL,'A set of compounds not present in the training set.',16,5,6,2,4,NULL,NULL,NULL,NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or +1','\"-1\" category for compounds with Log(BB) between -2.15 and 0 and \"+1\" for logBB between 0 and 1.7 ','/ADME/Absorption/Blood Brain Barrier/Blood Brain Barrier/2','Verified','na',NULL),(77,'2012-06-18 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset downloaded from eTOX web site, published on J Chem Inf Model (2010) 50; 1034','Logarithm of brain blood partition coefficient qualitative classification model','Blood Brain Barrier 3 LMD','1','BBB Log(BB) ADME','SAR QSAR Environ Res. 2013 Jan;24(1):61-74. doi: 10.1080/1062936X.2012.729224. Epub 2012 Oct 24','This model does not predict toxicity endpoints, but predicts whether a compound crosses or not the brain blood barrier.','\"-1\" for compounds that do not cross BBB, \"1\" for compounds that cross BBB','As described in Deliverable D3.12, this dataset is eTOX dataset BBB-03, published on J Chem Inf Model (2010) 50; 1034\r\n',445,'As described in Deliverable D3.12, this dataset is eTOX dataset BBB-03, published on J Chem Inf Model (2010) 50; 1034','Principal Component Analysis was used to exclude outliers. Original dataset did not include experimental values, just the qualitative classification: crosses blood brain barrier or not.',84.00,-1.0000,1.0000,NULL,'Yes','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','classification support vector machine (C-SVM)','eTOX deliverable D3.12 describes the binning scheme used. Model classifies compounds in two different categories (\"-1\" and \"1\", the former is for compounds that do not cross BBB, the later for compound that cross it) using a Volsurf+ descriptors-based support vector machine model (libSVM implementation).\r\n\r\n','35 compounds where randomly selected from the original dataset, assigned to the test set and never used in the model building step.','Descriptors calculated with Volsurf+ version 1.7l. Molecular Discovery Ltd. (UK)\r\nhttp://www.moldiscovery.com\r\n','All Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',240,48,1,1,NULL,NULL,NULL,'A set of 35 compounds not present in the training set, randomly chosen from the original dataset.',35,30,3,1,1,NULL,NULL,NULL,NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or +1','Predicted category \"-1\" for compounds that do not cross BBB, and  \"1\" for compounds that cross BBB','/ADME/Absorption/Blood Brain Barrier/Blood Brain Barrier/3','Verified','na',NULL),(80,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005) J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore for Nav 1.5 Inhibition','Nav 1.5 Inhibition 1 IL','1','Nav 1.5 , inhibition, pharmacophore',NULL,'Nav 1.5  Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Sodium channel protein type 5 subunit alpha is assigned a positive categorical value. A low likelihood for a molecule to inhibit Sodium channel protein type 5 subunit alpha is assigned a categorical value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','Molecules that inhibit Nav 1.5 were extracted from the chOX database.',25,'Inhibitors of Nav 1.5 were selected from the chOX database. Conformations were generated for each then clustered based on the 3D alignments of their pharmacophore features. 25 molecules from the clusters were selected based on the maximum chemical diversity.','The data was curated.  The most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments. 25 representative molecules for each of the clusters with the largest chemical diversity were selected for the training set.',100.00,NULL,NULL,NULL,NULL,'CHEMBL','NA','3D-Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The Test set included 38 inhibitors and 18 non-inhibitors taken from the chOX database.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',25,0,0,0,NULL,NULL,NULL,'The test set included 38 inhibitors and 18 non-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',56,25,16,2,13,NULL,NULL,'AUC 1,5,10,100 % = 1.00, 1.00, 1.00, 0.77',NULL,NULL,'3D Chemical Feature Ligand-Based Pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Compounds marked Positive are likely to inhibit Sodium channel protein type 5 subunit alpha. Compounds marked Negative have a low likelihood to inhibit Sodium channel protein type 5 subunit alpha.','/Toxicity/Target Safety Pharmacology/Nav 1.5 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(81,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for ADOR A1 Inhibition','ADOR A1 Inhibition 1 IL','1','ADOR A1, inhibition, pharmacophore, Adenosine',NULL,'ADOR A1 Inhibition:Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Adenosine receptor A1 is assigned a positive categorical value. A low likelihood for a molecule to inhibit Adenosine receptor A1 is assigned the categorical value-Negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','124 inhibitors of ADOR A1 were extracted from the chOX database. A subset of 25 were included in the training set.',25,'The training set included 124 molecules. Conformations were generated for each molecule and clustered based on the 3D alignments of their pharmacophore features. 25 molecules were selected from the clusters.','The data was curated.  124 active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) conformations with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). The molecules were clustered based on 3D-pharmacophore feature alignments. 25 representative molecules for each of the clusters with the largest number of molecules in the cluster and chemical diversity were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 88 inhibitors and 36 non-inhibitors from the chOX database','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',25,0,0,0,NULL,NULL,NULL,'The test set included 88 inhibitors and 36 non-inhibitors from the chOX database. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.',124,26,14,83,0,NULL,NULL,'AUC 1,5,10,100 % = 1.00,1.00,0.65,0.57',NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood inhibit Adenosine receptor A1. Molecules with the result of Negative have a low likelihood inhibit Adenosine receptor A1. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/ADOR A1 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(82,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for ADOR A2a Inhibition','ADOR A2a Inhibition 1 IL','1','ADOR A2a, inhibition, pharmacophore, Adenosine A2A',NULL,'ADOR A2a Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Adenosine receptor A2a is assigned a positive categorical value. A low likelihood for a molecule to inhibit Adenosine receptor A2a is assigned a categorical value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','123 active molecules for ADOR A2A were extracted from the chOX database. A subset of 51 inhibitors were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments. \r\n',51,'The training set included 123 active ADOR A2A inhibitors. A subset of 51 inhibitors were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments using LigandScout.','The data was curated.  123 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments. 51 representative molecules for each of the clusters with the largest number of molecules in the cluster and chemical diversity were selected for the training set.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D-Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 109 inhibitors and 14 non-inhibitors taken from the chOX database that were not included in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',45,0,0,6,NULL,NULL,NULL,'The test set included 109 inhibitors and 14 non-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',123,26,14,0,83,NULL,NULL,'AUC 1,5,10,100 % = 1.00,1.00,1.00,0.62',NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore modelling ','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules given a Positive categorical value have a likelihood to inhibit Adenosine receptor A2a. Molecules with the result of Negative have a low likelihood to inhibit Adenosine receptor A2a. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/ADOR A2a Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(83,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Ligand-Based Pharmacophore Model for ADOR A2b Inhibition','ADOR A2b Inhibition 1 IL','1','ADOR A2b, inhibition, pharmacophore, Adenosine A2b',NULL,'ADOR A2b Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit Adenosine receptor A2b is assigned a positive categorical value. A low likelihood for a molecule to inhibit Adenosine receptor A2b is assigned the categorical value-Negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','50 inhibitors of ADOR A2b extracted from the chOX database.',50,'The training set included 50 molecules. Conformations were generated for each molecule. A ligand-based pharmacophore model was generated after multiple alignment experiments using LigandScout.','The data was curated. 50 active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 181 inhibitors and 20 non-inhibitors from the chOX database that were not included in the training set. ','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',8,0,0,2,NULL,NULL,NULL,'The test set included 181 inhibitors and 20 non-inhibitors from the chOX database. The molecules were converted from 2D smiles format to 3D conformations in SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',201,151,10,10,30,NULL,NULL,'AUC 1,5,10,100 % =1.00,1.00,1.00,0.70',NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Query compounds given a Positive result are likely to inhibit Adenosine receptor A2b. Those with a Negative result have low likelihood to inhibit Adenosine A2b. Unknown indicates that a prediction could not be determined.\r\n','/Toxicity/Target Safety Pharmacology/ADOR A2b Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(84,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D-Ligand-Based Pharmacophore Model for ADOR A3 Inhibition','ADOR A3 Inhibition 1 IL','1','ADOR A3, inhibition, pharmacophore, Adenosine A3',NULL,'ADOR A3 Inhibition: Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a moleculeto inhibit Adenosine receptor A3 is assigned a Positive categorical value. A low likelihood for a molecule to inhibit Adenosine receptor A3 is assigned the categorical value-Negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','125 ADOR A3 inhibitors extracted from the chOX database.',125,'The training set included 125 molecules from the chOX database that were used to create a 3D Ligand-Based Pharmacophore model.','The data was curated. 125 active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type). 3D pharmacophore models were created based on multiple 3D- pharmacophore feature alignments using LigandScout.',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand-Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.',' The test set included 196 inhibitors and 14 non-inhibitors taken from the chOX database that were not included in the training set.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',138,NULL,NULL,58,NULL,NULL,NULL,'The test set included 196 inhibitors and 14 non-inhibitors from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.',210,138,12,2,58,NULL,NULL,'AUC 1,5,10,100 % = 1.00,1.00,1.00,0.83',NULL,NULL,'3D Chemical Feature Ligand-Based Pharmacophore Model',NULL,NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit Adenosine receptor A3. Molecules with the result of Negative have a low likelihood to inhibit Adenosine receptor A3. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/ADOR A3 Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(85,'2012-07-05 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03; see also: Training set source','Human total clearance','Total Clearance 1 MN','0.4','clearance','Obach et al. doi:10.1124/dmd.108.020479, Total Clearance 2 MN','Human total clearance (a sum of hepatic, renal, biliary and other types of clearance).','Values of the total clearance was expressed in log10 units of the original ml/min/kg units.','Data set downloaded form ChOX DB - accessed 2012.04.24 Orginal publication: Obach et al. doi:10.1124/dmd.108.020479',532,'published data','Compounds for which molecular descriptors cannot be calculated were discarded (boron, silicon, heavy metals containing compounds).',0.00,-2.4320,3.0290,0.5360,'no','ChOX DB ID','Drug-like molecules','Consensus kNN','Modeling set was randomly split 5 times in proportion 4/1 to training and validation sets. For each split, 10 different GA/kNN models were built for training sets using as an optimization criterion the GMFE statistic of validation set. The final model was built for the entire modeling set as the average consensus of 50 kNN models resulted form 10 models built for 5 different splits. All kNN models (city block distance with inverse weighting) were built for maximal k=5 optimized by LOO CV.','The initial dataset (666 compounds) was systematically split in to modelling set (532) used for actual model development and external set (134) used only for validation purposes. The splits were done in proportion 4/1 using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666','In total 71 molecular descriptors were calculated using Moses library: (1) Functional group counts – 16 different structural features (2) Charge type class – 4 classes of charge-type including  acids, bases, zwitterions and neutral compounds plus additional flag for quaternary amines and pyridinium ions (3) Global molecular descriptors – 28 descriptors (4) Topological descriptors – 6 descriptors (5) Shape descriptors – 12 descriptors (6) Net charge at pH 4, 7 and 10 – 3 pKa related descriptors (7) Metabolism related – 1 descriptor (metabolic stability)','Genetic Algorithm (GA) was used for the variable selection.','Predicted values have an approximate 95% CI of 0.87 log10 units. Reliability of prediction is calculated as standard deviation of consensus prediction. The model is meant to be applicable to drug like organic compounds. All compounds containing boron, silicon or heavy metal atoms (complex compounds) and compounds with the longest chain of non-hydrogen atoms shorter then 3 are excluded from the Applicability Domain a priori.','The domain of applicability was defined using the HDR method in PCA transformed space for PC2 and PC3 components. The HDR is the probability density distribution-based method. It measures the fraction of overall probability density distribution that spans object in question. See also Netzeva et al. (http://www.frame.org.uk/atla_article.php?art_id=116&abstract=true accessed 2013.11.14).',0,0,0,0,0.9999,0.0115,'N/A','External set was selected in a systematic way using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666',134,0,0,0,0,0.4354,0.4444,'GMFE = 2.0250, Fraction FE > 2 = 36%',0.3319,0.5398,'Insights on mechanistic basis of the model can be found in the eTOX deliverable D3.12','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03','N/A','N/A','MN','human','N/A','float','(-inf,inf)','Predicted values of total clearance are expressed in log10 units of the original ml/min/kg units.','/ADME/Excretion/Clearance/Total Clearance/1','Not verified','na',NULL),(86,'2012-07-05 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03; see also: Training set source','Rat total clearance','Total Clearance 2 MN','0.4','clearance','Total Clearance 1 MN','Rat total clearance - a collection of total clearance values from 1584 different assays.','Values of the total clearance was expressed in log10 units of the original ml/min/kg units.','Data set downloaded form ChOX DB: Rat CL in vivo dataset - accessed 2012.03.28',2848,'collection of different published data','Records for which dependent variable was not defined as being \"equal\" were discarded. Duplicated compounds were merged and the activity was averaged. Compounds for which molecular descriptors cannot be calculated were discarded (boron, silicon, heavy metals containing compounds).',0.00,-2.5290,2.0000,1.2370,'yes, negative','ChOX DB ID','Drug-like molecules','Consensus kNN','Modeling set was randomly split 10 times in proportion 4/1 to training and validation sets. For each split, 10 different GA/kNN models were built for training sets using as an optimization criterion the GMFE statistic of validation set. The final model was built for the entire modeling set as the average consensus of 100 kNN models resulted form 10 models built for 10 different splits. All kNN models (city block distance with inverse weighting) were built for maximal k=5 optimized by LOO CV.','The initial dataset (3561 compounds) was systematically split in to modelling set (2848) used for actual model development and external set (713) used only for validation purposes. The splits were done in proportion 4/1 using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666','In total 71 molecular descriptors were calculated using Moses library: (1) Functional group counts – 16 different structural features (2) Charge type class – 4 classes of charge-type including  acids, bases, zwitterions and neutral compounds plus additional flag for quaternary amines and pyridinium ions (3) Global molecular descriptors – 28 descriptors (4) Topological descriptors – 6 descriptors (5) Shape descriptors – 12 descriptors (6) Net charge at pH 4, 7 and 10 – 3 pKa related descriptors (7) Metabolism related – 1 descriptor (metabolic stability)','Genetic Algorithm (GA) was used for the variable selection.','Predicted values have an approximate 95% CI of 0.72 log10 units. Reliability of prediction is calculated as standard deviation of consensus prediction. The model is meant to be applicable to drug like organic compounds. All compounds containing boron, silicon or heavy metal atoms (complex compounds) and compounds with longest chain of non-hydrogen atoms shorter then 3 are excluded from the Applicability Domain a priori.','The domain of applicability was defined using the QVAR method defined per analogiam to SQSVAR measure described by Dragos et al doi:10.1021/ci9000579 as a variance of consensus prediction divided by RMST of the model.',0,0,0,0,0.9989,0.0257,'N/A','External set was selected in a systematic way using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666',713,0,0,0,0,0.5088,0.3648,'GMFE = 1.8642, Fraction FE > 2 = 34%',0.3988,0.4178,'Insights on mechanistic basis of the model can be found in the eTOX deliverable D3.12','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03','N/A','N/A','MN','rat','N/A','float','(-inf,inf)','Predicted values of total clearance are expressed in log10 units of the original ml/min/kg units.','/ADME/Excretion/Clearance/Total Clearance/2','Not verified','na',NULL),(87,'2012-07-05 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03; see also: Training set source','Human volume of distribution','Volume of Distribution 1 MN','0.4','volume of distribution','Obach et al. doi:10.1124/dmd.108.020479, Volume of Distribution 2 MN','Volume of distribution at steady-state.','Values of the volume of distribution was expressed in log10 units of the original l/kg units.','Data set downloaded form ChOX DB - accessed 2012.04.24 Orginal publication: Obach et al. doi:10.1124/dmd.108.020479',532,'published data','Compounds for which molecular descriptors cannot be calculated were discarded (boron, silicon, heavy metals containing compounds).',0.00,-1.4560,2.1460,-0.0230,'yes, positive','ChOX DB ID','Drug-like molecules','Consensus kNN','Modeling set was randomly split 5 times in proportion 4/1 to training and validation sets. For each split, 10 different GA/kNN models were built for training sets using as an optimization criterion the GMFE statistic of validation set. The final model was built for the entire modeling set as the average consensus of 50 kNN models resulted form 10 models built for 5 different splits. All kNN models (city block distance with inverse weighting) were built for maximal k=5 optimized by LOO CV.','The initial dataset (666 compounds) was systematically split in to modelling set (532) used for actual model development and external set (134) used only for validation purposes. The splits were done in proportion 4/1 using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666','In total 71 molecular descriptors were calculated using Moses library: (1) Functional group counts – 16 different structural features (2) Charge type class – 4 classes of charge-type including  acids, bases, zwitterions and neutral compounds plus additional flag for quaternary amines and pyridinium ions (3) Global molecular descriptors – 28 descriptors (4) Topological descriptors – 6 descriptors (5) Shape descriptors – 12 descriptors (6) Net charge at pH 4, 7 and 10 – 3 pKa related descriptors (7) Metabolism related – 1 descriptor (metabolic stability)','Genetic Algorithm (GA) was used for the variable selection.','Predicted values have an approximate 95% CI of 0.75 log10 units. Reliability of prediction is calculated as standard deviation of consensus prediction. The model is meant to be applicable to drug like organic compounds. All compounds containing boron, silicon or heavy metal atoms (complex compounds) and compounds with the longest chain of non-hydrogen atoms shorter then 3 are excluded from the Applicability Domain a priori.','The domain of applicability was defined using the QVAR method defined per analogiam to SQSVAR measure described by Dragos et al doi:10.1021/ci9000579 as a variance of consensus prediction divided by RMST of the model.',0,0,0,0,0.9999,0.0098,'N/A','External set was selected in a systematic way using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666',134,0,0,0,0,0.6383,0.3809,'GMFE = 1.8090 Fraction FE > 2 = 30%',0.5453,0.4284,'Insights on mechanistic basis of the model can be found in the eTOX deliverable D3.12','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03','N/A','N/A','MN','human','N/A','float','(-inf,inf)','Predicted values of volume of distribution at steady state are expressed in log10 units of the original l/kg units.','/ADME/Distribution/Volume of Distribution/Volume of Distribution/1','Not verified','na',NULL),(88,'2012-07-05 00:00:00','Tomasz Magdziarz','magdziarz@molecular-networks.com','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03; see also: Training set source','Rat volume of distribution','Volume of Distribution 2 MN','0.4','volume of distribution','Volume of Distribution 1 MN','Rat volume of distribution at steady-state - a collection of volume of distribution values from 996 different assays.','Values of the volume of distribution was expressed in log10 units of the original l/kg units.','Data set downloaded form ChOX DB: Rat VD in vivo dataset - accessed 2012.07.02',1721,'collection of different published data','Records for which dependent variable was not defined as being \"equal\" were discarded. Duplicated compounds were merged and the activity was averaged. Compounds for which molecular descriptors cannot be calculated were discarded (boron, silicon, heavy metals containing compounds).',0.00,-3.2220,1.9930,0.3330,'no','ChOX DB ID','Drug-like molecules','Consensus kNN','Modeling set was randomly split 10 times in proportion 4/1 to training and validation sets. For each split, 10 different GA/kNN models were built for training sets using as an optimization criterion the GMFE statistic of validation set. The final model was built for the entire modeling set as the average consensus of 100 kNN models resulted form 10 models built for 10 different splits. All kNN models (city block distance with inverse weighting) were built for maximal k=5 optimized by LOO CV.','The initial dataset (2152 compounds) was systematically split in to modelling set (1721) used for actual model development and external set (431) used only for validation purposes. The splits were done in proportion 4/1 using Kennard-Stone CADEX algorithm doi:10.1080/00401706.1969.10490666','In total 71 molecular descriptors were calculated using Moses library: (1) Functional group counts – 16 different structural features (2) Charge type class – 4 classes of charge-type including  acids, bases, zwitterions and neutral compounds plus additional flag for quaternary amines and pyridinium ions (3) Global molecular descriptors – 28 descriptors (4) Topological descriptors – 6 descriptors (5) Shape descriptors – 12 descriptors (6) Net charge at pH 4, 7 and 10 – 3 pKa related descriptors (7) Metabolism related – 1 descriptor (metabolic stability)','Genetic Algorithm (GA) was used for the variable selection.','Predicted values have an approximate 95% CI of 0.61 log10 units. Reliability of prediction is calculated as standard deviation of consensus prediction. The model is meant to be applicable to drug like organic compounds. All compounds containing boron, silicon or heavy metal atoms (complex compounds) and compounds with the longest chain of non-hydrogen atoms shorter then 3 are excluded from the Applicability Domain a priori.','The domain of applicability was defined using the HDR method in PCA transformed space for PC2 and PC3 components. The HDR is the probability density distribution-based method. It measures the fraction of overall probability density distribution that spans object in question. See also Netzeva et al. (http://www.frame.org.uk/atla_article.php?art_id=116&abstract=true accessed 2013.11.14)',0,0,0,0,0.9997,0.0122,'N/A','External set was selected in a systematic way using Kennard-Stone CADEX algorithmdoi:10.1080/00401706.1969.10490666',431,0,0,0,0,0.7200,0.3118,'GMFE = 1.6845, Fraction FE > 2 = 26%',0.5480,0.4003,'Insights on mechanistic basis of the model can be found in the eTOX deliverable D3.12','eTOX deliverable D3.12 (http://lamia.imim.es/etox-intranet/wp-content/uploads/D3.12_In_silico_models_for_prediction_of_different_animal_in_vivo_ADME_parameter_v1.4_FINAL.pdf) - accessed 2014.09.03','N/A','N/A','MN','rat','N/A','float','(-inf,inf)','Predicted values of volume of distribution are expressed in log10 units of the original l/kg units.','/ADME/Distribution/Volume of Distribution/Volume of Distribution/2\r\n','Not verified','na',NULL),(92,'2013-11-11 00:00:00','Claire Ellison','c.m.ellison@ljmu.ac.uk','M. Hewitt, S.J. Enoch, J.C. Madden, K.R. Przybylak and M.T.D. Cronin. Crit. Rev. Toxicol. 2013. 43, 537-558.','Drug-Induced Liver Injury (DILI) Structural Alerts','DILI Human LJMU 1','1','DILI, Liver, Hepatotoxicity','None','Positive prediction indicates prossible drug-induced liver injury in humans.','Binary classification (positive or negative DILI)','Fourches et al. Chem. Res. Toxicol. 2010. 23, 171-183.',932,'Diverse set of pharmaceutical compounds, both in terms of structure and therapeutic class.','Original dataset comprised 951 compounds mined from literature. Additional curation involved removal of 19 duplicates. ',72.21,NULL,NULL,NULL,'N/A','Chemical name','N/A','Structural alert model','16 structural alerts were developed from 932 compounds available from Fourches et al via the formation of structurally related categories. These were coded in SMARTS strings and then implemented into a KNIME workflow.','N/A. See Supporting information.','N/A','N/A','Applicability domain is intrinsic','Defined by the presence of structural alerts',111,252,43,526,NULL,NULL,'N/A','N/A. See supporting infromation.',0,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,'Each of the 16 structural alerts were studied and mechanistic rational was proposed in all cases. See reference source.','Further information can be found in original paper: M. Hewitt, S. J. Enoch, J. C. Madden and M. T. D. Cronin, Crit. Rev. Toxicol. 2013, 43(7), 537-558, DOI: 10.3109/10408444.2013.811215','N/A','These structural alerts act as hazard identification rather than a true predictive model. For example, they do not acurately predict compounds as inactive. Therefore, no external validation statistics are supplied and \"performance\" is poorly represented by goodness-of-fit statistics..','LJMU','human','http://etox.ljmu.ac.uk/etoxwsapi/v2/dir','boolean','Positive or negative','Results are given as positive (Inducers) or negative (non-inducer) for Drug-Induced Liver Injury (DILI). Drugs which are flagged as hepatotoxic are likely to cause liver injury in humans. The negative outcome should not be taken as a negative prediction for hepatotoxicity; only that none of the 16 alerts are present in the query chemical.','/Toxicity/Organ Toxicity/Hepatotoxicity/DILI human/1 ','Verified','na',NULL),(91,'2012-02-01 00:00:00','Katharine Briggs','Katharine.briggs@lhasalimited.org','[1] Sanderson M & Earnshaw CG (1991). Computer Prediction of Possible Toxic Action from Chemical Structure; The DEREK System. Human and Experimental Toxicology 10, 261-273. doi: 10.1177/096032719101000405 \r\n[2] Judson PN, Marchant CA & Vessey JD (2003). Using Argumentation for Absolute Reasoning about the Potential Toxicity of Chemicals. Journal of Chemical Information and Computer Sciences 43, 1364-1370. doi: 10.1021/ci020272g','Derek Nexus - hepatotoxicity mammal','DILI Mammal 1 LL','Beta','DILI, drug induced liver injury, hepatotoxicity','DILI general 1 & LB pharmacophore model for DILI in human, non-rodent & rodent','Predicts the likelihood of a structure being hepatotoxic by comparing the query compound to multiple structural alerts (2D SARs)','Term to express the likelihood of a structure being toxic.','References are provided with each alert. The training set underpinning a given alert is proprietary, though generally based on publicly available data:\r\nGreene N, Fisk L, Naven RT, Note RR, Patel ML, Pelletier DJ. Developing structure-activity relationships for the prediction of hepatotoxicity. Chem Res Toxicol. 2010; 23 (7), 1215-22. doi: 10.1021/tx1000865. ',1262,'Large and diverse dataset of human and preclinical hepatotoxicity data for drugs and other chemicals.','A structure searchable database was manually created from the source data & supplemented with additional supporting data in the published literature. Zimmerman includes 1266 substances but there is no structure available for the antibiotic mixture gramicidin and processing methimazole, chlortetracycline & demeclocycline resulted in an error.',100.00,NULL,NULL,NULL,'Yes','CAS No','CAS No, name & structure','Expert system','Expert system based on multiple structure alerts (2D SARs). A structure searchable database was manually created from the source data & supplemented with additional supporting data in the published literature. Suitable classes for investigation were then identified by visual analysis and consideration of therapeutic endpoint. Where sufficient biological and mechanistic data was available then a structural alert was created for that class. Each structural alert was then tested and peer-reviewed before being implemented.','N/A','N/A','N/A','If a chemical matches an alert describing a structure-activity for hepatotoxicity it can be considered to be within the applicability domain. The applicability domain of each alert is defined by the alert developer on the basis of the training set data and expert judgement on the chemical and biological factors which affect the mechanism of action for each alert.','N/A',484,0,0,778,NULL,NULL,'N/A','Liver Toxicity Knowledgebase (LTKB) from FDA available at http://www.fda.gov/ScienceResearch/BioinformaticsTools/LiverToxicityKnowledgeBase/ucm2024036.htm last accessed 14/05/2013. This dataset was chosen as it was publically available and provided the necessary binary calls. Other suitable datasets would have needed further expert interpretation to create these binary calls. LTKB has 287 substances but there is no structure available for the monoclonal antibody gemtuzumab.',286,79,53,12,142,NULL,NULL,'N/A',NULL,NULL,'All alerts describing structure-activity relationships for the hepatotoxicity endpoint have a mechanistic basis wherever possible. Mechanistic information is detailed in the comments associated with an alert and can include information on both the mechanism of action and biological target.\r\nThe mechanistic basis of the model was developed a priori by examining the active and inactive structures before developing the structure-activity relationship.','All references supporting the mechanistic basis of an alert are detailed and available for inspection within the software.','N/A','N/A','LL','mammal','http://etoxdev.lhasalimited.org:10000/services/etox-prediction-derek-rest','boolean','Probable, Plausible, Equivocal, Doubted, Improbable, Impossible, Nothing to report','Probable = there is at least one strong argument that the proposition is true and there are no arguments against it, Plausible = weight of evidence supports the proposition, Equivocal = equal weight of evidence for and against the proposition, Doubted = weight of evidence opposes the proposition, Improbable = there is at least one strong argument that the proposition is false and there are no arguments that it is true, Impossible = there is proof that the proposition is false & Nothing to report = no prediction provided as no structural alerts fired','/Toxicity/Organ Toxicity/Hepatotoxicity/DILI Mammal/1','Not verified','na',NULL),(222,'2012-05-16 00:00:00','Olivier Taboureau','otab@cbs.dtu.dk','Model developed for eTOX','Phospholipidosis','DIPL 1 DTU','beta','Phospholipidosis','Lowe R. et al. Mol. Pharm. 2010, 7, 1708-1714; Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27','Phospholipidosis','The model is qualitative with a positive or negative outcome. Positive: Strong evidence of phospholipidosis. Negative: No strong evidence of phospholipidosis.','Lowe R. et al. Mol. Pharm. 2010, 7, 1708-1714.',185,'small molecules','No data extra curation was done',55.00,0.0000,1.0000,0.0000,'N/A','compound name','small molecules','Binary classification','Support Vector Machine','185/485','881 PubChem fingerprints','N/A','No Applicability Domain measure was applied.','No Applicability Domain measure was applied.',87,76,7,15,NULL,NULL,NULL,'485 compounds from Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27.',485,87,256,10,132,NULL,NULL,NULL,NULL,NULL,'The model reflects molecular fragments associated with phospholipidosis. ','N/A','N/A','Information about the data can be found at these references:Lowe R. et al. Mol. Pharm. 2010, 7, 1708-1714; Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27','DTU','mammal','http://potentia.cbs.dtu.dk/etox/ws/','boolean','negative,positive','Compounds predicted to be positive are likely to produce phospholipidosis',NULL,'Verification ongoing','na',NULL),(96,'2013-03-13 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Szakács G et al. (2004) Cancer Cell 6: 129-137','ABCB1 substrate classification model','ABCB1 Transport 1 UNIVIE','1.0','ATP-binding cassette transporter, P-gp, P-glycoprotein, ABCB1','N/A','ABCB1 substrate classification model','Positive=substrate; negative=non-substrate','The database, containing the NSC identifiers for 1429 compounds, was available as supplementary material at the http://www.sciencedirect.com/science/article/pii/S1535610804002065 webpage. Chemical structures of these compounds were retrieved according to their NSC identifier from the NCI database file NCI_aug00_2D.sdf, downloaded from the http://cactus.nci.nih.gov/download/nci/ web site. No structure was found for 16 of these compounds. Accordingly, a database formed by 1223 compounds was obtained.',1190,'Mostly natural products','As described in [Pinto M et al. (2012) Mol Inf 31:547-553].',10.30,0.0000,0.0000,0.0000,'yes (unbalanced dataset)','NSC','N/A','Tree','The model was obtained using Random Forest (I=70) in combination with the feature selection algorithm \"CfsSubsetEval + BestFirst\" and the MetaCost (cost matrix [0.0 1.0; 9.0 0.0]) as implemented in Weka v.3.7.10 (http://www.cs.waikato.ac.nz/~ml/weka).','10-fold cross-validation','PaDEL descriptors have been calculated using the node implemented in KNIME v.2.6.2 (http://knime.org).','Descriptors were selected using the feature selection algorithm \"CfsSubsetEval + BestFirst\" as implemented in Weka v.3.7.10.','ADAN as implemented in eTOXlab','ADAN as implemented in eTOXlab',123,951,116,0,0.0000,0.0000,'N/A','10-fold cross-validation',1190,81,955,112,42,0.0000,0.0000,'Sensitivity=0.66, Specificity=0.90, MCC=0.46',0.0000,0.0000,'The model predicts if a query compound will be or not transported by ABCB1 based on 2D physicochemical descriptors. ','N/A','N/A','5-fold cross validation: Specifity=0.93; Sensitivity=0.62; Precision=0.49; G-Mean=0.76; Matthew correlation coefficient=0.49; Accuracy=0.89; ROC area= 0.87; 10-fold cross validation: Specifity=0.94; Sensitivity=0.57; Precision=0.51; G-Mean=0.73; Matthew correlation coefficient=0.48; Accuracy=0.90; ROC area= 0.87','UNIVIE','human','http://131.130.129.209/etoxwsapi/v2/info','boolean','negative, positive','The model predicts if a query compound will be or not transported by ABCB1. If \"negative\" is obtained, then the compound is predicted as not being transported by ABCB1. If \"positive\" is obtained, then the compound is predicted as substrate of ABCB1.','/ADME/Transport/Transporters/ABCB1 Transport/1','Verified','na',NULL),(97,'2013-03-13 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Szakács G et al. (2004) Cancer Cell 6: 129-137','ABCB11 substrate classification model','ABCB11 Transport 1 UNIVIE','1.0','ATP-binding cassette transporter, Bile salt export pump, BSEP, ABCB11','N/A','ABCB11 substrate classification model','Positive=substrate; negative=non-substrate','The database, containing the NSC identifiers for 1429 compounds, was available as supplementary material at the http://www.sciencedirect.com/science/article/pii/S1535610804002065 webpage. Chemical structures of these compounds were retrieved according to their NSC identifier from the NCI database file NCI_aug00_2D.sdf, downloaded from the http://cactus.nci.nih.gov/download/nci/ web site. No structure was found for 16 of these compounds. Accordingly, a database formed by 1223 compounds was obtained.',1190,'Mostly natural products','As described in [Pinto M et al. (2012) Mol Inf 31:547-553].',14.50,0.0000,0.0000,0.0000,'yes (unbalanced dataset)','NSC','N/A','Tree','The model was obtained using the cost sensitive meta-classifier MetaCost (cost matrix [0.0 1.0; 6.0, 0.0]) and the classifier Random Forest (I=90)  as implemented in Weka v.3.7.10 (http://www.cs.waikato.ac.nz/~ml/weka).','10-fold cross-validation','PaDEL descriptors have been calculated using the node implemented in KNIME v.2.6.2 (http://knime.org).','None','ADAN as implemented in eTOXlab','ADAN as implemented in eTOXlab',172,906,112,0,0.0000,0.0000,'N/A','10-fold cross-validation',1190,66,906,112,106,0.0000,0.0000,'Sensitivity=0.38, Specificity=0.89, MCC=0.27',0.0000,0.0000,'The model predicts if a query compound will be or not transported by ABCB11 using 2D descriptors.','N/A','N/A','N/A','UNIVIE','human','http://131.130.129.209/etoxwsapi/v2/info','boolean','negative, positive','The model predicts if a query compound will be or not transported by ABCB11. If \"negative\" is obtained, then the compound is predicted as not transported by ABCB11. If \"positive\" is obtained, then the compound is predicted as substrate of ABCB11.','/ADME/Transport/Transporters/ABCB11 Transport/1','Verified','na',NULL),(108,'2014-09-17 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','[Montanari F. et al. (2014) Mol Inf 33:322-331]','ABCG2 inhibition classification model','ABCG2 Inhibition 1 UNIVIE','1.0','ATP-binding cassette transporter, breast cancer resistance protein, BCRP, ABCG2','N/A','ABCG2 (BCRP) inhibition classification model','Positive=inhibitor; negative=non-inhibitor','The training set corresponds to the publication \"BCRP Inhibition: from Data Collection to Ligand-Based Modeling\" by Montanari and Ecker (DOI: 10.1002/minf.201400012). It contains 978 compounds gathered and cleaned according to the publication. ',978,'Drugs, natural products, variations around some specific scaffolds','As described in [Montanari F. et al. (2014) Mol Inf 33:322-331].',44.00,0.0000,0.0000,0.0000,'no','common names or PubchemID','N/A','Logistic regression','The model was obtained using a parameter-optimized logistic regression as implemented in the scikit-learn libraries. The parameter grids to be searched are: \'penalty\': [\'l1\', \'l2\'] and \'C\': [0.001, 0.01, 0.1, 0.5, 1, 2, 5, 10]. The GridSearch function was called with an inner 5-fold cross-validation to select the best set of parameters according to the ROC AUC value.\r\n','10-fold cross-validation','ECFP-like descriptors were computed using the RDKit library. The diameter was set to 4 and the bits were folded to a final size of 1024.','None','ADAN as implemented in eTOXlab','ADAN as implemented in eTOXlab',404,530,15,29,0.0000,0.0000,'N/A','10 fold CV results are given below.',978,333,473,72,100,0.0000,0.0000,'Sensitivity=0.77, Specificity=0.87, MCC=0.64',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of ABCG2.','N/A','N/A','N/A','UNIVIE','human','N/A','boolean','negative, positive','The model predicts if a query compound will be or not an inhibitor of ABCG2 (BCRP). If \"negative\" is obtained, then the compound is predicted as non inhibitor of ABCG2. If \"positive\" is obtained, then the compound is predicted as inhibitor of ABCG2.','/ADME/Transport/Transporters/ABCG2 Inhibition/1','Verified','na',NULL),(98,'2013-03-13 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Szakács G et al. (2004) Cancer Cell 6: 129-137; Pinto M et al. (2012) Mol Inf 31:547-553','ABCC2 substrate classification model','ABCC2 Transport 1 UNIVIE','1.0','ATP-binding cassette transporter, Multidrug resistance-associated protein 2, MRP2, ABCC2','N/A','ABCC2 substrate classification model','Positive=substrate; negative=non-substrate','The database, containing the NSC identifiers for 1429 compounds, was available as supplementary material at the http://www.sciencedirect.com/science/article/pii/S1535610804002065 webpage. Chemical structures of these compounds were retrieved according to their NSC identifier from the NCI database file NCI_aug00_2D.sdf, downloaded from the http://cactus.nci.nih.gov/download/nci/ web site. No structure was found for 16 of these compounds. Accordingly, a database formed by 1223 compounds was obtained.',1190,'Mostly natural products','As described in [Pinto M et al. (2012) Mol Inf 31:547-553].',20.00,0.0000,0.0000,0.0000,'yes (unbalanced dataset)','NSC','N/A','Tree','The model was obtained using the cost-sensitive meta-classifier MetaCost (cost matrix [0.0 1.0; 4.0 0.0]) and the classifier Random Forest (I=120)  as implemented in Weka v.3.7.10 (http://www.cs.waikato.ac.nz/~ml/weka).','10-fold cross-validation','PaDEL descriptors have been calculated using the node implemented in KNIME v.2.6.2 (http://knime.org).','None.','ADAN as implemented in eTOXlab','ADAN as implemented in eTOXlab',247,878,65,0,0.0000,0.0000,'N/A','10-fold cross-validation',1190,97,869,74,150,0.0000,0.0000,'Sensitivity=0.39, Specificity=0.92, MCC=0.36',0.0000,0.0000,'The model predicts if a query compound will be or not transported by ABCC2 using 2D descriptors.','N/A','N/A','N/A','UNIVIE','human','http://131.130.129.209/etoxwsapi/v2/info','boolean','negative, positive','The model predicts if a query compound will be or not transported by ABCC2. If \"negative\" is obtained, then the compound is predicted as not being transported by ABCC2. If \"positive\" is obtained, then the compound is predicted as substrate of ABCC2.','/ADME/Transport/Transporters/ABCC2 Transport/1','Verified','na',NULL),(102,'2013-04-13 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Montanari et al., \"Flagging Drugs That Inhibit the Bile Salt Export Pump\", Molecular Pharmaceutics, 2016, 13 (1), pp 163–171\r\nDOI: 10.1021/acs.molpharmaceut.5b00594','ABCB11 inhibitor classification model','ABCB11 Inhibition 1 UNIVIE','1.0','ATP-binding cassette transporter, Bile salt export pump, BSEP, ABCB11','N/A','ABCB11 inhibitor classification model','Positive=inhibitor; negative=non-inhibitor','The database, formed by 1096 compounds, was provided by AstraZeneca.',669,'Different structural classes.','As described in [Pinto M et al. (2012) Mol Inf 31:547-553].',32.80,0.0000,0.0000,0.0000,'a bit','Confidential data','Confidential data','Tree','The model was obtained using the classifier Random Forest (I=70)  as implemented in Weka v.3.7.10 (http://www.cs.waikato.ac.nz/~ml/weka).','80% of each category (inhibitors and non-inhibitors, respectively) was randomly selected to use as training set using KNIME v.2.6.2 (http://knime.org). Remaining compounds were used as test set.','PaDEL descriptors have been calculated using the node implemented in KNIME v.2.6.2 (http://knime.org).','None.','None because of the confidentiality of the data','None because of the confidentiality of the data',219,450,0,0,0.0000,0.0000,'N/A','10-fold cross-validation',669,172,410,40,47,0.0000,0.0000,'Sensitivity=0.79, Specificity=0.91, MCC=0.70',0.0000,0.0000,'The model predicts if a query compound will be or not inhibitor of ABCB11 using 2D descriptors.','N/A','N/A',NULL,'UNIVIE','human','N/A','boolean','negative, positive','The model predicts if a query compound will be or not inhibitor of ABCB11. If \"negative\" is obtained, then the compound is predicted as non inhibitor of ABCB11. If \"positive\" is obtained, then the compound is predicted as inhibitor of ABCB11.','/ADME/Transport/Transporters/ABCB11 Inhibition/1','Not verified','na',NULL),(103,'2013-04-13 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','N/A','Abcc2 inhibitor classification model','ABCC2 Inhibition 1 UNIVIE','1.0','ATP-binding cassette transporter, Multidrug resistance-associated protein 2, Mrp2, Abcc2','N/A','ABCC2 inhibitor classification model','Positive=inhibitor; negative=non-inhibitor','The database, formed by 1689 compounds, was provided by AstraZeneca.',1653,'Different structural classes.','As described in [Pinto M et al. (2012) Mol Inf 31:547-553].',18.00,0.0000,0.0000,0.0000,'yes (unbalanced dataset)','Confidential data','Confidential data','Tree','The model was obtained using the classifier Random Forest (I=80) in combination with the MetaCost cost sensitive scoring (cost matrix [0.0, 1.0; 4.3, 0.0]) as implemented in Weka v.3.7.10 (http://www.cs.waikato.ac.nz/~ml/weka).','10-fold cross-validation','PaDEL 2D descriptors have been calculated using the node implemented in KNIME v.2.6.2 (http://knime.org).','None.','None because of the confidentiality of the data','None because of the confidentiality of the data',300,1250,103,0,0.0000,0.0000,'N/A','10-fold cross-validation',1653,156,1243,110,144,0.0000,0.0000,'Sensitivity=0.52, Specificity=0.92, MCC=0.46',0.0000,0.0000,'The model predicts if a query compound will be or not inhibitor of ABCC2 using 2D descriptors.','N/A','N/A','N/A','UNIVIE','rat','N/A','boolean','negative, positive','The model predicts if a query compound will be or not inhibitor of ABCC2. If \"negative\" is obtained, then the compound is predicted as non inhibitor of ABCC2. If \"positive\" is obtained, then the compound is predicted as inhibitor of ABCC2.','/ADME/Transport/Transporters/ABCC2 Inhibition/1','Not verified','na',NULL),(133,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT6 Affinity 1','5HT6 Affinity 1 CT','1','5HT6, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT6','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT6 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2153,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.2000,10.7000,7.2800,'Skewness=-0.46','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7700,0.5900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',380,NULL,NULL,NULL,NULL,0.7800,0.5700,'N/A',0.7700,0.5900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT6 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT6_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(100,'2013-03-13 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Szakács G et al. (2004) Cancer Cell 6: 129-137; Pinto M et al. (2012) Mol Inf 31:547-553','ABCG2 substrate classification model','ABCG2 Transport 1 UNIVIE','1.0','ATP-binding cassette transporter, Breast cancer resistant protein, BCRP, ABCG2','N/A','ABCG2 substrate classification model','Positive=substrate; negative=non-substrate','The database, containing the NSC identifiers for 1429 compounds, was available as supplementary material at the http://www.sciencedirect.com/science/article/pii/S1535610804002065 webpage. Chemical structures of these compounds were retrieved according to their NSC identifier from the NCI database file NCI_aug00_2D.sdf, downloaded from the http://cactus.nci.nih.gov/download/nci/ web site. No structure was found for 16 of these compounds. Accordingly, a database formed by 1223 compounds was obtained.',1190,'Mostly natural products','As described in [Pinto M et al. (2012) Mol Inf 31:547-553].',9.10,0.0000,0.0000,0.0000,'yes (unbalanced dataset)','NSC','N/A','Tree','The model was obtained using Metacost (FN=10; FP=1) and Random Forest (I=150)  as implemented in Weka v.3.7.10 (http://www.cs.waikato.ac.nz/~ml/weka).','10-fold cross-validation','PaDEL descriptors have been calculated using the node implemented in KNIME v.2.6.2 (http://knime.org).','None.','ADAN as implemented in eTOXlab','ADAN as implemented in eTOXlab',108,957,125,0,0.0000,0.0000,'N/A','10-fold cross-validation',1190,44,1006,76,64,0.0000,0.0000,'Sensitivity=0.41, Specificity=0.93, MCC=0.32',0.0000,0.0000,'The model predicts if a query compound will be or not transported by ABCG2 using 2D descriptors.','N/A','N/A','5-fold cross validation: Specifity=0.93; Sensitivity=0.39; Precision=0.35; G-Mean=0.60; Matthew correlation coefficient=0.30; Accuracy=0.88; ROC area= 0.80; 10-fold cross validation: Specifity=0.93; Sensitivity=0.40; Precision=0.36; G-Mean=0.61; Matthew correlation coefficient=0.31; Accuracy=0.88; ROC area= 0.80','UNIVIE','human','http://131.130.129.209/etoxwsapi/v2/info','boolean','negative, positive','The model predicts if a query compound will be or not transported by ABCG2. If \"negative\" is obtained, then the compound is predicted as not being transported by ABCG2. If \"positive\" is obtained, then the compound is predicted as substrate of ABCG2.','/ADME/Transport/Transporters/ABCG2 Transport/1','Verified','na',NULL),(109,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions,\r\n','OATP1B1 inhibition classification model 2','OATP1B1 Inhibition 2 UNIVIE','1.0','OATP1B1, organic anion transporting polypeptide B1, SLCO1B1, OATP-C','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B1 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1708,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',11.12,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','Tree','Random Forest as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the default settings, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 8.0, 0.0]\"','10-fold cross-validation','The following 11 1D and 2D PaDEL descriptors have been calculated: logP (CrippenLogP), Molecular Refractivity (CrippenMR), number of H-bond acceptors (nHBAcc_Lipinski), number of H-bond donors (nHBDon_Lipinski), Topological Polar Surface Area (TopoPSA), and Molecular Weight (MW), number of rotable bonds (nRotB), and a set of topological descriptors (topoRadius, topoDiameter, topoShape, globalTopoChargeIndex)','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',181,1345,173,9,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1708,113,1351,167,77,0.0000,0.0000,'Sensitivity=0.59, Specificity=0.89, MCC=0.41',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B1.','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B1. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B1. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B1.','/ADME/Transport/Transporters/OATP1B1 Inhibition/2','Verified','na',NULL),(110,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B1 inhibition classification model 1','OATP1B1 Inhibition 1 UNIVIE','1.0','OATP1B1, organic anion transporting polypeptide B1, SLCO1B1, OATP-C','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B1 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1708,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',11.12,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','Tree','Random Forest as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the default settings, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 8.0, 0.0]\"','10-fold cross-validation','The following 6 1D and 2D RDKit descriptors have been calculated:CrippenDescriptors (2 descriptors), NumLipinskiHBA, NumLipinskiHBD, TPSA, ExactMolWt','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',183,1360,158,7,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1708,117,1348,170,73,0.0000,0.0000,'Sensitivity=0.62, Specificity=0.89, MCC=0.42',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B1.','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B1. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B1. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B1.','/ADME/Transport/Transporters/OATP1B1 Inhibition/1','Verified','na',NULL),(111,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B1 inhibition classification model 3','OATP1B1 Inhibition 3 UNIVIE','1.0','OATP1B1, organic anion transporting polypeptide B1, SLCO1B1, OATP-C','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B1 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1708,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',11.12,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','SVM','SMO as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the Puk kernel, \"Build Logistic models\" option set to \"True\" and the rest settings at default, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 8.0, 0.0]\"','10-fold cross-validation','The following 6 1D and 2D RDKit descriptors have been calculated:CrippenDescriptors (2 descriptors), NumLipinskiHBA, NumLipinskiHBD, TPSA, ExactMolWt','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',97,1394,124,93,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1708,91,1390,128,99,0.0000,0.0000,'Sensitivity=0.48, Specificity=0.92, MCC=0.37',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B1.','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B1. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B1. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B1.','/ADME/Transport/Transporters/OATP1B1 Inhibition/3','Verified','na',NULL),(112,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B1 inhibition classification model 4','OATP1B1 Inhibition 4 UNIVIE','1.0','OATP1B1, organic anion transporting polypeptide B1, SLCO1B1, OATP-C','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B1 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1708,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',11.12,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','SVM','SMO as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the Puk kernel, \"Build Logistic models\" option set to \"True\" and the rest settings at default, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 8.0, 0.0]\"','10-fold cross-validation','The following 11 1D and 2D PaDEL descriptors have been calculated: logP (CrippenLogP), Molecular Refractivity (CrippenMR), number of H-bond acceptors (nHBAcc_Lipinski), number of H-bond donors (nHBDon_Lipinski), Topological Polar Surface Area (TopoPSA), and Molecular Weight (MW), number of rotable bonds (nRotB), and a set of topological descriptors (topoRadius, topoDiameter, topoShape, globalTopoChargeIndex)','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',101,1382,136,89,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1708,96,1378,140,94,0.0000,0.0000,'Sensitivity=0.51, Specificity=0.91, MCC=0.38',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B1.','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B1. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B1. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B1.','/ADME/Transport/Transporters/OATP1B1 Inhibition/4','Verified','na',NULL),(113,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B3 inhibition classification model 1','OATP1B3 Inhibition 1 UNIVIE','1.0','OATP1B3, organic anion transporting polypeptide B3, SLCO1B3, OATP-8','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B3 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1725,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',7.19,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','Tree','Random Forest as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the default settings, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 13.0, 0.0]\"','10-fold cross-validation','The following 6 1D and 2D RDKit descriptors have been calculated:CrippenDescriptors (2 descriptors), NumLipinskiHBA, NumLipinskiHBD, TPSA, ExactMolWt','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',115,1455,146,9,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1725,81,1425,176,43,0.0000,0.0000,'Sensitivity=0.65, Specificity=0.89, MCC=0.39',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B3. ','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B3. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B3. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B3.','/ADME/Transport/Transporters/OATP1B3 Inhibition/1','Verified','na',NULL),(114,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B3 inhibition classification model 2','OATP1B3 Inhibition 2 UNIVIE','1.0','OATP1B3, organic anion transporting polypeptide B3, SLCO1B3, OATP-8','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B3 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1725,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',7.19,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','Tree','Random Forest as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the default settings, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 13.0, 0.0]\"','10-fold cross-validation','The following 11 1D and 2D PaDEL descriptors have been calculated: logP (CrippenLogP), Molecular Refractivity (CrippenMR), number of H-bond acceptors (nHBAcc_Lipinski), number of H-bond donors (nHBDon_Lipinski), Topological Polar Surface Area (TopoPSA), and Molecular Weight (MW), number of rotable bonds (nRotB), and a set of topological descriptors (topoRadius, topoDiameter, topoShape, globalTopoChargeIndex)','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',116,1415,186,8,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1725,85,1414,187,39,0.0000,0.0000,'Sensitivity=0.69, Specificity=0.88, MCC=0.40',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B3','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B3. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B3. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B3.','/ADME/Transport/Transporters/OATP1B3 Inhibition/2','Verified','na',NULL),(115,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B3 inhibition classification model 3','OATP1B3 Inhibition 3 UNIVIE','1.0','OATP1B3, organic anion transporting polypeptide B3, SLCO1B3, OATP-8','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B3 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1725,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',7.19,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','SVM','SMO as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the Puk kernel, \"Build Logistic models\" option set to \"True\" and the rest settings at default, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 13.0, 0.0]\"','10-fold cross-validation','The following 6 1D and 2D RDKit descriptors have been calculated:CrippenDescriptors (2 descriptors), NumLipinskiHBA, NumLipinskiHBD, TPSA, ExactMolWt','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',72,1470,131,52,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1725,71,1478,123,53,0.0000,0.0000,'Sensitivity=0.57, Specificity=0.92, MCC=0.41',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B3. ','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B3. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B3. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B3.','/ADME/Transport/Transporters/OATP1B3 Inhibition/3','Verified','na',NULL),(116,'2014-09-18 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Kotsampasakou et al., 2015, Mol Pharmaceutics, 12, 4395-4404, Identification of Novel Inhibitors of Organic Anion Transporting\r\nPolypeptides 1B1 and 1B3 (OATP1B1 and OATP1B3) Using a\r\nConsensus Vote of Six Classification Models,\r\nDe Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors,\r\nKalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','OATP1B3 inhibition classification model 4','OATP1B3 Inhibition 4 UNIVIE','1.0','OATP1B3, organic anion transporting polypeptide B3, SLCO1B3, OATP-8','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267,  Karlgren et al., 2012, J Med Chem, 4740-4763','OATP1B3 inhibition classification model','positive = inhibitor, negative = non-inhibitor','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors',1725,'Drugs, natural products, chemicals; different structural classes','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',7.19,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','SVM','SMO as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the Puk kernel, \"Build Logistic models\" option set to \"True\" and the rest settings at default, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 13.0, 0.0]\"','10-fold cross-validation','The following 11 1D and 2D PaDEL descriptors have been calculated: logP (CrippenLogP), Molecular Refractivity (CrippenMR), number of H-bond acceptors (nHBAcc_Lipinski), number of H-bond donors (nHBDon_Lipinski), Topological Polar Surface Area (TopoPSA), and Molecular Weight (MW), number of rotable bonds (nRotB), and a set of topological descriptors (topoRadius, topoDiameter, topoShape, globalTopoChargeIndex)','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',82,1454,147,42,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',1725,72,1451,150,52,0.0000,0.0000,'Sensitivity=0.58, Specificity=0.91, MCC=0.38',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of OATP1B3','N/A','N/A','De Bruyn et al., 2013, Mol Pharmacol, 1257-1267, Structure-Based Identification of OATP1B1/3 Inhibitors, Kalgren et al., 2012, J Med Chem, 4740-4763, Classification of Hepatic Organic Anion Transporting Polypeptides (OATPs): Influence of Protein Expression on Drug-Drug Interactions','UNIVIE','human',' http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not an inhibitor of OATP1B3. If \"negative\" is obtained, then the compound is predicted as non inhibitor of OATP1B3. If \"positive\" is obtained, then the compound is predicted as inhibitor of OATP1B3.','/ADME/Transport/Transporters/OATP1B3 Inhibition/4','Verified','na',NULL),(117,'2014-09-18 00:00:00','Guillem Plasencia Gallofre','guillemp@leadmolecular.com','Dataset for human CYP 1A2 (Uniprot: P05177)  was downloaded from ChOX (v 3.1) datasets. Only IC50 entries were considered.','SVM Human CYP 1A2 Inhibition','CYP 1A2 Inhibition 1 LMD','1','SVM 1A2 Inhibition','Drug Metab Dispos. 2014 Nov;42(11):1811-9. doi: 10.1124/dmd.114.057893','Predicts if a compound inhibits human CYP 1A2. Possible predicted values are -1, 0 and 1. A compound is assigned to category \"-1\" if it has < 50% inhibition at 10 micromolar, is assigned to category \"0\" if it has 50% inhibition at > 10 micromolar, and to category \"1\" if it has 50% inhibition at < 10 micromolar.','Categories \"-1\", \"0\" and \"1\" as explained in endpoint definition.','ChOX datasets at https:/https://www.ebi.ac.uk/chembl/etox/index.php/chox/datasets/show/www.ebi.ac.uk/chembl/etox/index.php/chox/datasets/show',1744,'Dataset of compounds inhibiting CYP 1A2 compiled by EBI eTOX partner from chEBML database.','Compounds with Fe and V atoms were removed because they can\'t be handled by Volsurf+. Also, any compound containing any Volsurf+ descriptor value \"n/a\" was not included in the model because eTOXLab can\'t handle non-numeric values for molecular descriptors.',NULL,-1.0000,1.0000,-0.2580,'Skewness in a bimodal endpoint is already expressed by the endpoint avg.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in three categories using Volsurf+ descriptors','RBF SVM categorization based on all 128 Volsurf+ descriptors. ','A set 68 compounds were randomly chosen from the original dataset and assigned to the test set, never used in the model building steps.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,NULL,NULL,'Internal val. error rate=0.058','A set of 68 randomly chosen compounds from the original dataset, never used in the model building step.',68,NULL,NULL,NULL,NULL,NULL,NULL,'Error rate=0.193',0.0000,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or 0 or +1','A compound is assigned to category \"-1\" if it has < 50% inhibition at 10 micromolar, is assigned to category \"0\" if it has 50% inhibition at > 10 micromolar, and to category \"1\" if it has 50% inhibition at < 10 micromolar.','/ADME/Metabolism/CYP Inhibition/CYP 1A2 Inhibition/1','Verified','na',NULL),(118,'2014-09-18 00:00:00','Guillem Plasencia Gallofre','guillemp@leadmolecular.com','Dataset for human CYP 2D6 (Uniprot: P10635)  was downloaded from ChOX (v 3.1) datasets. Only IC50 entries were considered.','SVM Human CYP 2D6 Inhibition','CYP 2D6 Inhibition 1 LMD','1','SVM 2D6 Inhibition','Drug Metab Dispos. 2014 Nov;42(11):1811-9. doi: 10.1124/dmd.114.057893','Predicts if a compound inhibits human CYP 2D6 Possible predicted values are -1, 0 and 1. A compound is assigned to category \"-1\" if it has < 50% inhibition at 10 micromolar, is assigned to category \"0\" if it has 50% inhibition at > 10 micromolar, and to category \"1\" if it has 50% inhibition at < 10 micromolar.','Categories \"-1\", \"0\" and \"1\" as explained in endpoint definition.','ChOX datasets at https:/https://www.ebi.ac.uk/chembl/etox/index.php/chox/datasets/show/www.ebi.ac.uk/chembl/etox/index.php/chox/datasets/show',3043,'Dataset of CYP 2D6 inhibiting compounds compiled by EBI eTOX partner from chEMBL database.','Compounds with Fe and V atoms were removed because they can\'t be handled by Volsurf+. Also, any compound containing any Volsurf+ descriptor value \"n/a\" was not included in the model because eTOXLab can\'t handle non-numeric values for molecular descriptors.',NULL,-1.0000,1.0000,0.0388,'Skewness in a bimodal endpoint is already expressed by the endpoint avg.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in three categories using Volsurf+ descriptors','RBF SVM categorization based on all 128 Volsurf+ descriptors. ','A set 170 compounds were randomly chosen from the original dataset and assigned to the test set, never used in the model building steps.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,NULL,NULL,'Internal val. error rate=0.069','A set of 170 randomly chosen compounds from the original dataset, never used in the model building step.',170,NULL,NULL,NULL,NULL,NULL,NULL,'Error rate=0.269',0.0000,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or 0 or +1','A compound is assigned to category \"-1\" if it has < 50% inhibition at 10 micromolar, is assigned to category \"0\" if it has 50% inhibition at > 10 micromolar, and to category \"1\" if it has 50% inhibition at < 10 micromolar.','/ADME/Metabolism/CYP Inhibition/CYP 2D6 Inhibition/1','Verified','na',NULL),(119,'2014-09-18 00:00:00','Guillem Plasencia Gallofre','guillemp@leadmolecular.com','Dataset for human CYP 3A4 (Uniprot: P08684)  was downloaded from ChOX (v 3.1) datasets. Only IC50 entries were considered.','SVM Human CYP 3A4 Inhibition','CYP 3A4 Inhibition 1 LMD','1','SVM 3A4 Inhibition','Drug Metab Dispos. 2014 Nov;42(11):1811-9. doi: 10.1124/dmd.114.057893','Predicts if a compound inhibits human CYP 3A4. Possible predicted values are -1, 0 and 1. A compound is assigned to category \"-1\" if it has < 50% inhibition at 10 micromolar, is assigned to category \"0\" if it has 50% inhibition at > 10 micromolar, and to category \"1\" if it has 50% inhibition at < 10 micromolar.','Categories \"-1\", \"0\" and \"1\" as explained in endpoint definition.','ChOX datasets at https:/https://www.ebi.ac.uk/chembl/etox/index.php/chox/datasets/show/www.ebi.ac.uk/chembl/etox/index.php/chox/datasets/show',3664,'Compounds in the CYP 3A4 dataset compiled by EBI eTOX partner from chEMBL entries.','Compounds with Fe and V atoms were removed because they can\'t be handled by Volsurf+. Also, any compound containing any Volsurf+ descriptor value \"n/a\" was not included in the model because eTOXLab can\'t handle non-numeric values for molecular descriptors.',NULL,-1.0000,1.0000,0.1560,'Skewness in a bimodal endpoint is already expressed by the endpoint avg.','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in three categories using Volsurf+ descriptors','RBF SVM categorization based on all 128 Volsurf+ descriptors. ','A set 182 compounds were randomly chosen from the original dataset and assigned to the test set, never used in the model building steps.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',NULL,NULL,NULL,NULL,NULL,NULL,'Internal validation error rate=0.071','A set of 182 randomly chosen compounds from the original dataset, never used in the model building step.',182,NULL,NULL,NULL,NULL,NULL,NULL,'Error rate=0.231',0.0000,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or 0 or +1','A compound is assigned to category \"-1\" if it has < 50% inhibition at 10 micromolar, is assigned to category \"0\" if it has 50% inhibition at > 10 micromolar, and to category \"1\" if it has 50% inhibition at < 10 micromolar.','/ADME/Metabolism/CYP Inhibition/CYP 3A4 Inhibition/1','Verified','na',NULL),(120,'2012-05-25 00:00:00','Daan Geerke','d.p.geerke@vu.nl','C. R. Vosmeer, R. Pool, M. F. van Stee, L. Perić-Hassler, N. P. E. Vermeulen and D. P. Geerke. \"Towards automated binding affinity prediction using an iterative linear energy approach.\" Int. J. Mol. Sci. 2014, 15, 798-816. DOI:10.3390/ijms15010798','LIE prediction of binding affinity to CYP450 2D6','CYP 2D6 Affinity 2 VUA','1.0','CYP 2D6, binding affinity, free energies of binding, IC50s','CYP 1A2 Affinity 1 VUA, CYP 2C9 Affinity 1 VUA, CYP 2D6 Affinity 1 VUA, MAPK1 Affinity 1 VUA','CYP 2D6 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5) from [Vaz et al. \"A 3D-QSAR model for CYP2D6 inhibition in the aryloxypropanolamine series.\" Bioorg. Med. Chem. Lett. 2005, 15, 3816-3820.]',9,'A structurally homogeneous set of compounds (aryloxypropanolamines).','9 training compounds were selected from Vaz et al. (see \"Training set source\") to obtain a wide spread in experimental data and to represent a large spread in heterogeneity of the compounds.',NULL,-48.2200,-27.3100,-36.2800,'no','IDs were assigned according to the IDs used by Vaz et al.','Compound purities were not reported by Vaz et al.','LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using linear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverable D3.5).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5) from [Vaz et al. \"A 3D-QSAR model for CYP2D6 inhibition in the aryloxypropanolamine series.\" Bioorg. Med. Chem. Lett. 2005, 15, 3816-3820.] After selection of the 9 training compounds, and excluding 5 of the remaining compounds (ID=19,20,24,29,36) which were not in the applicability domain (see \"AD method\"), the remaining 22 compounds were used as test set.','DVele and DVvdw. Using LIE, binding free energies (affinities) are calculated from the average interaction energies (DVele and DVvdw) of the compound with its surroundings as obtained from MD simulations (see also deliverables D3.5 and D3.13).','N/A (cf. \"Descriptors definition\")','Model is calibrated based on aryloxypropanolamine compounds.','One or more AD criteria should be fulfilled, which are determined from (1) a Mahalanobis-distance analysis on measured simulation properties, (2) ligand-fingerprint similarities, (3) and (4) two residue-based protein-ligand interaction patterns in simulation (D3.13), and (5) spread in experimental data.',NULL,NULL,NULL,NULL,0.8000,3.2000,'N/A','See \"Test set selection\"',22,NULL,NULL,NULL,NULL,0.8100,4.3000,NULL,0.4700,5.3000,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 2D6.','See \"Mechanistic basis of the model\" and eTOX deliverable D3.13 for details on the Applicability Domain assessment.','LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034; L. Perić-Hassler, E. Stjernschantz, C. Oostenbrink and D. P. Geerke. Int. J. Mol. Sci. 2013, 14, 24514-24530. DOI:10.3390/ijms141224514',NULL,'VUA','human',NULL,'float','(-inf,inf)','Binding free energies (affinities) DG can be translated into dissociation or inhibition constants K by using DG = RT ln K. The more negative DG is for a compound, the stronger it binds to CYP 2D6.',NULL,'Verification ongoing','na',NULL),(121,'2014-09-24 00:00:00','Guillem Plasencia','guillemp@leadmolecular.com','Dataset for SLC22A8 (Uniprot: Q8TCC7) was downloaded from ChOX datasets website.','SLC22A8 inhibition prediction','SLC22A8 inhibition 1 LMD','1','SLC22A8 OAT3 inhibitor','PLoS One. 2014 Jun 26;9(6):e100278. doi: 10.1371/journal.pone.0100278. eCollection 2014','This model does not predict toxicity endpoints, but if a compound could be an inhibitor of  human organic anion transporter SLC22A8 (a.k.a. OAT3).','\"+1\" for a compound predicted to be an inhibitor, \"-1\" for those predicted to not be inhibitors','Dataset for SLC22A8 (Uniprot: Q8TCC7) was downloaded from ChOX datasets website.',49,'Compounds compiled from different published articles, as referenced in ChOX dataset table.','Compounds with Fe and V atoms were removed because they can\'t be handled by Volsurf+. Also, any compound containing any Volsurf+ descriptor value \"n/a\" was not included in the model because eTOXLab can\'t handle non-numeric values for molecular descriptors.',0.90,-1.0000,1.0000,0.7700,'Yes','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in 2 categories using Volsurf+ descriptors','RBF SVM categorization based on all 128 Volsurf+ descriptors. ','4 compounds (2 positives and 2 negatives) where selected from the original dataset and included in the test set, and never used in the model building step.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',39,5,0,0,NULL,NULL,NULL,'4 compounds from the original dataset were included in the test set, and never used in the model building step.',4,2,1,1,0,NULL,NULL,NULL,NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or +1','a \"-1\" is assigned to compounds predicted not to be inhibitors of SLC22A8, or \"+1\" to compounds predicted to inhibit SLC22A8.','/ADME/Transport/Transporters/SLC22A8 Inhibition/1','Verified','na',NULL),(513,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','Model for LQT potency 10%','LQTpotp','2','QT prolongation, LQT','CAV12p; HERG4p; KCNQ1p','This model predicts the expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. For example, a value of 20 means that a concentration of 20nM will be enough to produce a 10% increase in the QT segment. This','Predicted expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. This result expresses the potency of the effect (it is a concentration, NOT an effect value)','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of O\'Hara-Rudy model of human cardiomyocytes. Experimental values of the ','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,inf)','This model predicts the expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. For example, a value of 20 means that a concentration of 20nM will be enough to produce a 10% increase in the QT segment. This','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/10','Verified','nM',NULL),(512,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','Model for LQT potency 10%','LQTpot','2','QT prolongation, LQT','CAV12; HERG4; KCNQ1','This model predicts the expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. For example, a value of 20 means that a concentration of 20nM will be enough to produce a 10% increase in the QT segment. This','Predicted expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. This result expresses the potency of the effect (it is a concentration, NOT an effect value)','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of O\'Hara-Rudy model of human cardiomyocytes. Experimental values of the ','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,inf)','This model predicts the expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. For example, a value of 20 means that a concentration of 20nM will be enough to produce a 10% increase in the QT segment. This','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/8','Verified','nM',NULL),(510,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','Model for LQT increase','LQTincp','2','QT prolongation, LQT','CAV12p; HERG4p; KCNQ1p','This model predicts the expected value of the QT interval after the drug administration, expressed as the percentage of the normal QT value. For example, a result of 115 means that after the administration of the drug the QT value will be 115% of the norm','Predicted QT interval after the drug administration, expressed as the % of the normal QT value','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of O\'Hara-Rudy model of human cardiomyocytes. Experimental values of the ','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','This model predicts the expected value of the QT interval after the drug administration, expressed as the percentage of the normal QT value. For example, a result of 115 means that after the administration of the drug the QT value will be 115% of the norm','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/9','Verified','% (of normal QT in ms)',NULL),(511,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','Model for LQT potency 10%','LQTpot','1','QT prolongation; LQT; HERG; multiscale model','CAV12; HERG4; KCNQ1','This model predicts the expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. For example, a value of 20 means that a concentration of 20nM will be enough to produce a 10% increase in the QT segment. This','Predicted expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. This result expresses the potency of the effect (it is a concentration, NOT an effect value)','N/A',0,'check Related models.','check Related models.',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','\"multiscale model: predicted blockade of CAV12, HERG and KCNQ1 ion channels plus integration using a 1D electrophysiological simulation of ventricular tissue\"','check Related models.','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,inf)','This model predicts the expected concentration of the drug required to increase a 10% the length of the QT segment, in nM units. For example, a value of 20 means that a concentration of 20nM will be enough to produce a 10% increase in the QT segment. This','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/8','Not verified','na',NULL),(509,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','Model for LQT increase','LQTinc','2','QT prolongation, LQT','CAV12; HERG4; KCNQ1','This model predicts the expected value of the QT interval after the drug administration, expressed as the percentage of the normal QT value. For example, a result of 115 means that after the administration of the drug the QT value will be 115% of the norm','Predicted QT interval after the drug administration, expressed as the % of the normal QT value','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of O\'Hara-Rudy model of human cardiomyocytes. Experimental values of the ','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','This model predicts the expected value of the QT interval after the drug administration, expressed as the percentage of the normal QT value. For example, a result of 115 means that after the administration of the drug the QT value will be 115% of the norm','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/7','Verified','% (of normal QT in ms)',NULL),(508,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','Model for LQT increase','LQTinc','1','QT prolongation; LQT; HERG; multiscale model','CAV12; HERG4; KCNQ1','This model predicts the expected value of the QT interval after the drug administration, expressed as the percentage of the normal QT value. For example, a result of 115 means that after the administration of the drug the QT value will be 115% of the norm','Predicted QT interval after the drug administration, expressed as the % of the normal QT value','N/A',0,'check Related models.','check Related models.',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','\"multiscale model: predicted blockade of CAV12, HERG and KCNQ1 ion channels plus integration using a 1D electrophysiological simulation of ventricular tissue\"','check Related models.','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','This model predicts the expected value of the QT interval after the drug administration, expressed as the percentage of the normal QT value. For example, a result of 115 means that after the administration of the drug the QT value will be 115% of the norm','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/7','Not verified','na',NULL),(507,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','QT prolongation (at 200nM)','LQT200p','2','QT prolongation, LQT','CAV12p; HERG4p; KCNQ1p','Predicted QT after the drug administration at 200nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 200nM','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of Luo-Rudy dynamic AP model of guinea pig cardiac cell. The effect of th','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 200nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/6','Verified','% (of normal QT in ms)',NULL),(505,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','QT prolongation (at 200nM)','LQT200','1','QT prolongation; LQT; HERG; multiscale model','CAV12; HERG4; KCNQ1','Predicted QT after the drug administration at 200nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 200nM','N/A',0,'check Related models.','check Related models.',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','\"multiscale model: predicted blockade of CAV12, HERG and KCNQ1 ion channels plus integration using a 1D electrophysiological simulation of ventricular tissue\"','check Related models.','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 200nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/3','Not verified','na',NULL),(504,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','QT prolongation (at 100nM)','LQT100p','2','QT prolongation, LQT','CAV12p; HERG4p; KCNQ1p','Predicted QT after the drug administration at 100nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 100nM','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of Luo-Rudy dynamic AP model of guinea pig cardiac cell. The effect of th','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 100nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/5','Verified','% (of normal QT in ms)',NULL),(502,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','QT prolongation (at 100nM)','LQT100','1','QT prolongation; LQT; HERG; multiscale model','CAV12; HERG4; KCNQ1','Predicted QT after the drug administration at 100nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 100nM','N/A',0,'check Related models.','check Related models.',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','\"multiscale model: predicted blockade of CAV12, HERG and KCNQ1 ion channels plus integration using a 1D electrophysiological simulation of ventricular tissue\"','check Related models.','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 100nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/2','Not verified','na',NULL),(503,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','QT prolongation (at 100nM)','LQT100','2','QT prolongation, LQT','CAV12; HERG4; KCNQ1','Predicted QT after the drug administration at 100nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 100nM','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of Luo-Rudy dynamic AP model of guinea pig cardiac cell. The effect of th','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 100nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/2','Verified','% (of normal QT in ms)',NULL),(221,'2014-09-26 00:00:00','Guillem Plasencia Gallofre','guillemp@leadmolecular.com','Dataset for target FXR (Q96RI1) retrieved from chEMBL version 19. ','SVM Human Farnesoid X receptor agonists','FXR agonists 1 LMD','1','FXR agonist SVM','Mol Biosyst. 2015 May;11(5):1305-18. doi: 10.1039/c5mb00137d','Agonism EC50 versus FXR < 10 micromolar (category \"1\") or >= 10 micromolar (category \"-1\").','Agonist EC50 versus FXR','Dataset for target FXR (Q96RI1) retrieved from chEMBL version 19.',244,'Compounds compiled in chEMBL version 19 from different sources','Dataset for target FXR (Q96RI1) retrieved from chEMBL version 19 was curated by selecting only entries functional assays (AssayType = F) with an EC50 value reported. Entries with an EC50 value < 10 micromolar were assigned to category \"1\", and entries with EC50 >= 10 micromolar to category \"-1\". EC50 values were obtained using many different assay types, thus only a \"general\" classification between strong/weak activators could be done that allowed merging all the EC50 values together.',83.00,-1.0000,1.0000,0.6600,'Yes','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in 2 categories using Volsurf+ descriptors','RBF SVM categRBF SVM categorization based on all 128 Volsurf+ descriptors.','A set of 19 compounds were randomly selected from the orignal dataset and assigned to the test set, never used in the model building step.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',320,73,5,1,NULL,NULL,'The model\'s goodness of fit was assessed using a confusion matrix.','A set of 19 compounds were randomly selected from the orignal dataset and assigned to the test set, never used in the model building step. Not all compounds in the external test set could be predicted (thus the external test size is lower than 18, to reflect the effective size of the external set that could be successfully assigned a prediction)',19,12,5,1,1,NULL,NULL,'The model\'s quality of predictions was assessed using a confusion matrix.',NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or +1','category \"-1\" is assigned to compounds predicted to have an agonist EC50 vs FXR >= 10micromolar, and category \"1\" is assigned to compounds predicted to have agonist EC50 vs FXR < 10microlar.','/ADME/Metabolism/Other/Nuclear receptors/FXR Agonist/1','Verified','na',NULL),(223,'2014-09-18 00:00:00','Guillem Plasencia Gallofre','guillemp@leadmolecular.com','Dataset for KCNH2 (hERG) was downloaded from chEMBL (version 19). Only compounds with reported IC50 were considered.','SVM hERG Inhibition','hERG inhibition 1 LMD','1','SVM hERG Inhibition','J Chem Inf Model. 2012 Jun 25;52(6):1660-73. doi: 10.1021/ci300060b','Compounds IC50 vs hERG < 10 micromolar were assigned to category \"+1\", otherwise to category \"-1\".','IC50 vs hERG below or above 10 micromolar','chEMBL dataset for KCNH2 (hERG) was downloaded from chEMBL (version 19). Only compounds with reported IC50 were considered.',2909,'KCNH2 dataset extracted by EBI  eTOX partner from chEMBL.','Compounds with IC50 vs hERG < 10 micromolar were assigned to category \"+1\", otherwise to category \"-1\".',48.22,-1.0000,1.0000,-0.0300,'No','SDF tag named \"name\"','Details for each compound should be checked by searching it in chEMBL using its ID.','C-SVM in 2 categories using Volsurf+ descriptors','RBF SVM categorization based on all 128 Volsurf+ descriptors. ','A set of 313 randomly selected compounds from the original dataset were assigned to the test set and never used in the model building steps.','Volsurf+ descriptors were described in  Journal of Molecular Structure: THEOCHEM\r\nVolume 503, Issues 1–2, 9 May 2000, Pages 17–30.','All 128 Volsurf+ descriptors were used.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.','In the virtual machine boxed version, using eTOXSys API v. 2, applicability domain is assessed by ADAN.',1247,1292,57,44,NULL,NULL,'The model\'s goodness of fit was assessed using a confusion matrix.','A set of 313 randomly selected compounds from the original dataset were included in the test set. Not all of them could be assigned, thus the effective test set size is 293.',293,103,98,44,48,NULL,NULL,'The model\'s quality of predictions was assessed using a confusion matrix.',NULL,NULL,'N/A','N/A','N/A','N/A','LMD','human','http://etoxsys.eu/current','float','-1 or +1','Compounds with predicted category \"+1\" are expected to have IC50 vs hERG < 10 micromolar. Compounds with predicted category \"-1\" are expected to have an IC50 vs hERG > 10 micromolar.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG Inhibition/1','Verified','na',NULL),(127,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT2B Affinity 1','5HT2B Affinity 1 CT','1','5HT2B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT2B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT2B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2625,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.5000,10.3000,6.6900,'Skewness=-0.12','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6500,0.6500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',464,NULL,NULL,NULL,NULL,0.6400,0.6600,'N/A',0.6500,0.6500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT2B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT2B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(128,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT2C Affinity 1','5HT2C Affinity 1 CT','1','5HT2C, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT2C','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT2C extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4157,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.7000,6.6000,'Skewness=-0.33','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',733,NULL,NULL,NULL,NULL,0.7000,0.6600,'N/A',0.7000,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT2C or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT2C_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(129,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT3A Affinity 1','5HT3A Affinity 1 CT','1','5HT3A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT3A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT3A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1530,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,14.0000,6.6000,'Skewness=-0.21','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7500,0.7900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',272,NULL,NULL,NULL,NULL,0.7600,0.7900,'N/A',0.7500,0.7900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT3A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT3A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(130,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT3B Affinity 1','5HT3B Affinity 1 CT','1','5HT3B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT3B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT3B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',947,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,14.0000,7.0200,'Skewness=-0.34','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.7900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',166,NULL,NULL,NULL,NULL,0.7100,0.7700,'N/A',0.7000,0.7900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT3B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT3B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(131,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT4 Affinity 1','5HT4 Affinity 1 CT','1','5HT4, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT4','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT4 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',704,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,11.0000,7.0700,'Skewness=-0.10','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7700,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',124,NULL,NULL,NULL,NULL,0.7700,0.6700,'N/A',0.7700,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT4 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT4_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(132,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT5A Affinity 1','5HT5A Affinity 1 CT','1','5HT5A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT5A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT5A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',381,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,9.7000,5.4200,'Skewness=-0.06','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8300,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',66,NULL,NULL,NULL,NULL,0.8300,0.6700,'N/A',0.8300,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT5A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT5A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(527,'2016-04-18 00:00:00','Katharine Briggs','Katharine.briggs@lhasalimited.org','[1] Sanderson M & Earnshaw CG (1991). Computer Prediction of Possible Toxic Action from Chemical Structure; The DEREK System. Human and Experimental Toxicology 10, 261-273. doi: 10.1177/096032719101000405 \r\n[2] Judson PN, Marchant CA & Vessey JD (2003). Using Argumentation for Absolute Reasoning about the Potential Toxicity of Chemicals. Journal of Chemical Information and Computer Sciences 43, 1364-1370. doi: 10.1021/ci020272g','Derek Nexus - hepatotoxicity mammal','DILI Mammal 1 LL','2','DILI, drug induced liver injury, hepatotoxicity','DILI general 1 & LB pharmacophore model for DILI in human, non-rodent & rodent','Predicts the likelihood of a structure being hepatotoxic by comparing the query compound to multiple structural alerts (2D SARs)','Term to express the likelihood of a structure being toxic.','References are provided with each alert. The training set underpinning a given alert is proprietary, though generally based on publicly available data:\r\nGreene N, Fisk L, Naven RT, Note RR, Patel ML, Pelletier DJ. Developing structure-activity relationships for the prediction of hepatotoxicity. Chem Res Toxicol. 2010; 23 (7), 1215-22. doi: 10.1021/tx1000865. ',1265,'Large and diverse dataset of human and preclinical hepatotoxicity data for drugs and other chemicals.','A structure searchable database was manually created from the source data & supplemented with additional supporting data in the published literature. Dataset includes 1266 substances but there is no structure available for the antibiotic mixture gramicidin.',100.00,NULL,NULL,NULL,'Yes','CAS No','CAS No, name & structure','Expert system','Expert system based on multiple structure alerts (2D SARs). A structure searchable database was manually created from the source data & supplemented with additional supporting data in the published literature. Suitable classes for investigation were then identified by visual analysis and consideration of therapeutic endpoint. Where sufficient biological and mechanistic data was available then a structural alert was created for that class. Each structural alert was then tested and peer-reviewed before being implemented.','N/A','N/A','N/A','If a chemical matches an alert describing a structure-activity for hepatotoxicity it can be considered to be within the applicability domain. The applicability domain of each alert is defined by the alert developer on the basis of the training set data and expert judgement on the chemical and biological factors which affect the mechanism of action for each alert.','N/A',650,0,0,615,NULL,NULL,'N/A','Liver Toxicity Knowledgebase (LTKB) from FDA available at http://www.fda.gov/ScienceResearch/BioinformaticsTools/LiverToxicityKnowledgeBase/ucm2024036.htm last accessed 14/05/2013. This dataset was chosen as it was publically available and provided the necessary binary calls. Other suitable datasets would have needed further expert interpretation to create these binary calls. LTKB has 287 substances but there is no structure available for the monoclonal antibody gemtuzumab.',286,113,50,15,108,NULL,NULL,'N/A',NULL,NULL,'All alerts describing structure-activity relationships for the hepatotoxicity endpoint have a mechanistic basis wherever possible. Mechanistic information is detailed in the comments associated with an alert and can include information on both the mechanism of action and biological target.\r\nThe mechanistic basis of the model was developed a priori by examining the active and inactive structures before developing the structure-activity relationship.','All references supporting the mechanistic basis of an alert are detailed and available for inspection within the Derek Nexus software.','N/A','N/A','LL','mammal','http://Etoxtest.lhasalimited.org:8080/etoxwsapi/v2/<>','boolean','Probable, Plausible, Equivocal, Doubted, Improbable, Impossible, Nothing to report','Probable = there is at least one strong argument that the proposition is true and there are no arguments against it, Plausible = weight of evidence supports the proposition, Equivocal = equal weight of evidence for and against the proposition, Doubted = weight of evidence opposes the proposition, Improbable = there is at least one strong argument that the proposition is false and there are no arguments that it is true, Impossible = there is proof that the proposition is false & Nothing to report = no prediction provided as no structural alerts fired','/Toxicity/Organ Toxicity/Hepatotoxicity/mammal','Not verified','na',NULL),(506,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Obiol-Pardo C, Gomis-Tena J, Sanz F, Saiz J, Pastor M. A Multiscale Simulation System for the Prediction of Drug-Induced Cardiotoxicity. J Chem Inf Model 2011; 51(2): 483-92. PMID: 21250697','QT prolongation (at 200nM)','LQT200','2','QT prolongation, LQT','CAV12; HERG4; KCNQ1','Predicted QT after the drug administration at 200nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 200nM','This model was not built from a training series and is based on a electrophysiological representation of the ventricular tisue',0,'N/A','N/A',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','multiscale model','The QT was obtained from a surface pseudo-electrocardiogram obtained using a 1D simularion of ventricular tissue. The AP of the individual cells was modeled using a modified version of Luo-Rudy dynamic AP model of guinea pig cardiac cell. The effect of th','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','NA',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 200nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/3','Verified','% (of normal QT in ms)',NULL),(260,'2014-09-17 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','[Broccatelli F. et al. (2011) J. Med. Chem. 54(6):1740-1751]','ABCB1 inhibition classification model','ABCB1 Inhibition 1 UNIVIE','1.0','ATP-binding cassette transporter, P-gp, ABCB1, machine learning, inhibition prediction, ECFP fingerprints','N/A','ABCB1 inhibition classification model','Positive=inhibitor; negative=non-inhibitor','The training set corresponds to the publication \"A Novel Approach for Predicting P-glycoprotein (ABCB1) Inhibition Using Molecular Interaction Fields\" by Broccatelli and colleagues (DOI: 10.1021/jm101421d). It contains initially 1275 compounds extracted from 61 publications. ',1180,'variations around some specific scaffolds','As described in [Pinto et al. (2012) Mol Inf 31:547-553]. Cleaning lead to a final dataset size of 1180.',53.00,0.0000,0.0000,0.0000,'no','common names','N/A','Support Vector Machine','The model was obtained using a parameter-optimized support vector machine as implemented in the scikit-learn library. The parameter grids to be searched are: C: [0.5, 1, 5, 10, 100], and gamma: [1e-4, 1e-3, 0.01, 0.1, 0]. The kernel is an RBF kernel. The GridSearch function was called with an inner 5-fold cross-validation to select the best set of parameters according to the ROC AUC value.\r\n','10-fold cross-validation','ECFP-like descriptors were computed using the RDKit library. The diameter was set to 4 and the bits were folded to a final size of 1024.','None','ADAN as implemented in eTOXlab','ADAN as implemented in eTOXlab',591,538,15,36,0.0000,0.0000,'N/A','10 fold CV results are given below.',1180,538,495,58,89,0.0000,0.0000,'Sensitivity=0.86, Specificity=0.90, MCC=0.75',0.0000,0.0000,'The model predicts if a query compound will be or not an inhibitor of ABCB1.','N/A','N/A','N/A','UNIVIE','human','N/A','boolean','negative, positive','The model predicts if a query compound will be or not an inhibitor of ABCB1 (P-glycoprotein). If \"negative\" is obtained, then the compound is predicted as non inhibitor of ABCB1. If \"positive\" is obtained, then the compound is predicted as inhibitor of ABCB1.','/ADME/Transport/Transporters/ABCB1 Inhibition/1','Verified','na',NULL),(147,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRB3 Affinity 1','ADRB3 Affinity 1 CT','1','ADRB3, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRB3','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRB3 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1552,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.5000,7.0800,'Skewness=-0.43','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7200,0.6100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',275,NULL,NULL,NULL,NULL,0.7200,0.6100,'N/A',0.7200,0.6100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRB3 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRB3_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(138,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','AchE Affinity 1','AchE Affinity 1 CT','1','AchE, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over AchE','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to AchE extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4263,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.5000,5.8200,'Skewness=-0.05','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8000,0.7000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',750,NULL,NULL,NULL,NULL,0.7900,0.7000,'N/A',0.8000,0.7000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over AchE or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/AchE_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(148,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ALB Affinity 1','ALB Affinity 1 CT','1','ALB, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ALB','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ALB extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',187,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.7000,7.0000,4.2100,'Skewness=1.21','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5600,0.3700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',33,NULL,NULL,NULL,NULL,0.5300,0.3600,'N/A',0.5600,0.3700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ALB or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ALB_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(149,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ALOX15 Affinity 1','ALOX15 Affinity 1 CT','1','ALOX15, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ALOX15','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ALOX15 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',349,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,9.0000,5.8600,'Skewness=-0.01','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.8100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',61,NULL,NULL,NULL,NULL,0.7300,0.7700,'N/A',0.7100,0.8100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ALOX15 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ALOX15_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(141,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRA1D Affinity 1','ADRA1D Affinity 1 CT','1','ADRA1D, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRA1D','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRA1D extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4289,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.3000,6.7000,'Skewness=-0.24','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7200,0.6500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',757,NULL,NULL,NULL,NULL,0.7300,0.6500,'N/A',0.7200,0.6500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRA1D or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRA1D_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(142,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRA2A Affinity 1','ADRA2A Affinity 1 CT','1','ADRA2A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRA2A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRA2A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2242,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.8000,6.3900,'Skewness=-0.23','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7400,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',397,NULL,NULL,NULL,NULL,0.7500,0.6600,'N/A',0.7400,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRA2A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRA2A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(140,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRA1B Affinity 1','ADRA1B Affinity 1 CT','1','ADRA1B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRA1B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRA1B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4315,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.3000,6.5700,'Skewness=-0.20','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.6500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',761,NULL,NULL,NULL,NULL,0.7100,0.6500,'N/A',0.7000,0.6500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRA1B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRA1B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(139,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRA1A Affinity 1','ADRA1A Affinity 1 CT','1','ADRA1A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRA1A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRA1A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4782,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.3000,6.9000,'Skewness=-0.22','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7300,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',846,NULL,NULL,NULL,NULL,0.7300,0.6700,'N/A',0.7300,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRA1A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRA1A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(146,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRB2 Affinity 1','ADRB2 Affinity 1 CT','1','ADRB2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRB2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRB2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1946,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.5000,10.9000,6.3200,'Skewness=0.23','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7700,0.7200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',343,NULL,NULL,NULL,NULL,0.7600,0.7100,'N/A',0.7700,0.7200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRB2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRB2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(145,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRB1 Affinity 1','ADRB1 Affinity 1 CT','1','ADRB1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRB1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRB1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1897,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.8000,6.1800,'Skewness=0.13','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6700,0.6500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',338,NULL,NULL,NULL,NULL,0.6900,0.6400,'N/A',0.6700,0.6500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRB1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRB1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(144,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRA2C Affinity 1','ADRA2C Affinity 1 CT','1','ADRA2C, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRA2C','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRA2C extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1902,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.8000,6.5400,'Skewness=-0.33','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7300,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',336,NULL,NULL,NULL,NULL,0.7200,0.6900,'N/A',0.7300,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRA2C or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRA2C_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(143,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ADRA2B Affinity 1','ADRA2B Affinity 1 CT','1','ADRA2B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ADRA2B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ADRA2B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1845,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.8000,6.3700,'Skewness=-0.29','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7300,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',327,NULL,NULL,NULL,NULL,0.7400,0.6600,'N/A',0.7300,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ADRA2B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ADRA2B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(136,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ACE Affinity 1','ACE Affinity 1 CT','1','ACE, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ACE','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ACE extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1361,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.6000,6.5600,'Skewness=-0.51','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7600,0.7900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',241,NULL,NULL,NULL,NULL,0.7600,0.7800,'N/A',0.7600,0.7900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ACE or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ACE_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(135,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ABCG2 Affinity 1','ABCG2 Affinity 1 CT','1','ABCG2, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over ABCG2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ABCG2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',202,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,7.8000,5.3200,'Skewness=-0.22','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6500,0.4500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',36,NULL,NULL,NULL,NULL,0.7100,0.4300,'N/A',0.6500,0.4500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ABCG2 or none if compound is outside applicability domain','/ADME/Transport/Transporters/ABCG2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(1514,'2016-05-12 20:44:07','Daan Geerke','d.p.geerke@vu.nl','In preparation','LIE prediction of binding affinity to CYP450 2E1','CYP 2E1 Affinity 1 VUA','3','CYP 2E1, CP2E1, CP2E1_HUMAN, Cytochrome P450 2E1, binding affinity, free energies of binding, IC50s','CYP 1A2 Affinity 2 VUA, CYP 2D6 Affinity 2 VUA','CYP 2E1 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol)','Inhibition constants K_i were taken from [Yano et al. \"Synthetic Inhibitors of Cytochrome P-450 2A6: Inhibitory Activity, Difference Spectra, Mechanism of Inhibition, and Protein Cocrystallization.\" J. Med. Chem. 2006, 49, 6987-7001.]\r\nand from [Denton et al. \"5-Substituted, 6-Substituted, and Unsubstituted 3-Heteroaromatic Pyridine Analogues of Nicotine as Selective Inhibitors of Cytochrome P-450 2A6.\" J. Med. Chem. 2005, 48, 224-239.]',16,'A structurally homogeneous set of 2E1 inhibitors wtih nicotin scaffold.','See \"Training set source\". Inhibition constants K_i were converted to DG using DG = RT ln K_i',NULL,-42.0300,-24.5400,-31.3600,'no','CHEMBL IDs; Training set: 215174, 214784, 179399, 360541, 368883, 360999, 178780, 178781, 14859, 217125, 179398, 178534, 179704, 179005, 386250, 410355; External test set: 386124, 214990, 179669, 178516, 179621, 179529, 179618, 425519','Only for compounds for which IC50s were taken from [Denton et al] (CHEMBLID=179399, 361364, 178780, 178781, 179398, 179529, 178534, 425519, 179005, 179618), the data source explicitly reports compound purities (>95% in those cases)','LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using linear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverables D3.5 and D3.13).','Inhibition data were used for 14 compounds taken from Yano et al and from Denton et al (see \"Training set source\" for reference details), which were not in the training set. External validation data (\"External val Q2\" and \"External val SDEP\") are reported based on the results for the 8 test compounds which fulfilled the applicability domain assessment (see \"AD assessment\").','DVele and DVvdw. Using LIE, binding free energies (affinities) are calculated from the average interaction energies (DVele and DVvdw) of the compound with its surroundings as obtained from MD simulations (see also deliverables D3.5 and D3.13).','N/A (cf. \"Descriptors definition\")','2E1 inhibitors with nicotin scaffold were included in the training and test set. Compounds vary in their molecular weight and the presence of functional groups.','AD criteria should be fulfilled, which are determined from (1) a Mahalanobis-distance analysis on measured simulation properties, (2) ligand-fingerprint similarities, (3) and (4) two residue-based protein-ligand interaction patterns in simulation (see D3.13 and L. Capoferri et al., PLOS ONE 2015, 10, e0142232), and (5) spread in experimental data.',NULL,NULL,NULL,NULL,0.5300,3.1600,'N/A','K_i\'s were taken from Yano et al and from Denton et al (cf. \"Training set source\")\r\n',8,NULL,NULL,NULL,NULL,0.0730,5.7600,NULL,0.5100,3.4400,'A molecular-dynamics (MD) based LIE model (with alpha=0.690 and beta=0.169) is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 2E1.','See \"Mechanistic basis of the model\". See [L. Capoferri et al., PLOS ONE 2015, 10, e0142232] for details on the Applicability Domain assessment. ','LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034',NULL,'VUA','human','http://etox.labs.vu.nl/etoxwsapi/v2','float','(-inf,inf)','Binding free energies (affinities) DG can be translated into inhibition constants K_i in M by using DG = RT ln K_i (with R: gas constant and T: temperature), and into IC50s in M at [S] = K_m using K_i = IC50. The more negative DG is for a compound, the stronger it binds to CYP 2E1.','/ADME/Metabolism/Phase I/CYP 2E1 Affinity/1','Verification ongoing','kJ/mol',NULL),(1515,'2016-05-12 20:44:07','Daan Geerke','d.p.geerke@vu.nl','In preparation','LIE prediction of binding affinity to CYP450 19A1','CYP 19A1 Affinity 1 VUA','1','CYP 19A1, CP19A1, CP19A1_HUMAN, Cytochrome P450 19A1, binding affinity, free energies of binding, IC50s','CYP 1A2 Affinity 2 VUA, CYP 2D6 Affinity 2 VUA, CYP 2E1 Affinity 2 VUA','CYP 19A1 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol)','Inhibition constants K_i were taken from Numuzawa et al. (J Med Chem 1994, 37, 2198; Biochem Biophys Res Commun 1989, 160, 1009;  Biochem Biophys Res Commun 1991, 177, 401; J Med Chem 1991, 34, 2496; J Med Chem 1994, 37, 1312) and converted to DG using DG = RT ln K_i',14,'A structurally homogeneous set of 19A1 binders with steroid scaffold.','See \"Training set source\". Inhibition constants K_i were converted to DG using DG = RT ln K_i',NULL,-13.4400,0.0000,-7.8900,'no','CHEMBLIDs','N/A','LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using linear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverables D3.5 and D3.13).','Inhibition data were used for 6 compounds taken from Numuzawa et al (see \"Training set source\"), which were not in the training set.','DVele and DVvdw. Using LIE, binding free energies (affinities) are calculated from the average interaction energies (DVele and DVvdw) of the compound with its surroundings as obtained from MD simulations (see also deliverables D3.5 and D3.13).','N/A (cf. \"Descriptors definition\")','19A1 inhibitors with steroid scaffold were included in the training and test set. Compounds vary in their molecular weight and the presence of functional groups.','3 or more AD criteria should be fulfilled, which are determined from (1) a Mahalanobis-distance analysis on measured simulation properties, (2) ligand-fingerprint similarities, (3) and (4) two residue-based protein-ligand interaction patterns in simulation (see D3.13 and L. Capoferri et al., PLOS ONE 2015, 10, e0142232), and (5) spread in experimental data.',NULL,NULL,NULL,NULL,0.7600,1.6300,'N/A','K_i\'s, converted to binding free energies using DG = RT ln K_i were taken from Numuzawa et al (see \"Training set source\")\r\n',6,NULL,NULL,NULL,NULL,0.0690,2.6200,NULL,0.7000,1.9800,'A molecular-dynamics (MD) based LIE model (with alpha=0.169 and beta=0.266) is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 19A1.','See \"Mechanistic basis of the model\". See [L. Capoferri et al., PLOS ONE 2015, 10, e0142232] for details on the Applicability Domain assessment. ','LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034',NULL,'VUA','human','http://etox.labs.vu.nl/etoxwsapi/v2','float','(-inf,inf)','Binding free energies (affinities) DG can be translated into inhibition constants K_i in M by using DG = RT ln K_i (with R: gas constant and T: temperature), and into IC50s in M at [S] = K_m using K_i = IC50/2. The more negative DG is for a compound, the stronger it binds to CYP 19A1.','/ADME/Metabolism/Phase I/CYP 19A1 Affinity/1','Verification ongoing','kJ/mol',NULL),(137,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ACE2 Affinity 1','ACE2 Affinity 1 CT','1','ACE2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ACE2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ACE2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',175,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.8000,9.9000,6.8900,'Skewness=-0.42','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6800,0.7700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',29,NULL,NULL,NULL,NULL,0.7000,0.7500,'N/A',0.6800,0.7700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ACE2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ACE2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(134,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT7 Affinity 1','5HT7 Affinity 1 CT','1','5HT7, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT7','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT7 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1357,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.5000,9.9000,6.7000,'Skewness=-0.39','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6500,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',239,NULL,NULL,NULL,NULL,0.6500,0.6600,'N/A',0.6500,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT7 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT7_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(525,'2015-12-17 00:00:00','Gerhard F. Ecker','gerhard.f.ecker@univie.ac.at','Liu Z, Shi Q, Ding D, Kelly R, Fang H, Tong W (2011) Translating clinical findings into knowledge in drug safety evaluation--drug induced liver injury prediction system &#40;DILIps&#41; PLoS Comput Biol 7:e1002310 doi:10.1371/journal.pcbi.1002310','Hyperbilirubinemia','HYPERBIL UNIVIE','1.0','hypebilirubinemia, bilirubin','N/A','Prediction whether a compound causes hyperbilirubinemia (= increase of conjugated or unconjugated bilirubin in sinusoidal blood)','positive = causing hyperbilirubinemia, negative = not causing hyperbilirubinemia','Liu Z, Shi Q, Ding D, Kelly R, Fang H, Tong W (2011) PLoS Comput Biol 7:e1002310 doi:10.1371/journal.pcbi.1002310',835,'Mostly drugs, public domain','Removal of inorganic compounds, salt parts, compounds containing rare or special elements, normalization of chemotypes, elimination of duplicates, removal of compounds containing permanent charges,energy minimization of the 3D structures (the dataset was from the beginning in a 3D form)',9.70,0.0000,0.0000,0.0000,'yes (highly imbalanced dataset)','chemical names or drug generic names','N/A','SVM','SMO as implemented in WEKA (http://www.cs.waikato.ac.nz/ml/weka/), using the RBF kernel, \"Build Logistic models\" option set to \"True\" and the rest settings at default, in combination with MetaCost meta-classifier in WEKA, using as cost-matrix \"[0.0, 1.0; 10.0, 0.0]\" and AttributeSelectedClassifier (SignificanceAttributeEval  + Ranker)','10-fold cross-validation','ECFP6 (1024 binary fingerprints)','None','ADAN as implemented in eTOX lab','ADAN as implemented in eTOX lab',78,551,198,8,0.0000,0.0000,'N/A','10-fold cross-validation results are given below',835,58,509,240,28,0.0000,0.0000,'Sensitivity=0.67, Specificity=0.68, MCC=0.23',0.0000,0.0000,'The model predicts if a query compound will be or not be able to cause hyperbilirubinemia. ','N/A','N/A','Liu Z, Shi Q, Ding D, Kelly R, Fang H, Tong W (2011) PLoS Comput Biol 7:e1002310 doi:10.1371/journal.pcbi.1002310','UNIVIE','human','http://131.130.129.209/etoxwsapi/v2/info','boolean','negative-positive','The model predicts if a query compound will be or not be able to cause hyperbilirubinemia. If \"negative\" is obtained, the compound is predicted as not causing hyperbilirubinemia. If \"positive\" is obtained, the compound is predicted as causing hyperbilirubinemia.','/Toxicity/Target Safety Pharmacology/Hyperbilirubinemia/1\r\n','Not verified','na',NULL),(499,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','QT prolongation (at 10nM)','LQT10','1','QT prolongation; LQT; HERG; multiscale model','CAV12; HERG4; KCNQ1','Predicted QT after the drug administration at 10nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','Predicted QT expresed as % of the normal QT, for a drug concentration at the receptors site of 10nM','N/A',0,'check Related models.','check Related models.',0.00,0.0000,0.0000,0.0000,'N/A','N/A','N/A','\"multiscale model: predicted blockade of CAV12, HERG and KCNQ1 ion channels plus integration using a 1D electrophysiological simulation of ventricular tissue\"','check Related models.','N/A','N/A','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(0,1000)','Predicted QT after the drug administration at 10nM, expresed as % of normal QT (e.g. 110 value means that after the compound has been administered for obtaining a 100nM conc. the QT is expected to be 10% higher than the normal value)','/Toxicity/Organ Toxicity/Cardiotoxicity/QT Prolongation/1','Not verified','na',NULL),(122,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT1A Affinity 1','5HT1A Affinity 1 CT','1','5HT1A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT1A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT1A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',6380,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,12.8000,7.0700,'Skewness=-0.39','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.6400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',1128,NULL,NULL,NULL,NULL,0.6500,0.6400,'N/A',0.6400,0.6400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT1A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT1A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(123,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT1B Affinity 1','5HT1B Affinity 1 CT','1','5HT1B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT1B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT1B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1641,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.0000,6.5500,'Skewness=-0.22','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7400,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',289,NULL,NULL,NULL,NULL,0.7400,0.6700,'N/A',0.7400,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT1B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT1B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(124,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT1D Affinity 1','5HT1D Affinity 1 CT','1','5HT1D, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT1D','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT1D extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1555,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.7000,7.1200,'Skewness=-0.40','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8200,0.6400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',276,NULL,NULL,NULL,NULL,0.8200,0.6400,'N/A',0.8200,0.6400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT1D or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT1D_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(125,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT1F Affinity 1','5HT1F Affinity 1 CT','1','5HT1F, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT1F','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT1F extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',437,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,9.3000,6.3900,'Skewness=-0.20','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8000,0.5600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',79,NULL,NULL,NULL,NULL,0.7900,0.5700,'N/A',0.8000,0.5600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT1F or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT1F_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(126,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','5HT2A Affinity 1','5HT2A Affinity 1 CT','1','5HT2A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over 5HT2A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to 5HT2A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',5398,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.7000,6.7000,'Skewness=-0.11','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6700,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',951,NULL,NULL,NULL,NULL,0.6800,0.6500,'N/A',0.6700,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over 5HT2A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/5HT2A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(524,'2016-02-22 12:00:19','Daan Geerke','d.p.geerke@vu.nl','C. R. Vosmeer, R. Pool, M. F. van Stee, L. Perić-Hassler, N. P. E. Vermeulen and D. P. Geerke. \"Towards automated binding affinity prediction using an iterative linear energy approach.\" Int. J. Mol. Sci. 2014, 15, 798-816. DOI:10.3390/ijms15010798','LIE prediction of binding affinity to CYP450 2D6','CYP 2D6 Affinity 2 VUA','1','CYP 2D6, CP2D6, CP2D6_HUMAN, Cytochrome P450 2D6, binding affinity, free energies of binding, IC50s','CYP 1A2 Affinity 2 VUA','CYP 2D6 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5) from [Vaz et al. \"A 3D-QSAR model for CYP2D6 inhibition in the aryloxypropanolamine series.\" Bioorg. Med. Chem. Lett. 2005, 15, 3816-3820.]',8,'A structurally homogeneous set of compounds (aryloxypropanolamines).','8 training compounds were selected from Vaz et al. (see \"Training set source\") to obtain a wide spread in experimental data and to represent a large spread in heterogeneity of the compounds. Note that this training set included 1 compound less than in the original Vosmeer paper, because of issues in assigning model (force field) parameters using the Acpype freeware.',NULL,-48.2100,-27.3100,-36.3800,'no','IDs were assigned according to the IDs used by Vosmeer et al.','Compound purities were not reported by Vaz et al.','LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using linear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverable D3.5).','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5) from [Vaz et al. \"A 3D-QSAR model for CYP2D6 inhibition in the aryloxypropanolamine series.\" Bioorg. Med. Chem. Lett. 2005, 15, 3816-3820.] 7 test compounds were selected. Note that this test set included 1 compound less than in the original Vosmeer paper, because of issues in assigning model (force field) parameters using the Acpype freeware.','DVele and DVvdw. Using LIE, binding free energies (affinities) are calculated from the average interaction energies (DVele and DVvdw) of the compound with its surroundings as obtained from MD simulations (see also deliverables D3.5 and D3.13).','N/A (cf. \"Descriptors definition\")','Model is calibrated based on aryloxypropanolamine compounds.','At least 2 AD criteria should be fulfilled, which are determined from (1) a Mahalanobis-distance analysis on measured simulation properties, (2) ligand-fingerprint similarities, (3) and (4) two residue-based protein-ligand interaction patterns in simulation (see D3.13, and Capoferri et al., PLOS ONE 10, e0142232), and (5) spread in experimental data.',NULL,NULL,NULL,NULL,0.9100,2.0700,'N/A','See \"Test set selection\"',7,NULL,NULL,NULL,NULL,-0.9900,7.8200,NULL,0.6600,3.9800,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 2D6.','See \"Mechanistic basis of the model\", and see eTOX deliverable D3.13 and [Capoferri et al., PLOS ONE 10, e0142232] for details on the Applicability Domain assessment. Note that model statistics are slightly different here than in the original IJMS paper of Vosmeer et al. because of the use of other docking and MD software than reported in the manuscript (for licensing reasons, here we use the ParaDocks and GROMACS freeware) and because of the use of Acpype freeware to obtain force field parameters. Use of the implemented protocol resulted in an alpha and beta value of 0.44 and 0.16 in the central LIE equation, respectively.','LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034; L. Perić-Hassler, E. Stjernschantz, C. Oostenbrink and D. P. Geerke. Int. J. Mol. Sci. 2013, 14, 24514-24530. DOI:10.3390/ijms141224514',NULL,'VUA','human','http://etox.labs.vu.nl/etoxws/v2','float','(-inf,inf)','Binding free energies (affinities) DG can be translated into inhibition constants K_i in M by using DG = RT ln K_i (with R: gas constant and T: temperature), and into IC50s in M at [S] = K_m using K_i = IC50. The more negative DG is for a compound, the stronger it binds to CYP 2D6.','/ADME/Metabolism/Phase I/CYP 2D6 Affinity/2','Verified','kJ/mol',NULL),(522,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','RAR Retinoic acid receptor alpha activation','RARp','2','RAR, Retinoic acid receptor alpha, RAR-alpha, Nuclear receptor subfamily 1 group B member 1','N/A','Activation of receptor for retinoic acid (RAR). Retinoic acid receptors bind as heterodimers to their target response elements in response to their ligands: all-trans or 9-cis retinoic acid, and regulate gene expression in various biological processes','Quantitative. It represents activation of the receptor.','Collection ligands annotated to RAR from ChEMBL database.',96,'public domain. Small molecules. (MolWeight data: Min. 277.1;1st Qu. 330..2;Median 352.1;Mean 361.0;3rd Qu. 377.5;Max. 593.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,5.0000,8.3980,5.9000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.4100,0.7000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the activation for RAR Receptor.','/Toxicity/Target Safety Pharmacology/RAR activation/2','Verified','M (EC50 in minus log scale)',NULL),(514,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','NAV1.5 Sodium channel protein type 5 subunit alpha blockade','NAV15','1','NAV1.5; Sodium channel protein type 5 subunit alpha','N/A','Blockade of voltage-gated sodium channel (Nav1.5)','Quantitative. It represents inhibition (blockade) of the receptor.','Collection ligands annotated to NAV1.5 from ChEMBL database.',339,'public domain. Small Molecules. (MolWeight data: Min. 278.1;1st Qu. 331.2;Median 352.7;Mean 361.8;3rd Qu. 378.5;Max. 593.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,3.5230,7.6990,5.3120,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5430,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.4990,0.5460,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for NAV15 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/Nav15 inhibition/1','Not verified','na',NULL),(515,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','NAV1.5 Sodium channel protein type 5 subunit alpha blockade','NAV15','2','NAV1.5, Sodium channel protein type 5 subunit alpha, Sodium channel protein type V subunit alpha, Voltage-gated sodium channel subunit alpha Nav1.5','N/A','Blockade of voltage-gated sodium channel (Nav1.5)','Quantitative. It represents inhibition (blockade) of the receptor.','Collection ligands annotated to NAV1.5 from ChEMBL database.',339,'public domain. Small Molecules. (MolWeight data: Min. 278.1;1st Qu. 331.2;Median 352.7;Mean 361.8;3rd Qu. 378.5;Max. 593.1','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,3.5230,7.6990,5.3120,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 3.494 Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6300,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.5600,0.5100,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for NAV15 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/Nav15 inhibition/1','Verified','M (IC50 in minus log scale)',NULL),(521,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','RAR Retinoic acid receptor alpha activation','RAR','2','RAR, Retinoic acid receptor alpha, RAR-alpha, Nuclear receptor subfamily 1 group B member 1','N/A','Activation of receptor for retinoic acid (RAR). Retinoic acid receptors bind as heterodimers to their target response elements in response to their ligands: all-trans or 9-cis retinoic acid, and regulate gene expression in various biological processes','Quantitative. It represents activation of the receptor.','Collection ligands annotated to RAR from ChEMBL database.',96,'public domain. Small molecules. (MolWeight data: Min. 277.1;1st Qu. 330..2;Median 352.1;Mean 361.0;3rd Qu. 377.5;Max. 593.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,5.0000,8.3980,5.9000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5700,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.4400,0.6900,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the activation for RAR Receptor.','/Toxicity/Target Safety Pharmacology/RAR activation/1','Verified','M (EC50 in minus log scale)',NULL),(520,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','RAR Retinoic acid receptor alpha activation','RAR','1','RAR; Retinoic acid receptor alpha','N/A','Activation of receptor for retinoic acid (RAR). Retinoic acid receptors bind as heterodimers to their target response elements in response to their ligands: all-trans or 9-cis retinoic acid, and regulate gene expression in various biological processes','Quantitative. It represents activation of the receptor.','Collection ligands annotated to RAR from ChEMBL database.',96,'public domain. Small molecules. (MolWeight data: Min. 277.1;1st Qu. 330..2;Median 352.1;Mean 361.0;3rd Qu. 377.5;Max. 593.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,5.0000,8.3980,5.9000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5330,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.4090,0.7050,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the activation for RAR Receptor.','/Toxicity/Target Safety Pharmacology/RAR activation/1','Not verified','na',NULL),(519,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','PXR Nuclear receptor subfamily 1 group I member 2 activation','PXRp','2','PXR, Nuclear receptor subfamily 1 group I member 2, Orphan nuclear receptor PAR1, Orphan nuclear receptor PXR, Pregnane X receptor, Steroid and xenobiotic receptor, SXR','N/A','Activation of PXR nuclear receptor. PXR is a transcription factor that activates the transcription of multiple genes involved in the metabolism and secretion of potentially harmful xenobiotics, drugs and endogenous compounds.','Binary. It represents activation of the receptor.','Yoshida et al. doi=10.2133/dmpk.DMPK-11-RG-159',507,'public domain. Small molecules. (MolWeight data: Min. 78.01;1st Qu. 282.4;Median 334.3;Mean 360.3;3rd Qu. 410.9;Max. 1202)','No extra data curation was done',0.52,0.0000,1.0000,0.5220,'NO','compound name','small molecules','PLS-DA and 3D QSAR','Training set 2D structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a PLS DA with 4 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',194,180,62,71,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Positive indicates activation of human PXR.','/Toxicity/Target Safety Pharmacology/PXR activation/2','Verified','na',NULL),(518,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','PXR Nuclear receptor subfamily 1 group I member 2 activation','PXR','2','PXR, Nuclear receptor subfamily 1 group I member 2, Orphan nuclear receptor PAR1, Orphan nuclear receptor PXR, Pregnane X receptor, Steroid and xenobiotic receptor, SXR','N/A','Activation of PXR nuclear receptor. PXR is a transcription factor that activates the transcription of multiple genes involved in the metabolism and secretion of potentially harmful xenobiotics, drugs and endogenous compounds.','Binary. It represents activation of the receptor.','Yoshida et al. doi=10.2133/dmpk.DMPK-11-RG-159',508,'public domain. Small molecules. (MolWeight data: Min. 78.01;1st Qu. 282.4;Median 334.3;Mean 360.3;3rd Qu. 410.9;Max. 1202)','No extra data curation was done',0.52,0.0000,1.0000,0.5220,'NO','compound name','small molecules','PLS-DA and 3D QSAR','Training set 2D structures were converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS DA with 4 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',202,183,59,64,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Positive indicates activation of human PXR.','/Toxicity/Target Safety Pharmacology/PXR activation/1','Verified','na',NULL),(517,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','PXR Nuclear receptor subfamily 1 group I member 2 activation','PXR','1','PXR; Nuclear receptor subfamily 1 group I member 2','N/A','Activation of PXR nuclear receptor. PXR is a transcription factor that activates the transcription of multiple genes involved in the metabolism and secretion of potentially harmful xenobiotics, drugs and endogenous compounds.','Binary. It represents activation of the receptor.','Yoshida et al. doi=10.2133/dmpk.DMPK-11-RG-159',508,'public domain. Small molecules. (MolWeight data: Min. 78.01;1st Qu. 282.4;Median 334.3;Mean 360.3;3rd Qu. 410.9;Max. 1202)','No extra data curation was done',0.52,0.0000,1.0000,0.5220,'NO','compound name','small molecules','PLS-DA and 3D QSAR','Training set 2D structures were converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS DA with 4LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',195,177,64,71,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Positive indicates activation of human PXR.','/Toxicity/Target Safety Pharmacology/PXR activation/1','Not verified','na',NULL),(516,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','NAV1.5 Sodium channel protein type 5 subunit alpha blockade','NAV15p','2','NAV1.5, Sodium channel protein type 5 subunit alpha, Sodium channel protein type V subunit alpha, Voltage-gated sodium channel subunit alpha Nav1.5','N/A','Blockade of voltage-gated sodium channel (Nav1.5)','Quantitative. It represents inhibition (blockade) of the receptor.','Collection ligands annotated to NAV1.5 from ChEMBL database.',339,'public domain. Small Molecules. (MolWeight data: Min. 278.1;1st Qu. 331.2;Median 352.7;Mean 361.8;3rd Qu. 378.5;Max. 593.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,3.5230,7.6990,5.3120,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6500,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.5500,0.5200,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for NAV15 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/Nav15 inhibition/2','Verified','M (IC50 in minus log scale)',NULL),(478,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','CAV1.2 Voltage-dependent L-type calcium channel subunit alpha-1C blockade','CAV12','1','CAV1.2; Voltage-dependent L-type calcium channel subunit alpha-1C','N/A','Blockade of voltage-sensitive calcium channels (Cav1.2): Cav1.2 mediates the entry of calcium ions into excitable cells and are also involved in a variety of calcium-dependent. processes, including muscle contraction, hormone or neurotransmitter release, gene expression, cell motility, cell division and cell death. Findings in variety of animal species ( rats, dogs, mice, monkeys, and humans )','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to CAV1.2 from ChEMBL database.',108,'public domain. Small molecules. (MolWeight data: Min. 310.2;1st Qu. 376.9;Median 406.1;Mean 418.04;3rd Qu. 457.4;Max. 572.3)','CHEMBL Extracted data was filtered with standard units == ”nM” and activity type == ”IC50”',0.00,4.5230,9.0000,5.8190,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were  converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode\r\n','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.7230,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.5970,0.8750,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for CAV12 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/Cav1.2 inhibition/1','Not verified','na',NULL),(479,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','CAV1.2 Voltage-dependent L-type calcium channel subunit alpha-1C blockade','CAV12','2','CAV1.2, Voltage-dependent L-type calcium channel subunit alpha-1C, Calcium channel L type, Voltage-gated calcium channel subunit alpha Cav1.2','N/A','Blockade of voltage-sensitive calcium channels (Cav1.2): Cav1.2 mediates the entry of calcium ions into excitable cells and are also involved in a variety of calcium-dependent. processes, including muscle contraction, hormone or neurotransmitter release, gene expression, cell motility, cell division and cell death. Findings in variety of animal species ( rats, dogs, mice, monkeys, and humans )','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to CAV1.2 from ChEMBL database.',108,'public domain. Small molecules. (MolWeight data: Min. 310.2;1st Qu. 376.9;Median 406.1;Mean 418.04;3rd Qu. 457.4;Max. 572.3)','CHEMBL Extracted data was filtered with standard units == ”nM” and activity type == ”IC50”',0.00,4.5230,9.0000,5.8190,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were  converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.8100,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.7600,0.6700,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for CAV12 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/Cav1.2 inhibition/1','Verified','M (IC50 in minus log scale)',NULL),(480,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','CAV1.2 Voltage-dependent L-type calcium channel subunit alpha-1C blockade','CAV12p','2','CAV1.2, Voltage-dependent L-type calcium channel subunit alpha-1C, Calcium channel L type, Voltage-gated calcium channel subunit alpha Cav1.2','N/A','Blockade of voltage-sensitive calcium channels (Cav1.2): Cav1.2 mediates the entry of calcium ions into excitable cells and are also involved in a variety of calcium-dependent. processes, including muscle contraction, hormone or neurotransmitter release, gene expression, cell motility, cell division and cell death. Findings in variety of animal species ( rats, dogs, mice, monkeys, and humans )','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to CAV1.2 from ChEMBL database.',108,'public domain. Small molecules. (MolWeight data: Min. 310.2;1st Qu. 376.9;Median 406.1;Mean 418.04;3rd Qu. 457.4;Max. 572.3)','CHEMBL Extracted data was filtered with standard units == ”nM” and activity type == ”IC50”',0.00,4.5230,9.0000,5.8190,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.7300,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.6500,0.8100,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for CAV12 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/Cav1.2 inhibition/2','Verified','M (IC50 in minus log scale)',NULL),(481,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','Drug-induced Phospholipidosis QSAR-based','DIPL1','1','Phospholipidosis; DIPL; Drug-induced phospholipidosis','N/A','Phospholipidosis findings in variety of animal species (rats, dogs, mice, monkeys, and humans).','Binary. It shows existence of multiple vacuoles filled with phospholipids that show on electron microscopic examination or based on the presence of foamy macrophages and vacuolations.','The source is the curated data set by Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27. doi: 10.1080/15376510701857262.',460,'Small molecules. (MolWeight data: Min. 32.045 1st Qu. 235.33 Median  294.43 Mean 299.19 3rd Qu.  361.391  Max. 810.974)','Commercial compounds were removed. Erythromycin was duplicate with different activities and was removed. The original dependent values were mapped to 0 when 10  and to 1 when 35.',19.70,0.0000,1.0000,0.2100,'NO','compound name','small molecules','Decision tree plus PLS-DA and 3D QSAR','Compounds with charge less than 1 are negative. Otherwise, compounds are converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode\r\n','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',280,110,39,31,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Compounds predicted to be positive are likely to produce phospholipidosis.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/1','Not verified','na',NULL),(482,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','Drug-induced Phospholipidosis QSAR-based','DIPL1','2','Phospholipidosis, DIPL, Drug-induced phospholipidosis','N/A','Phospholipidosis findings in variety of animal species (rats, dogs, mice, monkeys, and humans).','Binary. It shows existence of multiple vacuoles filled with phospholipids that show on electron microscopic examination or based on the presence of foamy macrophages and vacuolations.','The source is the curated data set by Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27. doi: 10.1080/15376510701857262.',464,'Small molecules. (MolWeight data: Min. 32.045 1st Qu. 235.33 Median  294.43 Mean 299.19 3rd Qu.  361.391  Max. 810.974)','Commercial compounds were removed. Erythromycin was duplicate with different activities and was removed. The original dependent values were mapped to 0 when 10  and to 1 when 35.',19.70,0.0000,1.0000,0.2100,'NO','compound name','small molecules','Decision tree plus PLS-DA and 3D QSAR','Compounds with charge less than 1 are negative. Otherwise, compounds are converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',75,285,82,22,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Compounds predicted to be positive are likely to produce phospholipidosis.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/1','Verified','na',NULL),(483,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','Drug-induced Phospholipidosis QSAR-based','DIPL1p','2','Phospholipidosis, DIPL, Drug-induced phospholipidosis','N/A','Phospholipidosis findings in variety of animal species (rats, dogs, mice, monkeys, and humans).','Binary. It shows existence of multiple vacuoles filled with phospholipids that show on electron microscopic examination or based on the presence of foamy macrophages and vacuolations.','The source is the curated data set by Kruhlak et al. Toxicol Mech Methods. 2008;18(2-3):217-27. doi: 10.1080/15376510701857262.',459,'Small molecules. (MolWeight data: Min. 32.045 1st Qu. 235.33 Median  294.43 Mean 299.19 3rd Qu.  361.391  Max. 810.974)','Commercial compounds were removed. Erythromycin was duplicate with different activities and was removed. The original dependent values were mapped to 0 when 10  and to 1 when 35.',19.70,0.0000,1.0000,0.2100,'NO','compound name','small molecules','Decision tree plus PLS-DA and 3D QSAR','Training set 2D structures were protonated with Moka at 4.8pH. Compounds with charge less than 1 are negative. Otherwise, compounds are converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a P','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','none','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',64,324,40,33,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Compounds predicted to be positive are likely to produce phospholipidosis.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/2','Verified','na',NULL),(484,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Tomizawa et al. J Toxicol Sci. (2006) 31:315-24. PMID: 17077586','Drug-induced Phospholipidosis decision-tree based','DIPL2','1','Phospholipidosis; DIPL; Drug-induced phospholipidosis; charge; logP; decision tree','N/A','Accumulation of fluorescence-labeled lipid using isolated rat hepatocytes.','Binary. It shows existence of multiple vacuoles filled with phospholipids that show on electron microscopic examination.','Tomizawa et al. J Toxicol Sci. (2006) 31:315-24. PMID: 17077586',0,'Small molecules','No extra data curation was done',0.00,0.0000,0.0000,0.0000,'N/A','compound name','small molecules','Decision tree based on formal charge at pH 4.0 and logP','Net Charge (NC) at pH 4 and LogP are calculated for the compound. If NC < 1 is considered to not induce Phospholipidosis. If NC = 1 and  logP  <= 1.61 is considered to not induce Phospholipidosis If NC = 1 and  1.61 <=  logP is considered to induce Phosph','No test set was used.','Formal charge at pH 4.0 and logP','none','none','none',0,0,0,0,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Compounds predicted to be positive are likely to produce phospholipidosis.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/3','Not verified','na',NULL),(485,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Tomizawa et al. J Toxicol Sci. (2006) 31:315-24. PMID: 17077586','Drug-induced Phospholipidosis decision-tree based','DIPL2','2','Phospholipidosis, DIPL, Drug-induced phospholipidosis','N/A','Accumulation of fluorescence-labeled lipid using isolated rat hepatocytes.','Binary. It shows existence of multiple vacuoles filled with phospholipids that show on electron microscopic examination.','Tomizawa et al. J Toxicol Sci. (2006) 31:315-24. PMID: 17077586',52,'The model is a decision tree and has not been built from a training set. However, the set of small molecules used to derive the rules has been used to compute quality parameters and for verification purposes.','Public structures in the original paper (52 out of 63) were converted to 3D structures with CORINA 3.494. The most populated ionization state at pH 4.0 was computed using program Moka 3.0.',40.40,0.0000,0.0000,0.0000,'no','compound name','small molecules','Decision tree based on formal charge at pH 4.0 and logP','Net Charge (NC) at pH 4 and LogP are calculated for the compound. If NC < 1 is considered to not induce Phospholipidosis. If NC = 1 and  logP  <= 1.61 is considered to not induce Phospholipidosis If NC = 1 and  1.61 <=  logP is considered to induce Phospholipidosis','No test set was used.','Formal charge at pH 4.0 and logP','none','none','none',30,16,1,5,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Compounds predicted to be positive are likely to produce phospholipidosis.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/3','Verified','na',NULL),(486,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Tomizawa et al. J Toxicol Sci. (2006) 31:315-24. PMID: 17077586','Drug-induced Phospholipidosis decision-tree based','DIPL2p','2','Phospholipidosis, DIPL, Drug-induced phospholipidosis','N/A','Accumulation of fluorescence-labeled lipid using isolated rat hepatocytes.','Binary. It shows existence of multiple vacuoles filled with phospholipids that show on electron microscopic examination.','Tomizawa et al. J Toxicol Sci. (2006) 31:315-24. PMID: 17077586',52,'The model is a decision tree and has not been built from a training set.\r\nHowever, the set of small molecules used to derive the rules has been\r\nused to compute quality parameters and for verification purposes','No extra data curation was performed',40.40,0.0000,0.0000,0.0000,'no','compound name','small molecules','Decision tree based on formal charge at pH 4.0 and logP','Net Charge (NC) at pH 4 and LogP are calculated for the compound. If NC < 1 is considered to not induce Phospholipidosis. If NC = 1 and  logP  <= 1.61 is considered to not induce Phospholipidosis If NC = 1 and  1.61 <=  logP is considered to induce Phosph','No test set was used.','Formal charge at pH 4.0 and logP','none','none','none',30,17,0,5,0.0000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.0000,0.0000,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','boolean','negative, positive','Compounds predicted to be positive are likely to produce phospholipidosis.','/Toxicity/Organ Toxicity/Phospholipidosis/DIPL/4','Verified','na',NULL),(487,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','FXR Bile acid receptor activation','FXR','1','FXR; Bile acid receptor','N/A','Activation of FXR transcription factor. Receptor for bile acids such as chenodeoxycholic acid, lithocholic acid and deoxycholic acid.','Quantitative. It represents activation of the receptor.','Collection ligands annotated to FXR from ChEMBL database.',254,'public domain. Small molecules. (MolWeight data: Min. 363.2;1st Qu. 446.5;Median 503.3;Mean 497.5;3rd Qu. 541.1;Max. 669.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,3.6540,8.6990,6.1570,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were  converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS with 4 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode\r\n','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5490,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.4170,0.7050,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the activation of Farnesoid X Receptor (FXR).','/Toxicity/Target Safety Pharmacology/FXR activation/1','Not verified','na',NULL),(488,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','FXR Bile acid receptor activation','FXR','2','FXR, Bile acid receptor, Farnesoid X-activated receptor, Retinoid X receptor-interacting protein 14','N/A','Activation of FXR transcription factor. Receptor for bile acids such as chenodeoxycholic acid, lithocholic acid and deoxycholic acid.','Quantitative. It represents activation of the receptor.','Collection ligands annotated to FXR from ChEMBL database.',254,'public domain. Small molecules. (MolWeight data: Min. 363.2;1st Qu. 446.5;Median 503.3;Mean 497.5;3rd Qu. 541.1;Max. 669.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,3.6540,8.6990,6.1570,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 4 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6200,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.5200,0.6400,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the activation of Farnesoid X Receptor (FXR).','/Toxicity/Target Safety Pharmacology/FXR activation/1','Verified','M (EC50 in minus log scale)',NULL),(489,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','FXR Bile acid receptor activation','FXRp','2','FXR, Bile acid receptor, Farnesoid X-activated receptor, Retinoid X receptor-interacting protein 14','N/A','Activation of FXR transcription factor. Receptor for bile acids such as chenodeoxycholic acid, lithocholic acid and deoxycholic acid.','Quantitative. It represents activation of the receptor.','Collection ligands annotated to FXR from ChEMBL database.',254,'public domain. Small molecules. (MolWeight data: Min. 363.2;1st Qu. 446.5;Median 503.3;Mean 497.5;3rd Qu. 541.1;Max. 669.1)','CHEMBL Extracted data was filtered with  standard units == ”nM” and activity type == ”EC50”',0.00,3.6540,8.6990,6.1570,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a PLS with 4LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5500,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.4100,0.7100,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the activation of Farnesoid X Receptor (FXR).','/Toxicity/Target Safety Pharmacology/FXR activation/2','Verified','M (EC50 in minus log scale)',NULL),(490,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','HERG Potassium voltage-gated channel subfamily H member 2 blockade','HERG1','1','HERG; Potassium voltage-gated channel subfamily H member 2','N/A','Blockade of hERG voltage-dependent potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to HERG from ChEMBL database.',807,'public domain. Small molecules. (MolWeight data: Min. 86.1;1st Qu. 319.1;Median 396.2;Mean 400.1;3rd Qu. 459.3;Max. 837.5)','Quinidine and Clofilium were removed. IC50 values were normalized to remove the effect of cell types.',0.00,1.2900,8.1700,5.0000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were  converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS with 2 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6170,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.6040,0.8510,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for HERG Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG inhibition/1','Not verified','na',NULL),(491,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','HERG Potassium voltage-gated channel subfamily H member 2 blockade','HERG1','2','HERG, Potassium voltage-gated channel subfamily H member 2, H-ERG, hERG-1, Voltage-gated potassium channel subunit Kv11.1','N/A','Blockade of hERG voltage-dependent potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated  from ChEMBL database.',807,'public domain. Small molecules. (MolWeight data: Min. 86.1;1st Qu. 319.1;Median 396.2;Mean 400.1;3rd Qu. 459.3;Max. 837.5)','Quinidine and Clofilium were removed. IC50 values were normalized to remove the effect of cell types.',0.00,1.2900,8.1700,5.0000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were  converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 2LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6000,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.5900,0.8700,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for HERG Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG inhibition/1','Verified','M (IC50 in minus log scale)',NULL),(492,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','HERG Potassium voltage-gated channel subfamily H member 2 blockade','HERG1p','2','HERG, Potassium voltage-gated channel subfamily H member 2, H-ERG, hERG-1, Voltage-gated potassium channel subunit Kv11.1','N/A','Blockade of hERG voltage-dependent potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to HERG from ChEMBL database.',807,'public domain. Small molecules. (MolWeight data: Min. 86.1;1st Qu. 319.1;Median 396.2;Mean 400.1;3rd Qu. 459.3;Max. 837.5)','Quinidine and Clofilium were removed. IC50 values were normalized to remove the effect of cell types.',0.00,1.2900,8.1700,5.0000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were protonated with Moka at 6.3pH and converted to 3D structures with CORINA 3.494. Then Pentacle molecular descriptors were calculated and used to build a PLS with 2LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6800,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.6600,0.7800,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for HERG Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG inhibition/2','Verified','M (IC50 in minus log scale)',NULL),(493,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','HERG Potassium voltage-gated channel subfamily H member 2 blockade, using local models','HERG4','1','HERG; Potassium voltage-gated channel subfamily H member 2; local model','N/A','Blockade of hERG voltage-dependent potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to HERG from ChEMBL database.',807,'public domain. Small molecules. (MolWeight data: Min. 86.1;1st Qu. 319.1;Median 396.2;Mean 400.1;3rd Qu. 459.3;Max. 837.5)','Quinidine and Clofilium were removed. IC50 values were normalized to remove the effect of cell types',0.00,1.2900,8.1700,5.0000,'NO','CHEMBLID','small molecules','\"MW split local models, PLS-R and 3D QSAR\"','Training set structures were converted to 3D structures with CORINA 2.4. The training set is splited in 4 by MW. Then Adriana molecular descriptors were calculated and used to build 4 PLS models with 2LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.7610,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.7620,0.6580,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for HERG Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG inhibition/3','Not verified','na',NULL),(494,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','HERG Potassium voltage-gated channel subfamily H member 2 blockade, using local models','HERG4','2','HERG, Potassium voltage-gated channel subfamily H member 2, H-ERG, hERG-1, Voltage-gated potassium channel subunit Kv11.1','N/A','Blockade of hERG voltage-dependent potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to HERG from ChEMBL database.',807,'public domain. Small molecules. (MolWeight data: Min. 86.1;1st Qu. 319.1;Median 396.2;Mean 400.1;3rd Qu. 459.3;Max. 837.5)','Quinidine and Clofilium were removed. IC50 values were normalized to remove the effect of cell types',0.00,1.2900,8.1700,5.0000,'NO','CHEMBLID','small molecules','\"MW split local models, PLS-R and 3D QSAR\"','Training set structures were converted to 3D structures with CORINA 3.494. The training set is splited in 4 by MW. Then Adriana molecular descriptors were calculated and used to build 4 PLS models with 2LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.7800,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.7800,0.6400,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for HERG Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG inhibition/3','Verified','M (IC50 in minus log scale)',NULL),(495,'2016-02-08 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','HERG Potassium voltage-gated channel subfamily H member 2 blockade, using local models','HERG4p','2','HERG, Potassium voltage-gated channel subfamily H member 2, H-ERG, hERG-1, Voltage-gated potassium channel subunit Kv11.1','N/A','Blockade of hERG voltage-dependent potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection ligands annotated to HERG from ChEMBL database.',807,'public domain. Small molecules. (MolWeight data: Min. 86.1;1st Qu. 319.1;Median 396.2;Mean 400.1;3rd Qu. 459.3;Max. 837.5)','Quinidine and Clofilium were removed. IC50 values were normalized to remove the effect of cell types',0.00,1.2900,8.1700,5.0000,'NO','CHEMBLID','small molecules','\"MW split local models, PLS-R and 3D QSAR\"','Training set structures were protonated with Moka at 6.3pH and converted to 3D structures with CORINA 3.494. The training set is splited in 4 by MW. Then Pentacle molecular descriptors were calculated and used to build 4 PLS models with 2 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.7200,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.7600,0.6700,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for HERG Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/hERG inhibition/4','Verified','M (IC50 in minus log scale)',NULL),(496,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','KCNQ1 Potassium voltage-gated channel blockade','KCNQ1','1','KCNQ1; Potassium voltage-gated channel','N/A','Blockade of KCNQ1 voltage-gated potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection of ligands annotated to KCNQ1 from ChEMBL database and extracted from J Chem Inf Model. 2011 Feb 28;51(2):483-92. doi: 10.1021/ci100423z.',225,'public domain. Small molecules. (MolWeight data: Min.150.1;1st Qu. 345.3;Median 440.2;Mean414.5;3rd Qu. 482.1;Max.645.0)','No extra data curation was done',0.00,3.0000,8.0000,5.2000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 2.4. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5890,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.3000,0.9750,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for KCNQ1 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/KCNQ1 inhibition/1','Not verified','na',NULL),(497,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','KCNQ1 Potassium voltage-gated channel blockade','KCNQ1','2','KCNQ1, Potassium voltage-gated channel subfamily KQT member 1, IKs producing slow voltage-gated potassium channel subunit alpha KvLQT1, Voltage-gated potassium channel subunit Kv7.1','N/A','Blockade of KCNQ1 voltage-gated potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection of ligands annotated to KCNQ1 from ChEMBL database and extracted from J Chem Inf Model. 2011 Feb 28;51(2):483-92. doi: 10.1021/ci100423z.',225,'public domain. Small molecules. (MolWeight data: Min.150;1st Qu. 345;Median 440;Mean415;3rd Qu. 482;Max.645)','No extra data curation was done',0.00,3.0000,8.0000,5.2000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 3 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.4800,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.3200,0.9600,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for KCNQ1 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/KCNQ1 inhibition/1','Verified','M (IC50 in minus log scale)',NULL),(498,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','N/A','KCNQ1 Potassium voltage-gated channel blockade','KCNQ1p','2','KCNQ1, Potassium voltage-gated channel subfamily KQT member 1, IKs producing slow voltage-gated potassium channel subunit alpha KvLQT1, Voltage-gated potassium channel subunit Kv7.1','N/A','Blockade of KCNQ1 voltage-gated potassium channel','Quantitative. It represents inhibition (blockade) of the channel.','Collection of ligands annotated to KCNQ1 from ChEMBL database and extracted from J Chem Inf Model. 2011 Feb 28;51(2):483-92. doi: 10.1021/ci100423z.',225,'public domain. Small molecules. (MolWeight data: Min.150.1;1st Qu. 345.3;Median 440.2;Mean414.5;3rd Qu. 482.1;Max.645.0)','No extra data curation was done',0.00,3.0000,8.0000,5.2000,'NO','CHEMBLID','small molecules','PLS-R and 3D QSAR','Training set structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494. Then Adriana molecular descriptors were calculated and used to build a PLS with 5 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5900,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.3600,0.9300,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-Inf,Inf)','Predicts the inhibition for KCNQ1 Receptor.','/Toxicity/Organ Toxicity/Cardiotoxicity/KCNQ1 inhibition/2','Verified','M (IC50 in minus log scale)',NULL),(523,'2015-12-05 20:44:07','Daan Geerke','d.p.geerke@vu.nl','L. Capoferri et al., PLOS ONE 2015, 10, e0142232.','LIE prediction of binding affinity to CYP450 1A2','CYP 1A2 Affinity 2 VUA','1','CYP 1A2, CP1A2, CP1A2_HUMAN, Cytochrome P450 1A2, binding affinity, free energies of binding, IC50s','CYP 2D6 Affinity 2 VUA','CYP 1A2 affinity','Binding affinities are expressed in terms of the free energy of binding (DG) of the compound (in kJ/mol)','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5, with [S] = K_m), were taken from the Marketed Drugs (MTB04) and World Drug Index Drugs (METB05) CYP inhibition datasets supplied by Bayer (see eTOX intranet), from \r\n[Cohen et al. \"In vitro drug interactions of cytochrome P450: an evaluation of fluorogenic to conventional substrates.\" Drug Metab. Dispos. 2003, 31, 1005-1015.] DOI: 10.1124/dmd.31.8.1005 ,\r\nand from\r\n[Korhonen et al. \"Predictive three-dimensional quantitative structure-activity relationship of Cytochrome P450 1A2 inhibitors.\" J. Med. Chem. 2005, 48, 3808-3815.] DOI: 10.1021/jm0489713].',35,'A structurally heterogeneous set of 1A2 inhibitors/substrates (in terms of size, number of rings, and presence of number of different functional groups).','See \"Training set source\". IC50 values were converted into binding constants and free energies using the Cheng-Prusoff equation.',NULL,-42.5900,-10.7900,-25.5600,'yes','IDs according to original reference (L. Capoferri et al)','Only for compounds for which IC50s were taken from [Korhonen et al] (ID=27,28,30,31,32,33,34,35,36,38,39,40,41,42,43,50,51,52,53,54,55,56,57,58,59,60,61,62,64,65,67,68,70,72,74), the data source explicitly reports compound purities (>95% in those cases)','LIE model (molecular dynamics based)','Results from molecular dynamics (MD) simulations of a compound bound to protein and unbound in water are combined using linear interaction energy (LIE) theory, to calculate the binding free energy (see also deliverables D3.5 and D3.13).','In total, 57 compounds (training+test set) were selected from the 73 compounds for which CYP 1A2 inhibition was quantitatively determined in the data sources reported under \"Training set source\". The compounds were selected (automatically, based on predicted binding modes) such that only non-heme-coordinating binders were included in training and test sets. The 57 compounds were divided in three main structural classes of compounds. 35 compounds were randomly selected as training compounds, evenly distributed over the three subclasses. For the remaining 22 compounds, external validation data (\"External val Q2\" and \"External val SDEP\") are reported based on the results for the 15 test compounds which fulfilled the applicability domain assessment (see \"AD assessment\").','DVele and DVvdw. Using LIE, binding free energies (affinities) are calculated from the average interaction energies (DVele and DVvdw) of the compound with its surroundings as obtained from MD simulations (see also deliverables D3.5 and D3.13).','N/A (cf. \"Descriptors definition\")','Diverse set of 1A2 binders was included in the training and test set. Compounds vary in their molecular weight, the number of rings, and the presence of functional groups.','AD criteria should be fulfilled, which are determined from (1) a Mahalanobis-distance analysis on measured simulation properties, (2) ligand-fingerprint similarities, (3) and (4) two residue-based protein-ligand interaction patterns in simulation (see D3.13 and L. Capoferri et al.), and (5) spread in experimental data.',NULL,NULL,NULL,NULL,0.6900,3.9200,'N/A','IC50s, converted to binding free energies using Cheng-Prusoff (see D3.5, with [S] = K_m), were taken from \r\n[Korhonen et al. \"Predictive three-dimensional quantitative structure-activity relationship of Cytochrome P450 1A2 inhibitors.\" J. Med. Chem. 2005, 48, 3808-3815.] DOI: 10.1021/jm0489713]. See \"Test set selection\" for further details.',15,NULL,NULL,NULL,NULL,0.4100,4.9100,NULL,0.6200,4.3400,'A molecular-dynamics (MD) based LIE model is constructed for binding affinity prediction to account for the protein flexibility (plasticity) of CYP 1A2.','See \"Mechanistic basis of the model\". See [L. Capoferri et al., PLOS ONE 2015, 10, e0142232] for details on the Applicability Domain assessment. Note that model statistics are slightly different here than in the original PLOS ONE manuscript because of the use of other docking software than reported in the manuscript (for licensing reasons, here we use the ParaDocks freeware) and because of the use of two simulations per ligand pose to obtain improved convergence of simulation data. Use of the implemented protocol resulted in an alpha and beta value of 0.594 and 0.315 in the central LIE equation, respectively.','LIE theory:  J. Åqvist, C. Medina and J.E. Samuelsson. Protein Eng. 1994, 7, 385-391, DOI: 10.1093/protein/7.3.385 ; T. Hansson, J. Marelius and J. Åqvist. J. Comput. Aided Mol. Des. 1998, 12, 27-35, DOI: 10.1023/A:1007930623000;  E. Stjernschantz and C. Oostenbrink. Biophys. J. 2010, 98, 2662-2691, DOI: 10.1016/j.bpj.2010.02.034',NULL,'VUA','human','http://etox.labs.vu.nl/etoxwsapi/v2','float','(-inf,inf)','Binding free energies (affinities) DG can be translated into inhibition constants K_i in M by using DG = RT ln K_i (with R: gas constant and T: temperature), and into IC50s in M at [S] = K_m using K_i = IC50/2. The more negative DG is for a compound, the stronger it binds to CYP 1A2.','/ADME/Metabolism/Phase I/CYP 1A2 Affinity/2','Verified','kJ/mol',NULL),(477,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Hou et al. J Chem Inf Comput Sci (2004) 44; 1585. doi: 10.1021/ci049884m','Caco-2 Cell permeability (logP_app)','CACO2p','2','CACO2, GI absorption','N/A','CACO2-Permeability: It measures in CACO-2 cell line the transport of the drug across the cell membrane from apical-to-basal.','Quatitative. Log P_app represents transport of the drug across the cell membrane from apical-to-basal, as log(cm/sec).','Hou et al. J Chem Inf Comput Sci (2004) 44; 1585. doi: 10.1021/ci049884m',100,'Structurally heterogeneous series covers a relatively wide range of molecular size and lipophilicity. All compounds were regarded as being transported by passive diffusion.','No extra data curation was done',0.00,-6.9600,-4.1100,-5.1850,'NO','compound name','small molecules','PLS-R and 3D QSAR','Training set 2D structures were protonated with Moka at 7.4pH and converted to 3D structures with CORINA 3.494.Then Pentacle molecular descriptors were calculated and used to build a PLS model with 3 LV.','No separation was performed','GRIND-2 descriptors were used. Pastor M,  J Med Chem. 2000;43(17):3233-43. doi: 10.1021/jm000941m','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5700,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.3600,0.6200,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-inf,0)','Predicted ability of the query compound to be transported across the cell membrane of CACO-2 cell lines from apical-to-basal, expressed as log of apparent permeability coefficient (Papp) in cm/sec units.','/ADME/Absorption/Gastro Intestinal/CACO2 permeability/2','Verified','cm/sec (log of apparent permeability coefficient, Papp)',NULL),(475,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Hou et al. J Chem Inf Comput Sci (2004) 44; 1585. doi: 10.1021/ci049884m','Caco-2 Cell permeability (logP_app)','CACO2','1','CACO2; absorption; GI absorption; ADME; apparent permeability','N/A','CACO2-Permeability: It measures in CACO-2 cell line the transport of the drug across the cell membrane from apical-to-basal.','Quatitative. Log P_app represents transport of the drug across the cell membrane from apical-to-basal, as log(cm/sec).','Hou et al. J Chem Inf Comput Sci (2004) 44; 1585. doi: 10.1021/ci049884m',99,'Structurally heterogeneous series covers a relatively wide range of molecular size and lipophilicity. All compounds were regarded as being transported by passive diffusion.','No extra data curation was done',0.00,-6.9600,-4.1100,-5.1850,'NO','compound name','small molecules','PLS-R and 3D QSAR','Training set 2D structures were converted to 3D structures with CORINA 2.4.Then Adriana molecular descriptors were calculated and used to build a PLS model with 2 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.5730,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.3600,0.6230,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-inf,0)','Predicted ability of the query compound to be transported across the cell membrane of CACO-2 cell lines from apical-to-basal, expressed as log of apparent permeability coefficient (Papp) in cm/sec units.','/ADME/Absorption/Gastro Intestinal/CACO2 permeability/1','Not verified','na',NULL),(476,'2015-10-02 00:00:00','Manuel Pastor','manuel.pastor@upf.edu','Hou et al. J Chem Inf Comput Sci (2004) 44; 1585. doi: 10.1021/ci049884m','Caco-2 Cell permeability (logP_app)','CACO2','2','CACO2, GI absorption','N/A','CACO2-Permeability: It measures in CACO-2 cell line the transport of the drug across the cell membrane from apical-to-basal.','Quatitative. Log P_app represents transport of the drug across the cell membrane from apical-to-basal, as log(cm/sec).','Hou et al. J Chem Inf Comput Sci (2004) 44; 1585. doi: 10.1021/ci049884m',100,'Structurally heterogeneous series covers a relatively wide range of molecular size and lipophilicity. All compounds were regarded as being transported by passive diffusion.','No extra data curation was done',0.00,-6.9600,-4.1100,-5.1850,'NO','compound name','small molecules','PLS-R and 3D QSAR','Training set 2D structures were converted to 3D structures with CORINA 3.494.Then Adriana molecular descriptors were calculated and used to build a PLS model with 2 LV.','No separation was performed','AdrianaCode descriptors were used. http://www.molecular-networks.com/products/adrianacode','2FFD variable selection. Baroni, Massimo et al. DOI=10.1002/qsar.19930120103','ADAN, Carrio et al. DOI: 10.1021/ci500172z','ADAN, Carrio et al. DOI: 10.1021/ci500172z',0,0,0,0,0.6500,0.0000,'N/A','N/A',0,0,0,0,0,0.0000,0.0000,'N/A',0.5800,0.5100,'N/A','N/A','N/A','N/A','FIMIM','N/A','http://damocles.imim.es/etoxwsapi/v2/dir','float','(-inf,0)','Predicted ability of the query compound to be transported across the cell membrane of CACO-2 cell lines from apical-to-basal, expressed as log of apparent permeability coefficient (Papp) in cm/sec units.','/ADME/Absorption/Gastro Intestinal/CACO2 permeability/1','Verified','cm/sec (log of apparent permeability coefficient, Papp)',NULL),(150,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','AR Affinity 1','AR Affinity 1 CT','1','AR, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over AR','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to AR extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1885,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.4000,10.0000,6.3500,'Skewness=-0.25','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7200,0.6500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',335,NULL,NULL,NULL,NULL,0.7100,0.6600,'N/A',0.7200,0.6500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over AR or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/AR_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(151,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','AT2R1 Affinity 1','AT2R1 Affinity 1 CT','1','AT2R1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over AT2R1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to AT2R1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3065,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,11.0000,7.1800,'Skewness=-0.60','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8100,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',543,NULL,NULL,NULL,NULL,0.8100,0.6700,'N/A',0.8100,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over AT2R1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/AT2R1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(152,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA1 Affinity 1','CA1 Affinity 1 CT','1','CA1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2509,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.5000,6.1600,'Skewness=-0.11','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6900,0.6100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',444,NULL,NULL,NULL,NULL,0.6900,0.6000,'N/A',0.6900,0.6100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(153,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA12 Affinity 1','CA12 Affinity 1 CT','1','CA12, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA12','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA12 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',722,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.7000,7.2200,'Skewness=-0.97','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7400,0.6100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',129,NULL,NULL,NULL,NULL,0.7500,0.6100,'N/A',0.7400,0.6100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA12 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA12_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(154,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA13 Affinity 1','CA13 Affinity 1 CT','1','CA13, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA13','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA13 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',193,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.7000,6.3600,'Skewness=-0.35','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7800,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',35,NULL,NULL,NULL,NULL,0.8000,0.6100,'N/A',0.7800,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA13 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA13_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(155,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA14 Affinity 1','CA14 Affinity 1 CT','1','CA14, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA14','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA14 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',331,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.7000,6.7300,'Skewness=-0.72','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.6200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',57,NULL,NULL,NULL,NULL,0.7800,0.5300,'N/A',0.7100,0.6200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA14 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA14_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(156,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA2 Affinity 1','CA2 Affinity 1 CT','1','CA2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3099,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.1000,7.0900,'Skewness=-1.18','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7500,0.6100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',546,NULL,NULL,NULL,NULL,0.7400,0.6200,'N/A',0.7500,0.6100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(157,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA4 Affinity 1','CA4 Affinity 1 CT','1','CA4, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA4','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA4 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1218,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.7000,6.8400,'Skewness=-1.32','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.5400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',216,NULL,NULL,NULL,NULL,0.7100,0.5400,'N/A',0.7100,0.5400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA4 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA4_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(158,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA5A Affinity 1','CA5A Affinity 1 CT','1','CA5A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA5A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA5A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',206,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.5000,6.2500,'Skewness=-0.63','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6100,0.8400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',35,NULL,NULL,NULL,NULL,0.6600,0.7400,'N/A',0.6100,0.8400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA5A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA5A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(159,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA5B Affinity 1','CA5B Affinity 1 CT','1','CA5B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA5B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA5B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',156,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,9.7000,6.1900,'Skewness=-0.41','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6100,0.9000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',28,NULL,NULL,NULL,NULL,0.6700,0.8000,'N/A',0.6100,0.9000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA5B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA5B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(160,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA6 Affinity 1','CA6 Affinity 1 CT','1','CA6, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA6','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA6 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',171,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.7000,5.6400,'N/A','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.7700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',29,NULL,NULL,NULL,NULL,0.7400,0.6800,'N/A',0.7000,0.7700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA6 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA6_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(161,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA7 Affinity 1','CA7 Affinity 1 CT','1','CA7, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA7','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA7 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',320,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.7000,6.6300,'Skewness=-0.67','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.7600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',58,NULL,NULL,NULL,NULL,0.7000,0.7600,'N/A',0.7000,0.7600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA7 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA7_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(162,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CA9 Affinity 1','CA9 Affinity 1 CT','1','CA9, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CA9','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CA9 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1181,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,9.9000,6.9900,'Skewness=-0.88','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.5700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',206,NULL,NULL,NULL,NULL,0.7100,0.5700,'N/A',0.7100,0.5700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CA9 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CA9_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(163,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CANPL1 Affinity 1','CANPL1 Affinity 1 CT','1','CANPL1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CANPL1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CANPL1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',569,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.7000,6.7800,'Skewness=-0.08','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6900,0.6400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',102,NULL,NULL,NULL,NULL,0.6800,0.6300,'N/A',0.6900,0.6400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CANPL1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CANPL1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(164,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','Cav1.2 Affinity 1','Cav1.2 Affinity 1 CT','1','Cav1.2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over Cav1.2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to Cav1.2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',461,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.8000,11.1000,6.4700,'Skewness=0.20','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5900,0.8400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',83,NULL,NULL,NULL,NULL,0.6200,0.8000,'N/A',0.5900,0.8400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over Cav1.2 or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/Cav1.2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(165,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','Cav1.3 Affinity 1','Cav1.3 Affinity 1 CT','1','Cav1.3, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over Cav1.3','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to Cav1.3 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',336,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.5000,11.1000,6.6600,'Skewness=0.51','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5600,0.7800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',60,NULL,NULL,NULL,NULL,0.6000,0.7700,'N/A',0.5600,0.7800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over Cav1.3 or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/Cav1.3_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(166,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','Cav2.2 Affinity 1','Cav2.2 Affinity 1 CT','1','Cav2.2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over Cav2.2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to Cav2.2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',267,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,7.4000,5.5600,'Skewness=-1.54','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8400,0.3700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',46,NULL,NULL,NULL,NULL,0.8400,0.3600,'N/A',0.8400,0.3700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over Cav2.2 or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/Cav2.2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(167,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CCKAR Affinity 1','CCKAR Affinity 1 CT','1','CCKAR, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CCKAR','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CCKAR extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1769,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,10.5000,6.0400,'Skewness=0.46','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',311,NULL,NULL,NULL,NULL,0.6400,0.6800,'N/A',0.6400,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CCKAR or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CCKAR_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(168,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CCKBR Affinity 1','CCKBR Affinity 1 CT','1','CCKBR, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CCKBR','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CCKBR extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1851,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,12.0000,6.4600,'Skewness=-0.03','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7900,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',327,NULL,NULL,NULL,NULL,0.8000,0.6600,'N/A',0.7900,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CCKBR or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CCKBR_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(169,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CHRM1 Affinity 1','CHRM1 Affinity 1 CT','1','CHRM1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CHRM1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CHRM1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3265,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.0000,6.1500,'Skewness=0.14','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7400,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',577,NULL,NULL,NULL,NULL,0.7300,0.6900,'N/A',0.7400,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CHRM1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CHRM1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(170,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CHRM2 Affinity 1','CHRM2 Affinity 1 CT','1','CHRM2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CHRM2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CHRM2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2936,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.0000,6.3800,'Skewness=0.12','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7600,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',517,NULL,NULL,NULL,NULL,0.7500,0.6800,'N/A',0.7600,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CHRM2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CHRM2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(171,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CHRM3 Affinity 1','CHRM3 Affinity 1 CT','1','CHRM3, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CHRM3','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CHRM3 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2516,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.5000,6.5700,'Skewness=0.08','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8000,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',447,NULL,NULL,NULL,NULL,0.8100,0.6800,'N/A',0.8000,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CHRM3 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CHRM3_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(172,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CHRM4 Affinity 1','CHRM4 Affinity 1 CT','1','CHRM4, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CHRM4','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CHRM4 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1711,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.8000,6.0400,'Skewness=0.09','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7400,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',300,NULL,NULL,NULL,NULL,0.7400,0.6700,'N/A',0.7400,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CHRM4 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CHRM4_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(173,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CHRM5 Affinity 1','CHRM5 Affinity 1 CT','1','CHRM5, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over CHRM5','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CHRM5 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1657,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.3000,5.9100,'Skewness=0.15','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7200,0.6700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',294,NULL,NULL,NULL,NULL,0.7200,0.6800,'N/A',0.7200,0.6700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CHRM5 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/CHRM5_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(174,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','COX-1 Affinity 1','COX-1 Affinity 1 CT','1','COX-1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over COX-1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to COX-1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3340,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.0000,4.4900,'Skewness=0.56','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5500,0.7300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',592,NULL,NULL,NULL,NULL,0.5600,0.7300,'N/A',0.5500,0.7300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over COX-1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/COX-1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(175,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','COX-2 Affinity 1','COX-2 Affinity 1 CT','1','COX-2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over COX-2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to COX-2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3562,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.7000,5.6500,'Skewness=-0.06','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5700,0.8200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',630,NULL,NULL,NULL,NULL,0.5700,0.8200,'N/A',0.5700,0.8200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over COX-2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/COX-2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(176,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 11B1 Inhibition 1','CYP 11B1 Inhibition 1 CT','1','CYP, 11B1, Inhibition, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',384,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.3000,6.3300,'Skewness=-0.53','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6900,0.5200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',67,NULL,NULL,NULL,NULL,0.6700,0.4900,'N/A',0.6900,0.5200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/CYP_Inhibition/CYP_11B1_Inhibition/1','Verification ongoing','M (in minus log scale)',NULL),(177,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 19A1 Inhibition 1','CYP 19A1 Inhibition 1 CT','1','CYP, 19A1, Inhibition, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1316,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.5000,11.7000,6.0800,'Skewness=0.13','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',233,NULL,NULL,NULL,NULL,0.6800,0.6800,'N/A',0.7000,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/CYP_Inhibition/CYP_19A1_Inhibition/1','Verification ongoing','M (in minus log scale)',NULL),(178,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 1A2 Inhibition 1','CYP 1A2 Inhibition 1 CT','1','CYP, 1A2, Inhibition, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',297,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,8.3000,5.1300,'Skewness=0.44','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6200,0.7000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',52,NULL,NULL,NULL,NULL,0.6200,0.7400,'N/A',0.6200,0.7000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/CYP_Inhibition/CYP_1A2_Inhibition/1','Verification ongoing','M (in minus log scale)',NULL),(179,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 2C19 Inhibition 1','CYP 2C19 Inhibition 1 CT','1','CYP, 2C19, Inhibition, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',307,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,8.5000,4.9700,'Skewness=0.77','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5700,0.7300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',55,NULL,NULL,NULL,NULL,0.5900,0.7700,'N/A',0.5700,0.7300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/CYP_Inhibition/CYP_2C19_Inhibition/1','Verification ongoing','M (in minus log scale)',NULL),(180,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 2C9 Affinity 1','CYP 2C9 Affinity 1 CT','1','CYP, 2C9, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',193,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,9.0000,4.5000,'Skewness=1.03','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5600,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',35,NULL,NULL,NULL,NULL,0.8000,0.5800,'N/A',0.5600,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/Phase_I/CYP_2C9_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(181,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 2D6 Affinity 1','CYP 2D6 Affinity 1 CT','1','CYP, 2D6, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',246,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.0000,5.2600,'Skewness=0.44','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',42,NULL,NULL,NULL,NULL,0.7800,0.5200,'N/A',0.6400,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/Phase_I/CYP_2D6_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(182,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 2D6 Inhibition 1','CYP 2D6 Inhibition 1 CT','1','CYP, 2D6, Inhibition, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',967,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.0000,5.0000,'Skewness=0.37','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',171,NULL,NULL,NULL,NULL,0.6300,0.6300,'N/A',0.6400,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/CYP_Inhibition/CYP_2D6_Inhibition/1','Verification ongoing','M (in minus log scale)',NULL),(183,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','CYP 3A4 Affinity 1','CYP 3A4 Affinity 1 CT','1','CYP, 3A4, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over CYP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to CYP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',629,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.3000,11.0000,5.1000,'Skewness=0.71','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6300,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',111,NULL,NULL,NULL,NULL,0.7000,0.6000,'N/A',0.6300,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over CYP or none if compound is outside applicability domain','/ADME/Metabolism/Phase_I/CYP_3A4_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(184,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','DRD1 Affinity 1','DRD1 Affinity 1 CT','1','DRD1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over DRD1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to DRD1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2387,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.8000,5.9400,'Skewness=0.23','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6800,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',418,NULL,NULL,NULL,NULL,0.6800,0.6800,'N/A',0.6800,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over DRD1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/DRD1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(185,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','DRD2 Affinity 1','DRD2 Affinity 1 CT','1','DRD2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over DRD2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to DRD2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',8653,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.5000,11.8000,6.3500,'Skewness=-0.03','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.6200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',1526,NULL,NULL,NULL,NULL,0.6300,0.6200,'N/A',0.6400,0.6200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over DRD2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/DRD2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(186,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','DRD3 Affinity 1','DRD3 Affinity 1 CT','1','DRD3, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over DRD3','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to DRD3 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3464,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.7000,7.0600,'Skewness=-0.33','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6800,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',612,NULL,NULL,NULL,NULL,0.6700,0.6600,'N/A',0.6800,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over DRD3 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/DRD3_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(187,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','DRD4 Affinity 1','DRD4 Affinity 1 CT','1','DRD4, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over DRD4','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to DRD4 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2231,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.6000,10.7000,6.8800,'Skewness=-0.48','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5800,0.6500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',395,NULL,NULL,NULL,NULL,0.6000,0.6400,'N/A',0.5800,0.6500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over DRD4 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/DRD4_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(188,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','DRD5 Affinity 1','DRD5 Affinity 1 CT','1','DRD5, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over DRD5','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to DRD5 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',674,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.5000,6.3100,'Skewness=-0.18','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6200,0.7800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',118,NULL,NULL,NULL,NULL,0.5900,0.7900,'N/A',0.6200,0.7800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over DRD5 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/DRD5_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(189,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','ER Affinity 1','ER Affinity 1 CT','1','ER, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over ER','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to ER extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2313,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.6000,6.2400,'Skewness=-0.29','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7800,0.7400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',405,NULL,NULL,NULL,NULL,0.7700,0.7500,'N/A',0.7800,0.7400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over ER or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/ER_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(190,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','F10 Affinity 1','F10 Affinity 1 CT','1','F10, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over F10','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to F10 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4636,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.3000,6.9300,'Skewness=-0.18','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8000,0.7000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',816,NULL,NULL,NULL,NULL,0.8100,0.6900,'N/A',0.8000,0.7000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over F10 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/F10_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(191,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','FYN Affinity 1','FYN Affinity 1 CT','1','FYN, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over FYN','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to FYN extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',736,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.4000,9.5000,5.3500,'Skewness=0.57','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5200,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',129,NULL,NULL,NULL,NULL,0.5300,0.6900,'N/A',0.5200,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over FYN or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/FYN_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(192,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','GABRA1 Affinity 1','GABRA1 Affinity 1 CT','1','GABRA1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over GABRA1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to GABRA1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2961,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,10.3000,6.5400,'Skewness=-0.34','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7500,0.7200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',524,NULL,NULL,NULL,NULL,0.7500,0.7300,'N/A',0.7500,0.7200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over GABRA1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/GABRA1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(193,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','hERG Affinity 1','hERG Affinity 1 CT','1','hERG, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over hERG','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to hERG extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',5224,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.4000,9.8000,5.1800,'Skewness=0.44','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6600,0.5700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',922,NULL,NULL,NULL,NULL,0.6700,0.5700,'N/A',0.6600,0.5700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over hERG or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/hERG_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(194,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','HMG-CoA Affinity 1','HMG-CoA Affinity 1 CT','1','HMG-CoA, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over HMG-CoA','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to HMG-CoA extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',855,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.5000,7.0600,'Skewness=-0.61','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.6300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',149,NULL,NULL,NULL,NULL,0.6900,0.6200,'N/A',0.7000,0.6300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over HMG-CoA or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/HMG-CoA_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(195,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','HRH1 Affinity 1','HRH1 Affinity 1 CT','1','HRH1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over HRH1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to HRH1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1850,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.0000,6.4100,'Skewness=-0.01','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7700,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',324,NULL,NULL,NULL,NULL,0.7800,0.6600,'N/A',0.7700,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over HRH1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/HRH1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(196,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','HRH2 Affinity 1','HRH2 Affinity 1 CT','1','HRH2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over HRH2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to HRH2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',722,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.0000,5.4100,'Skewness=1.17','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',126,NULL,NULL,NULL,NULL,0.7300,0.6500,'N/A',0.7100,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over HRH2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/HRH2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(197,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','HRH3 Affinity 1','HRH3 Affinity 1 CT','1','HRH3, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over HRH3','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to HRH3 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3429,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.7000,10.3000,7.5700,'Skewness=-0.66','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.5500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',605,NULL,NULL,NULL,NULL,0.6900,0.5400,'N/A',0.7000,0.5500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over HRH3 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/HRH3_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(198,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','MAPK1 Affinity 1','MAPK1 Affinity 1 CT','1','MAPK1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over MAPK1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to MAPK1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',813,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.0000,4.7400,'Skewness=1.62','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.6800,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',142,NULL,NULL,NULL,NULL,0.7100,0.6600,'N/A',0.7100,0.6800,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over MAPK1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/MAPK1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(199,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','MDR1 Affinity 1','MDR1 Affinity 1 CT','1','MDR1, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over MDR1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to MDR1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',583,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,8.2000,5.3400,'Skewness=-0.37','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.5700,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',104,NULL,NULL,NULL,NULL,0.6500,0.5700,'N/A',0.6400,0.5700,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over MDR1 or none if compound is outside applicability domain','/ADME/Transport/Transporters/MDR1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(200,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','MOP Affinity 1','MOP Affinity 1 CT','1','MOP, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over MOP','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to MOP extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',6130,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,12.5000,6.7400,'Skewness=-0.09','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7800,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',1082,NULL,NULL,NULL,NULL,0.7900,0.6800,'N/A',0.7800,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over MOP or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/MOP_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(201,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','MRP1 Affinity 1','MRP1 Affinity 1 CT','1','MRP1, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over MRP1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to MRP1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',221,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,7.7000,5.2400,'Skewness=-0.10','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7900,0.6200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',39,NULL,NULL,NULL,NULL,0.8100,0.5700,'N/A',0.7900,0.6200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over MRP1 or none if compound is outside applicability domain','/ADME/Transport/Transporters/MRP1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(202,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','NK1R Affinity 1','NK1R Affinity 1 CT','1','NK1R, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over NK1R','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to NK1R extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',2260,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,11.8000,7.8100,'Skewness=-0.71','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7100,0.6600,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',398,NULL,NULL,NULL,NULL,0.7100,0.6600,'N/A',0.7100,0.6600,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over NK1R or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/NK1R_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(203,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','NK2R Affinity 1','NK2R Affinity 1 CT','1','NK2R, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over NK2R','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to NK2R extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',949,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,10.1000,7.2600,'Skewness=-0.66','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7500,0.6300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',168,NULL,NULL,NULL,NULL,0.7600,0.6000,'N/A',0.7500,0.6300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over NK2R or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/NK2R_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(204,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','NR3C1 Affinity 1','NR3C1 Affinity 1 CT','1','NR3C1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over NR3C1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to NR3C1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1630,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.5000,10.0000,7.0500,'Skewness=-0.46','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6900,0.6000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',288,NULL,NULL,NULL,NULL,0.6900,0.5900,'N/A',0.6900,0.6000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over NR3C1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/NR3C1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(205,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','NR3C2 Affinity 1','NR3C2 Affinity 1 CT','1','NR3C2, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over NR3C2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to NR3C2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',465,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,11.0000,6.0500,'Skewness=0.43','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5700,0.6200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',80,NULL,NULL,NULL,NULL,0.5100,0.5900,'N/A',0.5700,0.6200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over NR3C2 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/NR3C2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(206,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','OPRD Affinity 1','OPRD Affinity 1 CT','1','OPRD, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over OPRD','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to OPRD extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4936,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.7000,6.5700,'Skewness=-0.18','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7900,0.7100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',870,NULL,NULL,NULL,NULL,0.7900,0.7100,'N/A',0.7900,0.7100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over OPRD or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/OPRD_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(207,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','OPRK Affinity 1','OPRK Affinity 1 CT','1','OPRK, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over OPRK','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to OPRK extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4832,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.5000,6.7200,'Skewness=0.01','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7600,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',852,NULL,NULL,NULL,NULL,0.7600,0.6800,'N/A',0.7600,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over OPRK or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/OPRK_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(208,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','PGR Affinity 1','PGR Affinity 1 CT','1','PGR, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over PGR','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to PGR extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',1546,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,4.0000,11.5000,6.8100,'Skewness=-0.09','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.6300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',272,NULL,NULL,NULL,NULL,0.6900,0.6200,'N/A',0.7000,0.6300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over PGR or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/PGR_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(209,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','RAR-alpha Affinity 1','RAR-alpha Affinity 1 CT','1','RAR-alpha, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over RAR-alpha','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to RAR-alpha extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',367,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.0000,5.9900,'Skewness=0.87','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6300,0.6200,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',65,NULL,NULL,NULL,NULL,0.6300,0.5900,'N/A',0.6300,0.6200,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over RAR-alpha or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/RAR-alpha_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(210,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SCN1A Affinity 1','SCN1A Affinity 1 CT','1','SCN1A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over SCN1A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SCN1A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',189,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.1000,8.1000,5.2800,'Skewness=-0.24','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8200,0.5000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',35,NULL,NULL,NULL,NULL,0.8200,0.5100,'N/A',0.8200,0.5000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SCN1A or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/SCN1A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(211,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SCN2A Affinity 1','SCN2A Affinity 1 CT','1','SCN2A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over SCN2A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SCN2A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',300,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.1000,8.1000,5.3400,'Skewness=-0.43','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7200,0.5500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',53,NULL,NULL,NULL,NULL,0.7600,0.5200,'N/A',0.7200,0.5500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SCN2A or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/SCN2A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(212,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SCN3A Affinity 1','SCN3A Affinity 1 CT','1','SCN3A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over SCN3A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SCN3A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',189,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.1000,8.1000,5.2800,'Skewness=-0.24','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.8200,0.5000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',35,NULL,NULL,NULL,NULL,0.8200,0.5000,'N/A',0.8200,0.5000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SCN3A or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/SCN3A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(213,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SCN5A Affinity 1','SCN5A Affinity 1 CT','1','SCN5A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over SCN5A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SCN5A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',262,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,3.0000,7.7000,5.0200,'Skewness=0.29','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7000,0.5500,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',45,NULL,NULL,NULL,NULL,0.7000,0.5400,'N/A',0.7000,0.5500,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SCN5A or none if compound is outside applicability domain','/Toxicity/Organ_Toxicity/Cardiotoxicity/SCN5A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(214,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SIGMAR1 Affinity 1','SIGMAR1 Affinity 1 CT','1','SIGMAR1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over SIGMAR1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SIGMAR1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3218,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.9000,11.0000,7.1600,'Skewness=-0.31','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.6300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',566,NULL,NULL,NULL,NULL,0.6300,0.6400,'N/A',0.6400,0.6300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SIGMAR1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/SIGMAR1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(215,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SLC6A2 Affinity 1','SLC6A2 Affinity 1 CT','1','SLC6A2, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over SLC6A2','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SLC6A2 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',3546,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.5000,11.5000,6.5300,'Skewness=-0.30','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6600,0.6300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',627,NULL,NULL,NULL,NULL,0.6700,0.6300,'N/A',0.6600,0.6300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SLC6A2 or none if compound is outside applicability domain','/ADME/Transport/Transporters/SLC6A2_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(216,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SLC6A3 Affinity 1','SLC6A3 Affinity 1 CT','1','SLC6A3, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over SLC6A3','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SLC6A3 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',4544,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.7000,6.4700,'Skewness=-0.24','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7300,0.6100,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',801,NULL,NULL,NULL,NULL,0.7300,0.6100,'N/A',0.7300,0.6100,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SLC6A3 or none if compound is outside applicability domain','/ADME/Transport/Transporters/SLC6A3_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(217,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','SLC6A4 Affinity 1','SLC6A4 Affinity 1 CT','1','SLC6A4, Affinity, ADME','All CT models','Estimated -log10 affinity of probe compound over SLC6A4','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to SLC6A4 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',6088,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,11.5000,6.8500,'Skewness=-0.31','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.7400,0.6300,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',1073,NULL,NULL,NULL,NULL,0.7400,0.6300,'N/A',0.7400,0.6300,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over SLC6A4 or none if compound is outside applicability domain','/ADME/Transport/Transporters/SLC6A4_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(218,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','TBXAS1 Affinity 1','TBXAS1 Affinity 1 CT','1','TBXAS1, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over TBXAS1','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to TBXAS1 extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',871,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.0000,9.0000,6.6400,'Skewness=-0.78','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.5100,0.6900,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',154,NULL,NULL,NULL,NULL,0.5100,0.6900,'N/A',0.5100,0.6900,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over TBXAS1 or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/TBXAS1_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(219,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','TOP2A Affinity 1','TOP2A Affinity 1 CT','1','TOP2A, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over TOP2A','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to TOP2A extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',292,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.2000,7.7000,4.1800,'Skewness=0.37','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6800,0.7000,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',50,NULL,NULL,NULL,NULL,0.6700,0.6900,'N/A',0.6800,0.7000,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over TOP2A or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/TOP2A_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(220,'2014-09-23 00:00:00','Ricard Garcia','ricard.garcia@chemotargets.com','N/A','TOP2B Affinity 1','TOP2B Affinity 1 CT','1','TOP2B, Affinity, Toxicity','All CT models','Estimated -log10 affinity of probe compound over TOP2B','pAffinity [-1 * log(affinity)] where affinity can be expressed in IC50, EC50, Ki, Kd or Kb','Ligands annotated to TOP2B extracted from public databases of ligand-protein interactions including BindingDB, ChemblDB, DrugBank, IUPHARdb, PDSP and PubchemBioassays',213,'Public domain','Compounds are standardized and an average affinity value is obtained if there is more than one measure',NULL,2.2000,7.7000,4.2700,'Skewness=0.36','Inchi key','N/A','kNN','Consensus prediction based on inverse distance weighting of neighbours detected using 2D descriptors and complemented with cross-pharmacology analysis','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds','Three descritptors are used: 1. PHRAGs are pharmacophoric-based descriptors derived from a direct fragmentation of the molecular graph into overlapping segments of a fixed length 2. FPDs are atom-pair molecular descriptors that capture the occurrence of a particular pair of features at different predefined bond-path distance ranges 3. SHEDs are derived from FPD by applying the concept of Shannon Entropy to determine the variability of the bin population along each feature-pair','N/A','0 if compound is in applicability domain or 6 if not','Compound is inside AD if model can raise a prediction',NULL,NULL,NULL,NULL,0.6400,0.7400,'N/A','The 15% of compounds was selected for the test set following the affinity distribution of the other 85% of training compounds',37,NULL,NULL,NULL,NULL,0.6100,0.7400,'N/A',0.6400,0.7400,'N/A','N/A','N/A','Being all predictions leave-one-out, GOF_R2 can not be computed for this model','CT','Multiple','N/A','float','(2,12)','Outcome can be either -log10 of predicted affinity of compound over TOP2B or none if compound is outside applicability domain','/Toxicity/Target_Safety_Pharmacology/TOP2B_Affinity/1','Verification ongoing','M (in minus log scale)',NULL),(1517,'2012-05-21 00:00:00','InteLigand Support','support@inteligand.com','chOX Database (https://www.ebi.ac.uk/chembl/etox/)\r\nchEMBL Database (https://www.ebi.ac.uk/chembl/)\r\nWolber G & Langer T. (2005)  J Chem Inf Model, 45(1):160-169.','3D Ligand-Based Pharmacophore Model for Serine Beta-Lactamase-Like Protein Inhibition','LACTB Inhibition 1 IL','1','Serine beta-lactamase-like protein, Inhibition, Pharmacophore, LACTB',NULL,'LACTB Inhibition:Predicted based on the 3D-geometric fit of the chemical features of the query ligand to a 3-Dimensional-feature based pharmacophore model.  The likelihood for a molecule to inhibit serine beta-lactamase-like protein is assigned a positive categorical value. A low likelihood for a molecule to inhibit serine beta-lactamase-like protein is assigned a value of negative.','The Dependent variable is Positive, when the query molecule is aligned with the pharmacophore model (hit), and negative when the model does not align with the query molecule. Unknown is the result provided when a prediction is not possible.','A subset of 30 highly active molecules were selected from an initial pool of the 50 most active molecules for inhibition of serine beta-lactamase-like protein found in the chOX database',30,'The training set included 50 molecules with the highest inhibition activity values. A subset of 30 highly active molecules were selected after clustering based on multiple 3D chemical feature pharmacophore alignment experiments. Molecules from each cluster were selected based on chemical diversity.','The data was curated.  50 of the most active molecules were selected from the chOX database.  The molecules were converted from 2D smiles format to 3D (SDF) with a maximum of 500 conformations generated per molecule using LigandScout. Each conformation of each molecule was annotated with its chemical features (geometry and type).The molecules were clustered based on 3D- pharmacophore feature alignments. 30 representative molecules for each of the clusters were selected based on chemical diversity. ',100.00,NULL,NULL,NULL,NULL,NULL,NULL,'3D Chemical Feature Ligand Based pharmacophore model','Training set molecules in smiles format were converted to 3D structures with the pattern matching virtual screening algorithm called idbgen developed at InteLigand and implemented into LigandScout. Pharmacophore alignment clustering and Ligand-Based models were generated using Espresso, the ligand based modelling algorithm in LigandScout developed at InteLigand.','The test set included 67 inhibitors and 72 non-inhibitors molecules that were not in the training set and taken from the chOX database.','LigandScout uses the following pharmacophore feature types (chemical features): Hydrogen bond donors and acceptors (vectors (when direction is congruent) and/or spheres (when direction is variable), hydrophobic, positive and negative ionizable, aromatic (pi - pi- stacking and Cation-pi), coordination to metal ions (iron, zinc, magnesium, manganese) and excluded volumes (restricted space). Each feature is encoded with the 3D geometric location, and tolerances.','Ligand-based pharmacophore features were automatically added to the model based on the results of multiple alignment experiments of the 3 dimensional geometry of the pharmacophore features for each conformation of each ligand in the training set. The resulting model contains the best consensus solution of position and combination of features. An excluded volume coat is added to the model based on the shape of the set of ligands in the training set.','na','na',28,0,0,2,NULL,NULL,NULL,'The test set included 67 inhibitors and 72 not-inhibitors taken from the chOX database. The molecules were converted from 2D smiles format to 3D SDF format. A maximum of 500 conformers were generated for each molecule and exported in LDB format to be used for virtual screening with LigandScout.\r\n',134,55,28,44,7,NULL,NULL,'AUC 1,5,10,100 % = 0.00,0.29, 0.35, 0.60',NULL,NULL,'3D chemical feature Ligand-Based pharmacophore modelling','NA',NULL,NULL,'IL','human',NULL,'boolean','Positive, Negative','Molecules with the result of Positive have a likelihood to inhibit serine beta-lactamase-like protein. Molecules with the result of Negative have a low likelihood to inhibit serine beta-lactamase-like protein. Unknown indicates that a prediction could not be determined.','/Toxicity/Target Safety Pharmacology/LACTB Inhibition/1','Verified','LigandScout Pharmacophore Fit Score',NULL),(1518,'2016-12-14 00:00:00','James Rathman','james.rathman@mn-am.com','Chen et al. (DOI:10.1016/j.drudis.2016.02.015); Garcı’a-Corte’s, M. et al. (DOI:10.1016/j.jhep.2011.02.007)\r\n','Human DILI - ToxGPS','DILI Human 1 MN','1.0','DILI (drug-induced liver injury), QSAR','eTOX: DILI Human LJMU 1','Human Drug Induced Liver Injury\r\n','The original dataset has 3-level categorization: “most”, “less”, and “no” concern. The first two were agregated into “any concern” (positive, may induce liver injury), the last is used as “no concern” (negative, likely will not induce liver injury).','The training set has been compiled from the data published by Chen et al. (DOI:10.1016/j.drudis.2016.02.015). 1036 drugs administered to humans (via various routes) which have been annotated in terms of their DILI risk based on FDA drug labels and ‘adjudicated’, ‘well-vetted’ cases verified for causality. See also Garcı’a-Corte’s, M. et al. (DOI:10.1016/j.jhep.2011.02.007)',466,'Diverse drugs molecules','Initial dataset of 1036 drugs were curated in three steps. (1) Confirmation of chemical compounds identity was performed by comparision of chemical structures, names, and CAS numbers. All conflicts were resolved by chemists, done by MN-AM with collaboration with NCTR. (2) mical structures curation was split into several sub steps. Validation of atom/bond features, valence, radical states, charges, multiple fragments was done by in-house set of CSRML rules. Next, confirmation of stereochemistry and double bond geometry was performed. Finally, Identification and removing from the dataset inorganic and organometallic compounds, metal complexes and metals, natural products, mixtures, polymers, etc. was done. (3) paration of the structures for the computational analysis was performed by applying CORINA CLEAN workflow in Corina Symphony software tool. The final modelling set consisted of 523 drugs (single organic molecules) applied via oral route of exposure, the compounds with Ambiguous DILI-concern have been excluded. This dataset has been split into training set (466) and validation set (57).',74.00,NULL,NULL,NULL,NULL,NULL,NULL,'Logistic regression version of PLS','Multiple QSAR models were built and then a quantitative weight-of-evidence (WoE) method was applied  to combine predictions from individual models to arrive at a final prediction. The logistic PLS modelling approach fits the categorical dependent variable to the PLS factors, linear combinations of the ToxPrint, physicochemical, and quantum mechanical descriptors. Each model is optimized with respect to the number of ToxPrints and number of PLS factors. The WoE method is based on Dempster-Shafer decision theory and takes into account the reliability of each model, which is obtained from model validation metrics. The QSAR models include global models as well as “local” mode-of-action models that take mechanistic knowledge into account.','Random selection.','Three types of descriptors were used: (1) ToxPrint chemotype structural features; (2) physicochemical descriptors from CORINA (global molecular descriptors plus shape descriptors); (3) quantum mechanical descriptors (heat of formation, HOMO, LUMO).\r\nAll descriptors are calculated in CORINA Symphony software suite.\r\n','For the ToxPrint chemotypes, a simple chi2 test is used to pre-select a subset of ToxPrints that provide statistically significant correlation with the dependent variable. This test was done independently for each model. Similarly, for physicochemical and quantum mechanical descriptors, a simple t-test is used to pre-select a subset of that provide statistically significant correlation with the dependent variable.','The model is meant to be applicable to any drug-like or small organic compounds.\r\n','A query compound that does not have at least 3 compounds in the global training set within the predefined distance is tagged as out of domain. The distance used in AD method is the Euclidean distance between the observations in the continuous-variable space defined by the PLS factors. For each global model, distances between all pairs of compounds in the training set were calculated and used to determine the distance at which 95% of the compounds in the training set have at least 3 neighbours. This distance is used in the AD assessment of query compounds.',79,88,10,18,NULL,NULL,NULL,'Due to lack of reliable human DILI data other than our training set, for external validation we randomly selected a set of 57 compounds from the full training set.',57,78,69,25,15,NULL,NULL,NULL,NULL,NULL,'The model comprises of multiple models including some “local” mode-of-action models that take mechanistic knowledge into account.',NULL,NULL,'Equivocal predictions on positives for training set = 3%, equivocal predictions on negatives for training set = 2%. Equivocal predictions on positives for validation set = 7%, equivocal predictions on negatives for validation set = 6%.','MN','human',NULL,'boolean','negative, positive, equivocal','Positive prediction means that compounds are likely to cause DILI concern. Negative prediction means that compounds are likely not to cause DILI concern. Equivocal prediction means uncertain prediction.','/Toxicity/Organ Toxicity/Hepatotoxicity/DILI Human/2 ','Not verified',NULL,NULL);
/*!40000 ALTER TABLE `model` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `modelsoftware`
--

LOCK TABLES `modelsoftware` WRITE;
/*!40000 ALTER TABLE `modelsoftware` DISABLE KEYS */;
INSERT INTO `modelsoftware` VALUES (1,56,1,3),(2,58,43,2),(3,59,1,3),(4,60,1,3),(5,61,1,3),(253,164,42,2),(7,56,3,1),(8,56,4,1),(9,55,3,1),(359,120,47,2),(358,120,46,2),(357,120,45,2),(13,22,10,1),(356,120,44,2),(107,85,18,2),(106,85,19,2),(17,55,3,2),(234,145,42,2),(19,56,12,2),(20,56,13,2),(21,56,14,2),(22,56,15,2),(23,56,16,2),(24,56,17,2),(25,56,18,2),(26,56,19,2),(355,120,43,2),(575,58,58,2),(353,58,52,2),(574,58,57,2),(351,58,50,2),(350,58,49,2),(33,59,33,2),(175,58,44,2),(35,59,23,2),(176,59,34,2),(250,161,42,2),(252,163,42,2),(251,162,42,2),(249,160,42,2),(41,60,33,2),(177,60,34,2),(43,60,23,2),(178,61,34,2),(45,61,33,2),(179,58,45,2),(47,61,23,2),(180,59,35,2),(49,52,29,2),(50,43,30,2),(51,47,30,2),(52,41,30,2),(53,49,30,2),(54,51,30,2),(55,50,30,2),(56,42,30,2),(57,32,30,2),(58,39,30,2),(59,31,30,2),(60,27,30,2),(61,28,30,2),(62,29,30,2),(63,30,30,2),(64,33,30,2),(65,35,30,2),(66,34,30,2),(67,37,30,2),(68,38,30,2),(69,36,30,2),(70,40,30,2),(71,46,30,2),(72,26,30,2),(73,45,30,2),(74,44,30,2),(75,48,30,2),(229,139,42,2),(228,142,42,2),(105,85,14,1),(227,123,42,2),(226,124,42,2),(225,127,42,2),(224,126,42,2),(223,129,42,2),(222,128,42,2),(221,130,42,2),(220,132,42,2),(219,131,42,2),(218,136,42,2),(217,135,42,2),(216,134,42,2),(215,133,42,2),(214,137,42,2),(213,138,42,2),(212,140,42,2),(211,141,42,2),(96,22,10,2),(97,22,11,2),(98,53,32,2),(99,58,46,2),(181,60,35,2),(101,58,47,2),(182,61,35,2),(104,NULL,NULL,NULL),(108,85,12,1),(109,85,15,2),(110,85,15,1),(111,85,16,1),(112,85,17,1),(113,85,16,2),(114,85,17,2),(115,85,15,3),(116,85,16,3),(117,85,17,3),(118,86,14,1),(119,86,12,1),(120,86,15,1),(121,86,16,1),(122,86,17,1),(123,86,19,2),(124,86,18,2),(125,86,15,2),(126,86,16,2),(127,86,17,2),(128,86,15,3),(129,86,16,3),(130,86,17,3),(131,87,14,1),(132,87,12,1),(133,87,15,1),(134,87,16,1),(135,87,17,1),(136,87,19,2),(137,87,18,2),(138,87,15,2),(139,87,16,2),(140,87,17,2),(141,87,15,3),(142,87,16,3),(143,87,17,3),(144,88,14,1),(145,88,12,1),(146,88,15,1),(147,88,16,1),(148,88,17,1),(149,88,19,2),(150,88,18,2),(151,88,15,2),(152,88,16,2),(153,88,17,2),(154,88,15,3),(155,88,16,3),(156,88,17,3),(246,157,42,2),(242,153,42,2),(238,149,42,2),(248,159,42,2),(245,156,42,2),(241,152,42,2),(237,148,42,2),(247,158,42,2),(244,155,42,2),(240,151,42,2),(236,147,42,2),(243,154,42,2),(239,150,42,2),(235,146,42,2),(183,58,48,2),(184,59,36,2),(185,60,36,2),(186,61,36,2),(202,73,32,2),(201,72,32,2),(200,71,32,2),(199,70,32,2),(191,68,6,2),(192,68,39,2),(193,68,38,2),(194,68,37,1),(233,144,42,2),(232,143,42,2),(231,122,42,2),(230,125,42,2),(203,74,32,2),(204,75,32,1),(205,75,40,2),(206,76,32,1),(207,76,40,2),(208,77,32,1),(209,77,40,2),(210,54,13,1),(254,165,42,2),(255,166,42,2),(256,167,42,2),(257,168,42,2),(258,169,42,2),(259,170,42,2),(260,171,42,2),(261,172,42,2),(262,173,42,2),(263,174,42,2),(264,175,42,2),(265,176,42,2),(266,177,42,2),(267,178,42,2),(268,179,42,2),(269,180,42,2),(270,181,42,2),(271,182,42,2),(272,183,42,2),(273,184,42,2),(274,185,42,2),(275,186,42,2),(276,187,42,2),(277,188,42,2),(278,189,42,2),(279,190,42,2),(280,191,42,2),(281,192,42,2),(282,193,42,2),(283,194,42,2),(284,195,42,2),(285,196,42,2),(286,197,42,2),(287,198,42,2),(288,199,42,2),(289,200,42,2),(290,201,42,2),(291,202,42,2),(292,203,42,2),(293,204,42,2),(294,205,42,2),(295,206,42,2),(296,207,42,2),(297,208,42,2),(298,209,42,2),(299,210,42,2),(300,211,42,2),(301,212,42,2),(302,213,42,2),(303,214,42,2),(304,215,42,2),(305,216,42,2),(306,217,42,2),(307,218,42,2),(308,219,42,2),(309,220,42,2),(310,246,7,1),(311,246,5,2),(312,246,12,2),(313,247,7,1),(314,247,5,2),(315,247,12,2),(316,248,7,1),(317,248,5,2),(318,248,12,2),(319,249,7,1),(320,249,5,2),(321,249,12,2),(322,250,7,1),(323,250,5,2),(324,250,12,2),(325,251,7,1),(326,251,5,2),(327,251,12,2),(328,252,7,1),(329,252,5,2),(330,252,12,2),(331,253,7,1),(332,253,5,2),(333,253,12,2),(334,254,7,1),(335,254,5,2),(336,254,12,2),(337,255,7,1),(338,255,5,2),(339,255,12,2),(340,256,7,1),(341,256,5,2),(342,256,12,2),(343,257,5,2),(344,257,12,2),(345,258,7,1),(346,258,5,2),(347,258,12,2),(348,259,7,1),(349,259,12,2),(360,120,48,2),(361,120,49,2),(362,120,50,2),(363,120,51,2),(364,120,52,2),(365,120,53,2),(408,326,7,1),(393,350,5,2),(394,353,5,2),(398,324,5,2),(396,357,5,2),(416,353,7,1),(414,338,7,1),(412,342,7,1),(401,330,5,2),(402,339,7,1),(405,329,7,1),(407,350,7,1),(411,341,7,1),(409,336,7,1),(392,342,5,2),(395,336,5,2),(399,346,5,2),(397,327,5,2),(417,357,7,1),(415,356,7,1),(413,346,7,1),(400,339,5,2),(403,330,7,1),(404,324,7,1),(406,327,7,1),(410,332,7,1),(418,352,7,1),(419,355,7,1),(420,345,7,1),(421,340,20,2),(422,342,20,2),(423,332,20,2),(424,343,20,2),(425,353,20,2),(426,354,20,2),(427,331,20,2),(428,339,20,2),(429,341,20,2),(430,355,20,2),(431,330,20,2),(432,344,54,1),(433,354,54,1),(434,325,54,1),(435,351,54,1),(436,331,54,1),(437,358,54,1),(438,340,54,1),(439,328,54,1),(440,343,54,1),(441,337,54,1),(442,326,55,2),(443,341,55,2),(444,355,55,2),(445,329,55,2),(446,356,55,2),(447,345,55,2),(448,352,55,2),(449,332,55,2),(450,338,55,2),(451,352,56,2),(452,358,56,2),(453,346,56,2),(454,345,56,2),(455,357,56,2),(456,337,56,2),(457,328,56,2),(458,324,56,2),(459,356,56,2),(460,344,56,2),(461,326,56,2),(462,351,56,2),(463,329,56,2),(464,338,56,2),(465,327,56,2),(466,350,56,2),(467,336,56,2),(468,325,56,2),(469,475,54,1),(470,475,20,2),(471,476,54,1),(472,476,56,2),(473,477,55,2),(474,477,7,1),(475,477,56,2),(476,478,54,1),(477,478,20,2),(478,479,54,1),(479,479,56,2),(480,480,56,2),(481,480,7,1),(482,480,55,2),(483,481,56,2),(484,481,54,1),(485,482,56,2),(486,482,54,1),(487,483,55,2),(488,483,56,2),(489,483,7,1),(490,487,20,2),(491,487,54,1),(492,488,56,2),(493,488,54,1),(494,489,55,2),(495,489,7,1),(496,489,56,2),(497,490,20,2),(498,490,54,1),(499,491,54,1),(500,491,56,2),(501,492,55,2),(502,492,7,1),(503,492,56,2),(504,493,54,1),(505,493,20,2),(506,494,54,1),(507,494,56,2),(508,495,7,1),(509,495,55,2),(510,495,56,2),(511,496,20,2),(512,496,54,1),(513,497,56,2),(514,497,54,1),(515,498,56,2),(516,498,55,2),(517,498,7,1),(518,499,20,2),(519,499,54,1),(520,500,54,1),(521,500,56,2),(522,502,20,2),(523,502,54,1),(524,503,54,1),(525,503,56,2),(526,504,55,2),(527,504,56,2),(528,504,7,1),(529,501,55,2),(530,501,7,1),(531,501,56,2),(532,505,54,1),(533,505,20,2),(534,506,54,1),(535,506,56,2),(536,507,55,2),(537,507,7,1),(538,507,56,2),(539,508,20,2),(540,508,54,1),(541,509,56,2),(542,509,54,1),(543,510,7,1),(544,510,55,2),(545,510,56,2),(546,511,20,2),(547,511,54,1),(548,512,56,2),(549,512,54,1),(550,513,56,2),(551,513,55,2),(552,513,7,1),(553,514,54,1),(554,514,20,2),(555,515,54,1),(556,515,56,2),(557,516,7,1),(558,516,56,2),(559,516,55,2),(560,517,20,2),(561,517,54,1),(562,518,54,1),(563,518,56,2),(564,519,7,1),(565,519,56,2),(566,519,55,2),(567,520,54,1),(568,520,20,2),(569,521,56,2),(570,521,54,1),(571,522,7,1),(572,522,55,2),(573,522,56,2),(576,523,43,2),(577,523,44,2),(578,523,45,2),(579,523,46,2),(580,523,47,2),(581,523,48,2),(582,523,49,2),(583,523,50,2),(584,523,52,2),(585,523,57,2),(586,523,58,2),(587,524,43,2),(588,524,44,2),(589,524,45,2),(590,524,46,2),(591,524,47,2),(592,524,48,2),(593,524,49,2),(594,524,50,2),(595,524,52,2),(596,524,57,2),(597,524,58,2),(598,1514,43,2),(599,1514,44,2),(600,1514,45,2),(601,1514,46,2),(602,1514,47,2),(603,1514,48,2),(604,1514,49,2),(605,1514,50,2),(606,1514,52,2),(607,1514,57,2),(608,1514,58,2),(609,1515,43,2),(610,1515,44,2),(611,1515,45,2),(612,1515,46,2),(613,1515,47,2),(614,1515,48,2),(615,1515,49,2),(616,1515,50,2),(617,1515,52,2),(618,1515,57,2),(619,1515,58,2),(620,NULL,30,2),(621,NULL,NULL,NULL);
/*!40000 ALTER TABLE `modelsoftware` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `partner`
--

LOCK TABLES `partner` WRITE;
/*!40000 ALTER TABLE `partner` DISABLE KEYS */;
INSERT INTO `partner` VALUES ('CT'),('DTU'),('FIMIM'),('FIMIM2'),('IL'),('LJMU'),('LL'),('LMD'),('MN'),('UNIVIE'),('VUA');
/*!40000 ALTER TABLE `partner` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `software`
--

LOCK TABLES `software` WRITE;
/*!40000 ALTER TABLE `software` DISABLE KEYS */;
INSERT INTO `software` VALUES (1,'In house python2 implementation',NULL,NULL,NULL,NULL,NULL),(2,'Molecular Operating Environment',' MOE 2010','Chemical Computing Group Inc',NULL,'NO','commercial'),(3,'AdrianaCode',NULL,'Molecular Networks',' ','YES','commercial'),(4,'Moses library',NULL,'Molecular Networks',NULL,'YES','commercial'),(5,'Moka','1.1.0','Molecular Discovery','Linux','NO','commercial'),(6,'OpenBabel','2.3.0',NULL,'Linux','NO','GNU GPL'),(7,'Pentacle','1.06','Molecular Discovery','Linux','NO','commercial'),(10,'RCDK',NULL,'Rajarshi Guha <rajarshi.guha@gmail.com>','Linux','NO','GNU LGPL'),(11,'WEKA',NULL,NULL,NULL,'NO','GNU GPL'),(12,'Corina','3.47','Molecular Networks','Linux','YES','commercial'),(13,'AdrianaCode','2.2.4','Molecular Networks',NULL,'YES','commercial'),(14,'Moses',NULL,'Molecular Networks',NULL,'YES','commercial'),(15,'Python','2.6',NULL,NULL,'NO','Python Software Fundation v2'),(16,'NumPy','1.6.1',NULL,NULL,'NO','BSD'),(17,'Scipy','0.10.0',NULL,NULL,'NO','BSD'),(18,'Jython','2.5.2',NULL,NULL,'NO','Python Software Fundation v2, Jython 2.1, Apache'),(19,'WEKA','3.6.6',NULL,NULL,'NO','GNU GPL 2.0'),(20,'Corina','2.4','Molecular Networks','Linux','YES','commercial'),(21,'GOLD','3.2','CCDC','Linux','NO','commercial'),(22,'GROMOS','2005','Biomos bv',NULL,'NO','commercial'),(23,'GROMACS','4.5.4',NULL,NULL,'NO','GNU GPL 3.0'),(24,'GROMOS MD++','2005','Biomos bv',NULL,'NO','commercial'),(25,'Instant Jchem','5.3','ChemAxon',NULL,'NO','commercial'),(26,'Standarizer',NULL,'ChemAxon',NULL,'NO','commercial'),(27,'WEKA','3.6',NULL,NULL,'NO','GNU GPL 2.0'),(28,'KNIME','2.4.0',NULL,NULL,'NO','GNU GPL 3.0'),(29,'KNIME','2.5.4',NULL,'Windows 7','NO','GPL 3.0'),(30,'LigandScout','4.0','InteLigand','Windows, Linux, Mac','YES','commercial'),(31,'PredictFX',NULL,'Certara',NULL,'NO','commercial'),(32,'Volsurf','1.0.4','Molecular Discovery','Linux','NO','commercial'),(33,'PLANTS','1.2','Universitat Konstanz',NULL,'NO','commercial'),(34,'OpenBabel-2.2.3','2.2.3',NULL,NULL,'NO','GNU GPL'),(35,'ATB','1.2','University of Queensland',NULL,'NO','commercial'),(36,'Pybel',NULL,NULL,NULL,'NO','GNU GPL; requires OpenBabel installation'),(37,'SHOP (scaffold hopping software)','2.0.5','Molecular Discovery','Linux Centos or Red Hat','NO','commercial'),(38,'Virtual Reaction ',NULL,'Molecular Discovery','Linux Centos or Red Hat','NO','commercial'),(39,'Mizer (3D conformation generation)',NULL,'Molecular Discovery','Linux Centos or Red Hat','NO','commercial'),(40,'libsvm (Support Vector Machine implementation)','3.1.2','Chih-Chung Chang and Chih-Jen Lin research group at National Taiwan University','Linux','NO','modified BSD license (http://www.csie.ntu.edu.tw/~cjlin/libsvm/faq.html#f203)'),(42,'Ctlinks',NULL,'Chemotargets S.L.',NULL,'NO','commercial'),(43,'ParaDocks','1.0.1','See http://europa.pharmazie.uni-halle.de/joomla/index.php/downloads',NULL,'NO','GPLv2'),(44,'AmberTools','15','See http://ambermd.org/AmberTools15-get.html',NULL,'NO','GPLv3'),(45,'AcPype','2014-04-21 Rev401','See http://code.google.com/p/acpype/',NULL,'NO','GPLv3'),(46,'OpenBabel','2.3.90','See http://openbabel.org/wiki/Main_Page','Linux','NO','GPLv2'),(47,'GROMACS','4.5.5','See http://www.gromacs.org/',NULL,'NO','LPGLv2.1'),(48,'Scipy','0.17.0','See http://www.scipy.org/index.html',NULL,'NO','BSD'),(49,'Numpy','1.10.4','See http://www.numpy.org/',NULL,'NO','BSD'),(50,'Scikit-learn','0.17','See http://scikit-learn.org/stable/',NULL,'NO','New BSD'),(51,'Beautiful-soap','4.3.2','See https://pypi.python.org/pypi/beautifulsoup4/4.3.2',NULL,'NO','MIT'),(52,'paramiko','1.16.0','See https://pypi.python.org/pypi/paramiko/1.16.0',NULL,'NO','LGPL'),(53,'logging','0.5.0.5','See https://pypi.python.org/pypi/logging',NULL,'NO','http://www.red-dove.com/python_logging.html#license'),(54,'AdrianaCode','2.2.6','Molecular Networks',NULL,'YES','commercial'),(55,'Moka','3.0','Molecular Discovery','Linux','NO','commercial'),(56,'Corina','3.494','Molecular Networks','Linux','YES','commercial'),(57,'logging','0.5.1.2','See https://pypi.python.org/pypi/logging',NULL,'NO','http://www.red-dove.com/python_logging.html#license'),(58,'pandas','0.17.1','See pandas.pydata.org',NULL,'NO','BSD');
/*!40000 ALTER TABLE `software` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `softwaretype`
--

LOCK TABLES `softwaretype` WRITE;
/*!40000 ALTER TABLE `softwaretype` DISABLE KEYS */;
INSERT INTO `softwaretype` VALUES (1,'Descriptor generation'),(2,'Modelling'),(3,'Applicability domain');
/*!40000 ALTER TABLE `softwaretype` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `species`
--

LOCK TABLES `species` WRITE;
/*!40000 ALTER TABLE `species` DISABLE KEYS */;
INSERT INTO `species` VALUES ('human'),('mammal'),('mouse'),('Multiple'),('N/A'),('non-rodent'),('rat'),('rodent');
/*!40000 ALTER TABLE `species` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2017-07-24  8:44:33
